{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "Trinity_tutorial_FINAL.ipynb",
      "provenance": [],
      "collapsed_sections": []
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    },
    "language_info": {
      "name": "python"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "c3uAuilaaqbl"
      },
      "source": [
        "**[Advanced Bioinformatics] Trinity tutorial**"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "qcsK43rCbGeO"
      },
      "source": [
        "**00-1. Mount Google Drive**"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "20euKaLGa9Vn",
        "outputId": "387cb7ad-4351-4112-b7c8-5a611ebef34c"
      },
      "source": [
        "from google.colab import drive\n",
        "drive.mount('/content/drive')"
      ],
      "execution_count": 2,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Drive already mounted at /content/drive; to attempt to forcibly remount, call drive.mount(\"/content/drive\", force_remount=True).\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "**00-2. Open terminal**"
      ],
      "metadata": {
        "id": "pYyyjz8JoUxn"
      }
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "c0zdF9zTb4Cr"
      },
      "source": [
        "**01.Installation**\n",
        "\n",
        "Trinity runs in the Linux environment and requires the following additional tools to run;\n",
        "\n",
        "1.  samtools\n",
        "2.  bowtie2\n",
        "3. jellyfish\n",
        "4. salmon\n",
        "5. python with numpy\n",
        "6. Trinity\n",
        "\n",
        "\n"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#INSTALLING JELLYFISH\n",
        "!sudo apt update\n",
        "!sudo apt install jellyfish"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "7aD34kmltDP8",
        "outputId": "c5fc715f-a0d8-41ed-95ad-59e488d8a9b4"
      },
      "execution_count": 3,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[33m\r0% [Working]\u001b[0m\r            \rGet:1 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]\n",
            "\u001b[33m\r0% [Connecting to archive.ubuntu.com (91.189.88.142)] [1 InRelease 14.2 kB/88.7\u001b[0m\u001b[33m\r0% [Connecting to archive.ubuntu.com (91.189.88.142)] [Connected to cloud.r-pro\u001b[0m\u001b[33m\r0% [1 InRelease gpgv 88.7 kB] [Waiting for headers] [Waiting for headers] [Wait\u001b[0m\r                                                                               \rGet:2 https://cloud.r-project.org/bin/linux/ubuntu bionic-cran40/ InRelease [3,626 B]\n",
            "Ign:3 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  InRelease\n",
            "Get:4 http://ppa.launchpad.net/c2d4u.team/c2d4u4.0+/ubuntu bionic InRelease [15.9 kB]\n",
            "Hit:5 http://archive.ubuntu.com/ubuntu bionic InRelease\n",
            "Ign:6 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  InRelease\n",
            "Hit:7 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  Release\n",
            "Hit:8 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  Release\n",
            "Get:9 http://archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]\n",
            "Get:10 http://security.ubuntu.com/ubuntu bionic-security/restricted amd64 Packages [691 kB]\n",
            "Get:11 http://security.ubuntu.com/ubuntu bionic-security/main amd64 Packages [2,461 kB]\n",
            "Hit:12 http://ppa.launchpad.net/cran/libgit2/ubuntu bionic InRelease\n",
            "Get:13 http://security.ubuntu.com/ubuntu bionic-security/universe amd64 Packages [1,450 kB]\n",
            "Get:14 http://archive.ubuntu.com/ubuntu bionic-backports InRelease [74.6 kB]\n",
            "Get:15 http://ppa.launchpad.net/deadsnakes/ppa/ubuntu bionic InRelease [15.9 kB]\n",
            "Hit:16 http://ppa.launchpad.net/graphics-drivers/ppa/ubuntu bionic InRelease\n",
            "Get:19 http://ppa.launchpad.net/c2d4u.team/c2d4u4.0+/ubuntu bionic/main Sources [1,819 kB]\n",
            "Get:20 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 Packages [2,898 kB]\n",
            "Get:21 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 Packages [2,228 kB]\n",
            "Get:22 http://ppa.launchpad.net/c2d4u.team/c2d4u4.0+/ubuntu bionic/main amd64 Packages [933 kB]\n",
            "Get:23 http://ppa.launchpad.net/deadsnakes/ppa/ubuntu bionic/main amd64 Packages [45.9 kB]\n",
            "Fetched 12.8 MB in 3s (4,088 kB/s)\n",
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "58 packages can be upgraded. Run 'apt list --upgradable' to see them.\n",
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "The following additional packages will be installed:\n",
            "  libjellyfish-2.0-2\n",
            "The following NEW packages will be installed:\n",
            "  jellyfish libjellyfish-2.0-2\n",
            "0 upgraded, 2 newly installed, 0 to remove and 58 not upgraded.\n",
            "Need to get 418 kB of archives.\n",
            "After this operation, 808 kB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu bionic/universe amd64 libjellyfish-2.0-2 amd64 2.2.8-3build1 [60.7 kB]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu bionic/universe amd64 jellyfish amd64 2.2.8-3build1 [358 kB]\n",
            "Fetched 418 kB in 1s (556 kB/s)\n",
            "debconf: unable to initialize frontend: Dialog\n",
            "debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 76, <> line 2.)\n",
            "debconf: falling back to frontend: Readline\n",
            "debconf: unable to initialize frontend: Readline\n",
            "debconf: (This frontend requires a controlling tty.)\n",
            "debconf: falling back to frontend: Teletype\n",
            "dpkg-preconfigure: unable to re-open stdin: \n",
            "Selecting previously unselected package libjellyfish-2.0-2:amd64.\n",
            "(Reading database ... 155222 files and directories currently installed.)\n",
            "Preparing to unpack .../libjellyfish-2.0-2_2.2.8-3build1_amd64.deb ...\n",
            "Unpacking libjellyfish-2.0-2:amd64 (2.2.8-3build1) ...\n",
            "Selecting previously unselected package jellyfish.\n",
            "Preparing to unpack .../jellyfish_2.2.8-3build1_amd64.deb ...\n",
            "Unpacking jellyfish (2.2.8-3build1) ...\n",
            "Setting up libjellyfish-2.0-2:amd64 (2.2.8-3build1) ...\n",
            "Setting up jellyfish (2.2.8-3build1) ...\n",
            "Processing triggers for libc-bin (2.27-3ubuntu1.3) ...\n",
            "/sbin/ldconfig.real: /usr/local/lib/python3.7/dist-packages/ideep4py/lib/libmkldnn.so.0 is not a symbolic link\n",
            "\n",
            "Processing triggers for man-db (2.8.3-2ubuntu0.1) ...\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#INSTALLING NUMPY\n",
        "!pip install numpy\n",
        "import numpy"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "AwVmXmrqpVuy",
        "outputId": "ff9e5dd6-be3f-4b92-ebb3-2350119c1c9d"
      },
      "execution_count": 4,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Requirement already satisfied: numpy in /usr/local/lib/python3.7/dist-packages (1.19.5)\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#INSTALLING BOWTIE\n",
        "#INSTALL BOWTIE2\n",
        "!sudo apt install bowtie2\n",
        "!sudo apt install bowtie"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "yqx3b-l-sufD",
        "outputId": "98b508e4-83ac-4230-eacc-9eff33fb3620"
      },
      "execution_count": 5,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "Suggested packages:\n",
            "  bowtie2-examples\n",
            "The following NEW packages will be installed:\n",
            "  bowtie2\n",
            "0 upgraded, 1 newly installed, 0 to remove and 58 not upgraded.\n",
            "Need to get 1,174 kB of archives.\n",
            "After this operation, 5,100 kB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu bionic/universe amd64 bowtie2 amd64 2.3.4.1-1 [1,174 kB]\n",
            "Fetched 1,174 kB in 1s (1,270 kB/s)\n",
            "debconf: unable to initialize frontend: Dialog\n",
            "debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 76, <> line 1.)\n",
            "debconf: falling back to frontend: Readline\n",
            "debconf: unable to initialize frontend: Readline\n",
            "debconf: (This frontend requires a controlling tty.)\n",
            "debconf: falling back to frontend: Teletype\n",
            "dpkg-preconfigure: unable to re-open stdin: \n",
            "Selecting previously unselected package bowtie2.\n",
            "(Reading database ... 155237 files and directories currently installed.)\n",
            "Preparing to unpack .../bowtie2_2.3.4.1-1_amd64.deb ...\n",
            "Unpacking bowtie2 (2.3.4.1-1) ...\n",
            "Setting up bowtie2 (2.3.4.1-1) ...\n",
            "Processing triggers for man-db (2.8.3-2ubuntu0.1) ...\n",
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "Suggested packages:\n",
            "  bowtie-examples\n",
            "The following NEW packages will be installed:\n",
            "  bowtie\n",
            "0 upgraded, 1 newly installed, 0 to remove and 58 not upgraded.\n",
            "Need to get 1,686 kB of archives.\n",
            "After this operation, 7,343 kB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu bionic/universe amd64 bowtie amd64 1.2.2+dfsg-2 [1,686 kB]\n",
            "Fetched 1,686 kB in 1s (1,835 kB/s)\n",
            "debconf: unable to initialize frontend: Dialog\n",
            "debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 76, <> line 1.)\n",
            "debconf: falling back to frontend: Readline\n",
            "debconf: unable to initialize frontend: Readline\n",
            "debconf: (This frontend requires a controlling tty.)\n",
            "debconf: falling back to frontend: Teletype\n",
            "dpkg-preconfigure: unable to re-open stdin: \n",
            "Selecting previously unselected package bowtie.\n",
            "(Reading database ... 155263 files and directories currently installed.)\n",
            "Preparing to unpack .../bowtie_1.2.2+dfsg-2_amd64.deb ...\n",
            "Unpacking bowtie (1.2.2+dfsg-2) ...\n",
            "Setting up bowtie (1.2.2+dfsg-2) ...\n",
            "Processing triggers for man-db (2.8.3-2ubuntu0.1) ...\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#INSTALLING SAMTOOLS\n",
        "!sudo apt install samtools"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "I1mBXZu9x3u3",
        "outputId": "37e9b824-75ad-46d5-f037-a6f3135dbf80"
      },
      "execution_count": 6,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "The following additional packages will be installed:\n",
            "  cwltool libc-ares2 libhts2 libjs-bootstrap nodejs nodejs-doc\n",
            "  python-asn1crypto python-avro python-cachecontrol python-certifi\n",
            "  python-cffi-backend python-chardet python-cryptography python-enum34\n",
            "  python-html5lib python-idna python-ipaddress python-isodate python-lockfile\n",
            "  python-mistune python-openssl python-pkg-resources python-pyparsing\n",
            "  python-rdflib python-rdflib-jsonld python-requests python-ruamel.yaml\n",
            "  python-schema-salad python-shellescape python-six python-sparqlwrapper\n",
            "  python-typing python-urllib3 python-webencodings\n",
            "Suggested packages:\n",
            "  python-cryptography-doc python-cryptography-vectors python-enum34-doc\n",
            "  python-genshi python-lxml python-lockfile-doc python-openssl-doc\n",
            "  python-openssl-dbg python-setuptools python-pyparsing-doc python-rdflib-doc\n",
            "  python-rdflib-tools python-socks python-ntlm\n",
            "The following NEW packages will be installed:\n",
            "  cwltool libc-ares2 libhts2 libjs-bootstrap nodejs nodejs-doc\n",
            "  python-asn1crypto python-avro python-cachecontrol python-certifi\n",
            "  python-cffi-backend python-chardet python-cryptography python-enum34\n",
            "  python-html5lib python-idna python-ipaddress python-isodate python-lockfile\n",
            "  python-mistune python-openssl python-pkg-resources python-pyparsing\n",
            "  python-rdflib python-rdflib-jsonld python-requests python-ruamel.yaml\n",
            "  python-schema-salad python-shellescape python-six python-sparqlwrapper\n",
            "  python-typing python-urllib3 python-webencodings samtools\n",
            "0 upgraded, 35 newly installed, 0 to remove and 58 not upgraded.\n",
            "Need to get 8,568 kB of archives.\n",
            "After this operation, 41.0 MB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-pkg-resources all 39.0.1-2 [128 kB]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-six all 1.11.0-2 [11.3 kB]\n",
            "Get:3 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-isodate all 0.6.0-1 [23.8 kB]\n",
            "Get:4 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-pyparsing all 2.2.0+dfsg1-2 [52.1 kB]\n",
            "Get:5 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-rdflib all 4.2.1-2 [237 kB]\n",
            "Get:6 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-certifi all 2018.1.18-2 [144 kB]\n",
            "Get:7 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-chardet all 3.0.4-1 [80.3 kB]\n",
            "Get:8 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-idna all 2.6-1 [32.4 kB]\n",
            "Get:9 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python-urllib3 all 1.22-1ubuntu0.18.04.2 [86.0 kB]\n",
            "Get:10 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python-requests all 2.18.4-2ubuntu0.1 [58.5 kB]\n",
            "Get:11 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-ruamel.yaml amd64 0.15.34-1 [183 kB]\n",
            "Get:12 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-cachecontrol all 0.11.7-1 [15.2 kB]\n",
            "Get:13 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-lockfile all 1:0.12.2-2 [14.6 kB]\n",
            "Get:14 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-mistune all 0.8.3-2 [13.8 kB]\n",
            "Get:15 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-rdflib-jsonld all 0.4.0-3 [12.2 kB]\n",
            "Get:16 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-typing all 3.6.2-1 [22.2 kB]\n",
            "Get:17 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-avro all 1.8.2+dfsg-1 [23.2 kB]\n",
            "Get:18 http://archive.ubuntu.com/ubuntu bionic/universe amd64 libjs-bootstrap all 3.3.7+dfsg-2 [228 kB]\n",
            "Get:19 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-schema-salad all 2.6.20171201034858-3 [260 kB]\n",
            "Get:20 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-shellescape all 3.4.1-2 [4,432 B]\n",
            "Get:21 http://archive.ubuntu.com/ubuntu bionic/universe amd64 cwltool all 1.0.20180302231433-1 [143 kB]\n",
            "Get:22 http://archive.ubuntu.com/ubuntu bionic/universe amd64 libhts2 amd64 1.7-2 [300 kB]\n",
            "Get:23 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 nodejs-doc all 8.10.0~dfsg-2ubuntu0.4 [752 kB]\n",
            "Get:24 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-asn1crypto all 0.24.0-1 [72.7 kB]\n",
            "Get:25 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-cffi-backend amd64 1.11.5-1 [63.4 kB]\n",
            "Get:26 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-enum34 all 1.1.6-2 [34.8 kB]\n",
            "Get:27 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-ipaddress all 1.0.17-1 [18.2 kB]\n",
            "Get:28 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 python-cryptography amd64 2.1.4-1ubuntu1.4 [276 kB]\n",
            "Get:29 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-webencodings all 0.5-2 [10.3 kB]\n",
            "Get:30 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-html5lib all 0.999999999-1 [83.6 kB]\n",
            "Get:31 http://archive.ubuntu.com/ubuntu bionic/main amd64 python-openssl all 17.5.0-1ubuntu1 [41.3 kB]\n",
            "Get:32 http://archive.ubuntu.com/ubuntu bionic/universe amd64 python-sparqlwrapper all 1.7.6-2 [21.8 kB]\n",
            "Get:33 http://archive.ubuntu.com/ubuntu bionic/universe amd64 samtools amd64 1.7-1 [288 kB]\n",
            "Get:34 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libc-ares2 amd64 1.14.0-1ubuntu0.1 [37.5 kB]\n",
            "Get:35 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 nodejs amd64 8.10.0~dfsg-2ubuntu0.4 [4,796 kB]\n",
            "Fetched 8,568 kB in 2s (4,064 kB/s)\n",
            "debconf: unable to initialize frontend: Dialog\n",
            "debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 76, <> line 35.)\n",
            "debconf: falling back to frontend: Readline\n",
            "debconf: unable to initialize frontend: Readline\n",
            "debconf: (This frontend requires a controlling tty.)\n",
            "debconf: falling back to frontend: Teletype\n",
            "dpkg-preconfigure: unable to re-open stdin: \n",
            "Selecting previously unselected package python-pkg-resources.\n",
            "(Reading database ... 155308 files and directories currently installed.)\n",
            "Preparing to unpack .../00-python-pkg-resources_39.0.1-2_all.deb ...\n",
            "Unpacking python-pkg-resources (39.0.1-2) ...\n",
            "Selecting previously unselected package python-six.\n",
            "Preparing to unpack .../01-python-six_1.11.0-2_all.deb ...\n",
            "Unpacking python-six (1.11.0-2) ...\n",
            "Selecting previously unselected package python-isodate.\n",
            "Preparing to unpack .../02-python-isodate_0.6.0-1_all.deb ...\n",
            "Unpacking python-isodate (0.6.0-1) ...\n",
            "Selecting previously unselected package python-pyparsing.\n",
            "Preparing to unpack .../03-python-pyparsing_2.2.0+dfsg1-2_all.deb ...\n",
            "Unpacking python-pyparsing (2.2.0+dfsg1-2) ...\n",
            "Selecting previously unselected package python-rdflib.\n",
            "Preparing to unpack .../04-python-rdflib_4.2.1-2_all.deb ...\n",
            "Unpacking python-rdflib (4.2.1-2) ...\n",
            "Selecting previously unselected package python-certifi.\n",
            "Preparing to unpack .../05-python-certifi_2018.1.18-2_all.deb ...\n",
            "Unpacking python-certifi (2018.1.18-2) ...\n",
            "Selecting previously unselected package python-chardet.\n",
            "Preparing to unpack .../06-python-chardet_3.0.4-1_all.deb ...\n",
            "Unpacking python-chardet (3.0.4-1) ...\n",
            "Selecting previously unselected package python-idna.\n",
            "Preparing to unpack .../07-python-idna_2.6-1_all.deb ...\n",
            "Unpacking python-idna (2.6-1) ...\n",
            "Selecting previously unselected package python-urllib3.\n",
            "Preparing to unpack .../08-python-urllib3_1.22-1ubuntu0.18.04.2_all.deb ...\n",
            "Unpacking python-urllib3 (1.22-1ubuntu0.18.04.2) ...\n",
            "Selecting previously unselected package python-requests.\n",
            "Preparing to unpack .../09-python-requests_2.18.4-2ubuntu0.1_all.deb ...\n",
            "Unpacking python-requests (2.18.4-2ubuntu0.1) ...\n",
            "Selecting previously unselected package python-ruamel.yaml.\n",
            "Preparing to unpack .../10-python-ruamel.yaml_0.15.34-1_amd64.deb ...\n",
            "Unpacking python-ruamel.yaml (0.15.34-1) ...\n",
            "Selecting previously unselected package python-cachecontrol.\n",
            "Preparing to unpack .../11-python-cachecontrol_0.11.7-1_all.deb ...\n",
            "Unpacking python-cachecontrol (0.11.7-1) ...\n",
            "Selecting previously unselected package python-lockfile.\n",
            "Preparing to unpack .../12-python-lockfile_1%3a0.12.2-2_all.deb ...\n",
            "Unpacking python-lockfile (1:0.12.2-2) ...\n",
            "Selecting previously unselected package python-mistune.\n",
            "Preparing to unpack .../13-python-mistune_0.8.3-2_all.deb ...\n",
            "Unpacking python-mistune (0.8.3-2) ...\n",
            "Selecting previously unselected package python-rdflib-jsonld.\n",
            "Preparing to unpack .../14-python-rdflib-jsonld_0.4.0-3_all.deb ...\n",
            "Unpacking python-rdflib-jsonld (0.4.0-3) ...\n",
            "Selecting previously unselected package python-typing.\n",
            "Preparing to unpack .../15-python-typing_3.6.2-1_all.deb ...\n",
            "Unpacking python-typing (3.6.2-1) ...\n",
            "Selecting previously unselected package python-avro.\n",
            "Preparing to unpack .../16-python-avro_1.8.2+dfsg-1_all.deb ...\n",
            "Unpacking python-avro (1.8.2+dfsg-1) ...\n",
            "Selecting previously unselected package libjs-bootstrap.\n",
            "Preparing to unpack .../17-libjs-bootstrap_3.3.7+dfsg-2_all.deb ...\n",
            "Unpacking libjs-bootstrap (3.3.7+dfsg-2) ...\n",
            "Selecting previously unselected package python-schema-salad.\n",
            "Preparing to unpack .../18-python-schema-salad_2.6.20171201034858-3_all.deb ...\n",
            "Unpacking python-schema-salad (2.6.20171201034858-3) ...\n",
            "Selecting previously unselected package python-shellescape.\n",
            "Preparing to unpack .../19-python-shellescape_3.4.1-2_all.deb ...\n",
            "Unpacking python-shellescape (3.4.1-2) ...\n",
            "Selecting previously unselected package cwltool.\n",
            "Preparing to unpack .../20-cwltool_1.0.20180302231433-1_all.deb ...\n",
            "Unpacking cwltool (1.0.20180302231433-1) ...\n",
            "Selecting previously unselected package libhts2:amd64.\n",
            "Preparing to unpack .../21-libhts2_1.7-2_amd64.deb ...\n",
            "Unpacking libhts2:amd64 (1.7-2) ...\n",
            "Selecting previously unselected package nodejs-doc.\n",
            "Preparing to unpack .../22-nodejs-doc_8.10.0~dfsg-2ubuntu0.4_all.deb ...\n",
            "Unpacking nodejs-doc (8.10.0~dfsg-2ubuntu0.4) ...\n",
            "Selecting previously unselected package python-asn1crypto.\n",
            "Preparing to unpack .../23-python-asn1crypto_0.24.0-1_all.deb ...\n",
            "Unpacking python-asn1crypto (0.24.0-1) ...\n",
            "Selecting previously unselected package python-cffi-backend.\n",
            "Preparing to unpack .../24-python-cffi-backend_1.11.5-1_amd64.deb ...\n",
            "Unpacking python-cffi-backend (1.11.5-1) ...\n",
            "Selecting previously unselected package python-enum34.\n",
            "Preparing to unpack .../25-python-enum34_1.1.6-2_all.deb ...\n",
            "Unpacking python-enum34 (1.1.6-2) ...\n",
            "Selecting previously unselected package python-ipaddress.\n",
            "Preparing to unpack .../26-python-ipaddress_1.0.17-1_all.deb ...\n",
            "Unpacking python-ipaddress (1.0.17-1) ...\n",
            "Selecting previously unselected package python-cryptography.\n",
            "Preparing to unpack .../27-python-cryptography_2.1.4-1ubuntu1.4_amd64.deb ...\n",
            "Unpacking python-cryptography (2.1.4-1ubuntu1.4) ...\n",
            "Selecting previously unselected package python-webencodings.\n",
            "Preparing to unpack .../28-python-webencodings_0.5-2_all.deb ...\n",
            "Unpacking python-webencodings (0.5-2) ...\n",
            "Selecting previously unselected package python-html5lib.\n",
            "Preparing to unpack .../29-python-html5lib_0.999999999-1_all.deb ...\n",
            "Unpacking python-html5lib (0.999999999-1) ...\n",
            "Selecting previously unselected package python-openssl.\n",
            "Preparing to unpack .../30-python-openssl_17.5.0-1ubuntu1_all.deb ...\n",
            "Unpacking python-openssl (17.5.0-1ubuntu1) ...\n",
            "Selecting previously unselected package python-sparqlwrapper.\n",
            "Preparing to unpack .../31-python-sparqlwrapper_1.7.6-2_all.deb ...\n",
            "Unpacking python-sparqlwrapper (1.7.6-2) ...\n",
            "Selecting previously unselected package samtools.\n",
            "Preparing to unpack .../32-samtools_1.7-1_amd64.deb ...\n",
            "Unpacking samtools (1.7-1) ...\n",
            "Selecting previously unselected package libc-ares2:amd64.\n",
            "Preparing to unpack .../33-libc-ares2_1.14.0-1ubuntu0.1_amd64.deb ...\n",
            "Unpacking libc-ares2:amd64 (1.14.0-1ubuntu0.1) ...\n",
            "Selecting previously unselected package nodejs.\n",
            "Preparing to unpack .../34-nodejs_8.10.0~dfsg-2ubuntu0.4_amd64.deb ...\n",
            "Unpacking nodejs (8.10.0~dfsg-2ubuntu0.4) ...\n",
            "Setting up python-idna (2.6-1) ...\n",
            "Setting up nodejs-doc (8.10.0~dfsg-2ubuntu0.4) ...\n",
            "Setting up python-asn1crypto (0.24.0-1) ...\n",
            "Setting up python-mistune (0.8.3-2) ...\n",
            "Setting up python-certifi (2018.1.18-2) ...\n",
            "Setting up python-avro (1.8.2+dfsg-1) ...\n",
            "Setting up python-pkg-resources (39.0.1-2) ...\n",
            "Setting up python-ruamel.yaml (0.15.34-1) ...\n",
            "Setting up python-cffi-backend (1.11.5-1) ...\n",
            "Setting up python-six (1.11.0-2) ...\n",
            "Setting up python-isodate (0.6.0-1) ...\n",
            "Setting up python-shellescape (3.4.1-2) ...\n",
            "Setting up python-enum34 (1.1.6-2) ...\n",
            "Setting up libjs-bootstrap (3.3.7+dfsg-2) ...\n",
            "Setting up python-lockfile (1:0.12.2-2) ...\n",
            "Setting up python-ipaddress (1.0.17-1) ...\n",
            "Setting up libc-ares2:amd64 (1.14.0-1ubuntu0.1) ...\n",
            "Setting up libhts2:amd64 (1.7-2) ...\n",
            "Setting up python-webencodings (0.5-2) ...\n",
            "Setting up python-typing (3.6.2-1) ...\n",
            "Setting up python-pyparsing (2.2.0+dfsg1-2) ...\n",
            "Setting up python-urllib3 (1.22-1ubuntu0.18.04.2) ...\n",
            "Setting up python-chardet (3.0.4-1) ...\n",
            "Setting up nodejs (8.10.0~dfsg-2ubuntu0.4) ...\n",
            "update-alternatives: using /usr/bin/nodejs to provide /usr/bin/js (js) in auto mode\n",
            "Setting up python-rdflib (4.2.1-2) ...\n",
            "Setting up python-sparqlwrapper (1.7.6-2) ...\n",
            "Setting up python-rdflib-jsonld (0.4.0-3) ...\n",
            "Setting up python-cryptography (2.1.4-1ubuntu1.4) ...\n",
            "Setting up python-requests (2.18.4-2ubuntu0.1) ...\n",
            "Setting up python-cachecontrol (0.11.7-1) ...\n",
            "Setting up python-schema-salad (2.6.20171201034858-3) ...\n",
            "Setting up python-html5lib (0.999999999-1) ...\n",
            "Setting up samtools (1.7-1) ...\n",
            "Setting up python-openssl (17.5.0-1ubuntu1) ...\n",
            "Setting up cwltool (1.0.20180302231433-1) ...\n",
            "Processing triggers for libc-bin (2.27-3ubuntu1.3) ...\n",
            "/sbin/ldconfig.real: /usr/local/lib/python3.7/dist-packages/ideep4py/lib/libmkldnn.so.0 is not a symbolic link\n",
            "\n",
            "Processing triggers for man-db (2.8.3-2ubuntu0.1) ...\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "fDh0okkLkpAD",
        "outputId": "2dca1ef5-ef4b-4899-b7d8-47e037073b2b"
      },
      "source": [
        "#INSTALLING SALMON\n",
        "#to quantify your RNA-seq data.\n",
        "!sudo apt update\n",
        "!sudo apt install salmon"
      ],
      "execution_count": 7,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\u001b[33m\r0% [Working]\u001b[0m\r            \rIgn:1 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  InRelease\n",
            "\u001b[33m\r0% [Waiting for headers] [Waiting for headers] [Waiting for headers] [Waiting f\u001b[0m\r                                                                               \rHit:2 https://cloud.r-project.org/bin/linux/ubuntu bionic-cran40/ InRelease\n",
            "\u001b[33m\r0% [Waiting for headers] [Waiting for headers] [Waiting for headers] [Waiting f\u001b[0m\u001b[33m\r0% [2 InRelease gpgv 3,626 B] [Waiting for headers] [Waiting for headers] [Wait\u001b[0m\r                                                                               \rHit:3 http://ppa.launchpad.net/c2d4u.team/c2d4u4.0+/ubuntu bionic InRelease\n",
            "\u001b[33m\r0% [2 InRelease gpgv 3,626 B] [Waiting for headers] [Waiting for headers] [Conn\u001b[0m\r                                                                               \rIgn:4 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  InRelease\n",
            "\u001b[33m\r0% [2 InRelease gpgv 3,626 B] [Waiting for headers] [Waiting for headers] [Conn\u001b[0m\r                                                                               \rHit:5 http://archive.ubuntu.com/ubuntu bionic InRelease\n",
            "\u001b[33m\r0% [2 InRelease gpgv 3,626 B] [Waiting for headers] [Connecting to ppa.launchpa\u001b[0m\r                                                                               \rGet:6 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]\n",
            "Hit:7 https://developer.download.nvidia.com/compute/cuda/repos/ubuntu1804/x86_64  Release\n",
            "Hit:8 https://developer.download.nvidia.com/compute/machine-learning/repos/ubuntu1804/x86_64  Release\n",
            "Hit:9 http://archive.ubuntu.com/ubuntu bionic-updates InRelease\n",
            "Hit:10 http://ppa.launchpad.net/cran/libgit2/ubuntu bionic InRelease\n",
            "Hit:11 http://archive.ubuntu.com/ubuntu bionic-backports InRelease\n",
            "Hit:12 http://ppa.launchpad.net/deadsnakes/ppa/ubuntu bionic InRelease\n",
            "Hit:15 http://ppa.launchpad.net/graphics-drivers/ppa/ubuntu bionic InRelease\n",
            "Fetched 88.7 kB in 1s (70.7 kB/s)\n",
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "58 packages can be upgraded. Run 'apt list --upgradable' to see them.\n",
            "Reading package lists... Done\n",
            "Building dependency tree       \n",
            "Reading state information... Done\n",
            "The following additional packages will be installed:\n",
            "  fonts-font-awesome fonts-lato javascript-common libdivsufsort3 libjemalloc1\n",
            "  libjs-jquery libjs-modernizr libjs-sphinxdoc libjs-underscore\n",
            "  libstaden-read11 sphinx-rtd-theme-common\n",
            "Suggested packages:\n",
            "  apache2 | lighttpd | httpd\n",
            "The following NEW packages will be installed:\n",
            "  fonts-font-awesome fonts-lato javascript-common libdivsufsort3 libjemalloc1\n",
            "  libjs-jquery libjs-modernizr libjs-sphinxdoc libjs-underscore\n",
            "  libstaden-read11 salmon sphinx-rtd-theme-common\n",
            "0 upgraded, 12 newly installed, 0 to remove and 58 not upgraded.\n",
            "Need to get 5,636 kB of archives.\n",
            "After this operation, 22.2 MB of additional disk space will be used.\n",
            "Get:1 http://archive.ubuntu.com/ubuntu bionic/main amd64 fonts-lato all 2.0-2 [2,698 kB]\n",
            "Get:2 http://archive.ubuntu.com/ubuntu bionic/main amd64 fonts-font-awesome all 4.7.0~dfsg-3 [513 kB]\n",
            "Get:3 http://archive.ubuntu.com/ubuntu bionic/main amd64 javascript-common all 11 [6,066 B]\n",
            "Get:4 http://archive.ubuntu.com/ubuntu bionic/universe amd64 libdivsufsort3 amd64 2.0.1-3 [44.4 kB]\n",
            "Get:5 http://archive.ubuntu.com/ubuntu bionic/main amd64 libjs-jquery all 3.2.1-1 [152 kB]\n",
            "Get:6 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 libjs-underscore all 1.8.3~dfsg-1ubuntu0.1 [60.4 kB]\n",
            "Get:7 http://archive.ubuntu.com/ubuntu bionic/main amd64 libjs-sphinxdoc all 1.6.7-1ubuntu1 [85.6 kB]\n",
            "Get:8 http://archive.ubuntu.com/ubuntu bionic/universe amd64 libstaden-read11 amd64 1.14.9-4 [218 kB]\n",
            "Get:9 http://archive.ubuntu.com/ubuntu bionic/universe amd64 libjemalloc1 amd64 3.6.0-11 [82.4 kB]\n",
            "Get:10 http://archive.ubuntu.com/ubuntu bionic/main amd64 libjs-modernizr all 2.6.2+ds1-1ubuntu1 [46.7 kB]\n",
            "Get:11 http://archive.ubuntu.com/ubuntu bionic/main amd64 sphinx-rtd-theme-common all 0.2.4-1 [224 kB]\n",
            "Get:12 http://archive.ubuntu.com/ubuntu bionic/universe amd64 salmon amd64 0.7.2+ds1-3 [1,505 kB]\n",
            "Fetched 5,636 kB in 1s (4,708 kB/s)\n",
            "debconf: unable to initialize frontend: Dialog\n",
            "debconf: (No usable dialog-like program is installed, so the dialog based frontend cannot be used. at /usr/share/perl5/Debconf/FrontEnd/Dialog.pm line 76, <> line 12.)\n",
            "debconf: falling back to frontend: Readline\n",
            "debconf: unable to initialize frontend: Readline\n",
            "debconf: (This frontend requires a controlling tty.)\n",
            "debconf: falling back to frontend: Teletype\n",
            "dpkg-preconfigure: unable to re-open stdin: \n",
            "Selecting previously unselected package fonts-lato.\n",
            "(Reading database ... 156745 files and directories currently installed.)\n",
            "Preparing to unpack .../00-fonts-lato_2.0-2_all.deb ...\n",
            "Unpacking fonts-lato (2.0-2) ...\n",
            "Selecting previously unselected package fonts-font-awesome.\n",
            "Preparing to unpack .../01-fonts-font-awesome_4.7.0~dfsg-3_all.deb ...\n",
            "Unpacking fonts-font-awesome (4.7.0~dfsg-3) ...\n",
            "Selecting previously unselected package javascript-common.\n",
            "Preparing to unpack .../02-javascript-common_11_all.deb ...\n",
            "Unpacking javascript-common (11) ...\n",
            "Selecting previously unselected package libdivsufsort3:amd64.\n",
            "Preparing to unpack .../03-libdivsufsort3_2.0.1-3_amd64.deb ...\n",
            "Unpacking libdivsufsort3:amd64 (2.0.1-3) ...\n",
            "Selecting previously unselected package libjs-jquery.\n",
            "Preparing to unpack .../04-libjs-jquery_3.2.1-1_all.deb ...\n",
            "Unpacking libjs-jquery (3.2.1-1) ...\n",
            "Selecting previously unselected package libjs-underscore.\n",
            "Preparing to unpack .../05-libjs-underscore_1.8.3~dfsg-1ubuntu0.1_all.deb ...\n",
            "Unpacking libjs-underscore (1.8.3~dfsg-1ubuntu0.1) ...\n",
            "Selecting previously unselected package libjs-sphinxdoc.\n",
            "Preparing to unpack .../06-libjs-sphinxdoc_1.6.7-1ubuntu1_all.deb ...\n",
            "Unpacking libjs-sphinxdoc (1.6.7-1ubuntu1) ...\n",
            "Selecting previously unselected package libstaden-read11:amd64.\n",
            "Preparing to unpack .../07-libstaden-read11_1.14.9-4_amd64.deb ...\n",
            "Unpacking libstaden-read11:amd64 (1.14.9-4) ...\n",
            "Selecting previously unselected package libjemalloc1.\n",
            "Preparing to unpack .../08-libjemalloc1_3.6.0-11_amd64.deb ...\n",
            "Unpacking libjemalloc1 (3.6.0-11) ...\n",
            "Selecting previously unselected package libjs-modernizr.\n",
            "Preparing to unpack .../09-libjs-modernizr_2.6.2+ds1-1ubuntu1_all.deb ...\n",
            "Unpacking libjs-modernizr (2.6.2+ds1-1ubuntu1) ...\n",
            "Selecting previously unselected package sphinx-rtd-theme-common.\n",
            "Preparing to unpack .../10-sphinx-rtd-theme-common_0.2.4-1_all.deb ...\n",
            "Unpacking sphinx-rtd-theme-common (0.2.4-1) ...\n",
            "Selecting previously unselected package salmon.\n",
            "Preparing to unpack .../11-salmon_0.7.2+ds1-3_amd64.deb ...\n",
            "Unpacking salmon (0.7.2+ds1-3) ...\n",
            "Setting up libjs-jquery (3.2.1-1) ...\n",
            "Setting up libstaden-read11:amd64 (1.14.9-4) ...\n",
            "Setting up libdivsufsort3:amd64 (2.0.1-3) ...\n",
            "Setting up libjs-underscore (1.8.3~dfsg-1ubuntu0.1) ...\n",
            "Setting up libjemalloc1 (3.6.0-11) ...\n",
            "Setting up libjs-sphinxdoc (1.6.7-1ubuntu1) ...\n",
            "Setting up fonts-font-awesome (4.7.0~dfsg-3) ...\n",
            "Setting up fonts-lato (2.0-2) ...\n",
            "Setting up javascript-common (11) ...\n",
            "Setting up libjs-modernizr (2.6.2+ds1-1ubuntu1) ...\n",
            "Setting up sphinx-rtd-theme-common (0.2.4-1) ...\n",
            "Setting up salmon (0.7.2+ds1-3) ...\n",
            "Processing triggers for man-db (2.8.3-2ubuntu0.1) ...\n",
            "Processing triggers for fontconfig (2.12.6-0ubuntu2) ...\n",
            "Processing triggers for libc-bin (2.27-3ubuntu1.3) ...\n",
            "/sbin/ldconfig.real: /usr/local/lib/python3.7/dist-packages/ideep4py/lib/libmkldnn.so.0 is not a symbolic link\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "NrNMa57u7oxR",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "2b3054fd-bb75-48e2-cbd1-2cd2db6d8387"
      },
      "source": [
        "#INSTALLING TRINITY\n",
        "!wget -O trinityrnaseq-v2.13.2.FULL.tar.gz https://github.com/trinityrnaseq/trinityrnaseq/releases/download/Trinity-v2.13.2/trinityrnaseq-v2.13.2.FULL.tar.gz"
      ],
      "execution_count": 8,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2021-12-10 03:38:06--  https://github.com/trinityrnaseq/trinityrnaseq/releases/download/Trinity-v2.13.2/trinityrnaseq-v2.13.2.FULL.tar.gz\n",
            "Resolving github.com (github.com)... 140.82.112.4\n",
            "Connecting to github.com (github.com)|140.82.112.4|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 Found\n",
            "Location: https://objects.githubusercontent.com/github-production-release-asset-2e65be/22736952/e0365cbf-b08e-4681-bddf-9e32f4fc8d0f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20211210%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20211210T033806Z&X-Amz-Expires=300&X-Amz-Signature=3d94ecf5204952df3b1b0b172c03c0a90ca9aee0ef9d65e7c0d24fd9abc771ab&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=22736952&response-content-disposition=attachment%3B%20filename%3Dtrinityrnaseq-v2.13.2.FULL.tar.gz&response-content-type=application%2Foctet-stream [following]\n",
            "--2021-12-10 03:38:06--  https://objects.githubusercontent.com/github-production-release-asset-2e65be/22736952/e0365cbf-b08e-4681-bddf-9e32f4fc8d0f?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Credential=AKIAIWNJYAX4CSVEH53A%2F20211210%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Date=20211210T033806Z&X-Amz-Expires=300&X-Amz-Signature=3d94ecf5204952df3b1b0b172c03c0a90ca9aee0ef9d65e7c0d24fd9abc771ab&X-Amz-SignedHeaders=host&actor_id=0&key_id=0&repo_id=22736952&response-content-disposition=attachment%3B%20filename%3Dtrinityrnaseq-v2.13.2.FULL.tar.gz&response-content-type=application%2Foctet-stream\n",
            "Resolving objects.githubusercontent.com (objects.githubusercontent.com)... 185.199.109.133, 185.199.108.133, 185.199.110.133, ...\n",
            "Connecting to objects.githubusercontent.com (objects.githubusercontent.com)|185.199.109.133|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 32034013 (31M) [application/octet-stream]\n",
            "Saving to: ‘trinityrnaseq-v2.13.2.FULL.tar.gz’\n",
            "\n",
            "trinityrnaseq-v2.13 100%[===================>]  30.55M  79.0MB/s    in 0.4s    \n",
            "\n",
            "2021-12-10 03:38:06 (79.0 MB/s) - ‘trinityrnaseq-v2.13.2.FULL.tar.gz’ saved [32034013/32034013]\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!tar -xf trinityrnaseq-v2.13.2.FULL.tar.gz"
      ],
      "metadata": {
        "id": "6mqXSzc0tp33"
      },
      "execution_count": 9,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "%cd trinityrnaseq-v2.13.2"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "RT9pwv6Pt1VC",
        "outputId": "8ad29c76-5efc-44f1-f2df-8676b979500e"
      },
      "execution_count": 10,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/trinityrnaseq-v2.13.2\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!make\n",
        "!make plugins\n",
        "!make install"
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "hkKb0kUet-LU",
        "outputId": "b4c49fce-efdf-423f-e517-34b781535ff5"
      },
      "execution_count": 11,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "Using gnu compiler for Inchworm and Chrysalis\n",
            "cd Inchworm && make\n",
            "make[1]: Entering directory '/content/trinityrnaseq-v2.13.2/Inchworm'\n",
            "mkdir -p build\n",
            "cd build && cmake -DCMAKE_INSTALL_PREFIX=\"\" ../ && make DESTDIR=../ install\n",
            "-- The C compiler identification is GNU 7.5.0\n",
            "-- The CXX compiler identification is GNU 7.5.0\n",
            "-- Check for working C compiler: /usr/bin/gcc\n",
            "-- Check for working C compiler: /usr/bin/gcc -- works\n",
            "-- Detecting C compiler ABI info\n",
            "-- Detecting C compiler ABI info - done\n",
            "-- Detecting C compile features\n",
            "-- Detecting C compile features - done\n",
            "-- Check for working CXX compiler: /usr/bin/g++\n",
            "-- Check for working CXX compiler: /usr/bin/g++ -- works\n",
            "-- Detecting CXX compiler ABI info\n",
            "-- Detecting CXX compiler ABI info - done\n",
            "-- Detecting CXX compile features\n",
            "-- Detecting CXX compile features - done\n",
            "-- No build type specified so building in RELEASE mode.\n",
            "-- system: Linux\n",
            "-- Configuring done\n",
            "-- Generating done\n",
            "-- Build files have been written to: /content/trinityrnaseq-v2.13.2/Inchworm/build\n",
            "make[2]: Entering directory '/content/trinityrnaseq-v2.13.2/Inchworm/build'\n",
            "make[3]: Entering directory '/content/trinityrnaseq-v2.13.2/Inchworm/build'\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Inchworm/build'\n",
            "\u001b[35m\u001b[1mScanning dependencies of target fastaToKmerCoverageStats\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Inchworm/build'\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Inchworm/build'\n",
            "[  3%] \u001b[32mBuilding CXX object CMakeFiles/fastaToKmerCoverageStats.dir/src/fastaToKmerCoverageStats.cpp.o\u001b[0m\n",
            "In file included from \u001b[01m\u001b[K/usr/include/c++/7/ext/hash_map:60:0\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/KmerCounter.hpp:53\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/IRKE.hpp:7\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/fastaToKmerCoverageStats.cpp:20\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/c++/7/backward/backward_warning.h:32:2:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K#warning This file includes at least one deprecated or antiquated header which may be removed without further notice at a future date. Please use a non-deprecated interface with equivalent functionality instead. For a listing of replacement headers and interfaces, consult the file backward_warning.h. To disable this warning use -Wno-deprecated. [\u001b[01;35m\u001b[K-Wcpp\u001b[m\u001b[K]\n",
            " #\u001b[01;35m\u001b[Kwarning\u001b[m\u001b[K \\\n",
            "  \u001b[01;35m\u001b[K^~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/fastaToKmerCoverageStats.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid populate_kmer_counter_from_reads(KmerCounter&, std::__cxx11::string&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/fastaToKmerCoverageStats.cpp:231:18:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kkmer_length\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     unsigned int \u001b[01;35m\u001b[Kkmer_length\u001b[m\u001b[K = kcounter.get_kmer_length();\n",
            "                  \u001b[01;35m\u001b[K^~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/fastaToKmerCoverageStats.cpp:250:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[Kend\u001b[m\u001b[K’ may be used uninitialized in this function [\u001b[01;35m\u001b[K-Wmaybe-uninitialized\u001b[m\u001b[K]\n",
            " #pragma \u001b[01;35m\u001b[Komp\u001b[m\u001b[K parallel private (myTid)\n",
            "         \u001b[01;35m\u001b[K^~~\u001b[m\u001b[K\n",
            "[  7%] \u001b[32mBuilding CXX object CMakeFiles/fastaToKmerCoverageStats.dir/src/argProcessor.cpp.o\u001b[0m\n",
            "[ 10%] \u001b[32mBuilding CXX object CMakeFiles/fastaToKmerCoverageStats.dir/src/Fasta_reader.cpp.o\u001b[0m\n",
            "[ 14%] \u001b[32mBuilding CXX object CMakeFiles/fastaToKmerCoverageStats.dir/src/Fasta_entry.cpp.o\u001b[0m\n",
            "[ 17%] \u001b[32mBuilding CXX object CMakeFiles/fastaToKmerCoverageStats.dir/src/sequenceUtil.cpp.o\u001b[0m\n",
            "[ 21%] \u001b[32mBuilding CXX object CMakeFiles/fastaToKmerCoverageStats.dir/src/string_util.cpp.o\u001b[0m\n",
            "[ 25%] \u001b[32mBuilding CXX object CMakeFiles/fastaToKmerCoverageStats.dir/src/stacktrace.cpp.o\u001b[0m\n",
            "[ 28%] \u001b[32mBuilding CXX object CMakeFiles/fastaToKmerCoverageStats.dir/src/KmerCounter.cpp.o\u001b[0m\n",
            "In file included from \u001b[01m\u001b[K/usr/include/c++/7/ext/hash_map:60:0\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/KmerCounter.hpp:53\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/KmerCounter.cpp:1\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/c++/7/backward/backward_warning.h:32:2:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K#warning This file includes at least one deprecated or antiquated header which may be removed without further notice at a future date. Please use a non-deprecated interface with equivalent functionality instead. For a listing of replacement headers and interfaces, consult the file backward_warning.h. To disable this warning use -Wno-deprecated. [\u001b[01;35m\u001b[K-Wcpp\u001b[m\u001b[K]\n",
            " #\u001b[01;35m\u001b[Kwarning\u001b[m\u001b[K \\\n",
            "  \u001b[01;35m\u001b[K^~~~~~~\u001b[m\u001b[K\n",
            "[ 32%] \u001b[32m\u001b[1mLinking CXX executable fastaToKmerCoverageStats\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Inchworm/build'\n",
            "[ 32%] Built target fastaToKmerCoverageStats\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Inchworm/build'\n",
            "\u001b[35m\u001b[1mScanning dependencies of target FastaToDeBruijn\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Inchworm/build'\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Inchworm/build'\n",
            "[ 35%] \u001b[32mBuilding CXX object CMakeFiles/FastaToDeBruijn.dir/src/FastaToDeBruijn.cpp.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/FastaToDeBruijn.cpp:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid createGraphPerRecord(std::vector<std::__cxx11::basic_string<char> >, int, bool, ArgProcessor)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/FastaToDeBruijn.cpp:215:39:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "                 if (\u001b[01;35m\u001b[Kseq_region.size() < kmer_length\u001b[m\u001b[K) { continue; }  // can be encountered in jaccard-clip mode (rarely)\n",
            "                     \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "[ 39%] \u001b[32mBuilding CXX object CMakeFiles/FastaToDeBruijn.dir/src/argProcessor.cpp.o\u001b[0m\n",
            "[ 42%] \u001b[32mBuilding CXX object CMakeFiles/FastaToDeBruijn.dir/src/Fasta_reader.cpp.o\u001b[0m\n",
            "[ 46%] \u001b[32mBuilding CXX object CMakeFiles/FastaToDeBruijn.dir/src/Fasta_entry.cpp.o\u001b[0m\n",
            "[ 50%] \u001b[32mBuilding CXX object CMakeFiles/FastaToDeBruijn.dir/src/sequenceUtil.cpp.o\u001b[0m\n",
            "[ 53%] \u001b[32mBuilding CXX object CMakeFiles/FastaToDeBruijn.dir/src/string_util.cpp.o\u001b[0m\n",
            "[ 57%] \u001b[32mBuilding CXX object CMakeFiles/FastaToDeBruijn.dir/src/stacktrace.cpp.o\u001b[0m\n",
            "[ 60%] \u001b[32mBuilding CXX object CMakeFiles/FastaToDeBruijn.dir/src/DeBruijnGraph.cpp.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/DeBruijnGraph.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kstd::__cxx11::string DeBruijnKmer::get_annotations_string()\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/DeBruijnGraph.cpp:66:21:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "     for (int i=0; \u001b[01;35m\u001b[Ki < _annotations.size()\u001b[m\u001b[K; i++) {\n",
            "                   \u001b[01;35m\u001b[K~~^~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/DeBruijnGraph.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kstd::__cxx11::string DeBruijnGraph::toDOT(bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/DeBruijnGraph.cpp:445:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         for (int i=0; \u001b[01;35m\u001b[Ki < prev_kmers.size()\u001b[m\u001b[K; i++) {\n",
            "                       \u001b[01;35m\u001b[K~~^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/DeBruijnGraph.cpp:494:25:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         for (int i=0; \u001b[01;35m\u001b[Ki < next_kmers.size()\u001b[m\u001b[K; i++) {\n",
            "                       \u001b[01;35m\u001b[K~~^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/DeBruijnGraph.cpp:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kstd::__cxx11::string DeBruijnGraph::toChrysalisFormat(int, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/DeBruijnGraph.cpp:725:35:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "                 for (int i = 0; \u001b[01;35m\u001b[Ki < collected_kmers.size()\u001b[m\u001b[K; i++) {\n",
            "                                 \u001b[01;35m\u001b[K~~^~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "[ 64%] \u001b[32m\u001b[1mLinking CXX executable FastaToDeBruijn\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Inchworm/build'\n",
            "[ 64%] Built target FastaToDeBruijn\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Inchworm/build'\n",
            "\u001b[35m\u001b[1mScanning dependencies of target inchworm\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Inchworm/build'\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Inchworm/build'\n",
            "[ 67%] \u001b[32mBuilding CXX object CMakeFiles/inchworm.dir/src/Fasta_entry.cpp.o\u001b[0m\n",
            "[ 71%] \u001b[32mBuilding CXX object CMakeFiles/inchworm.dir/src/IRKE_run.cpp.o\u001b[0m\n",
            "In file included from \u001b[01m\u001b[K/usr/include/c++/7/ext/hash_map:60:0\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/KmerCounter.hpp:53\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/IRKE.hpp:7\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/IRKE_run.cpp:11\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/c++/7/backward/backward_warning.h:32:2:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K#warning This file includes at least one deprecated or antiquated header which may be removed without further notice at a future date. Please use a non-deprecated interface with equivalent functionality instead. For a listing of replacement headers and interfaces, consult the file backward_warning.h. To disable this warning use -Wno-deprecated. [\u001b[01;35m\u001b[K-Wcpp\u001b[m\u001b[K]\n",
            " #\u001b[01;35m\u001b[Kwarning\u001b[m\u001b[K \\\n",
            "  \u001b[01;35m\u001b[K^~~~~~~\u001b[m\u001b[K\n",
            "[ 75%] \u001b[32mBuilding CXX object CMakeFiles/inchworm.dir/src/sequenceUtil.cpp.o\u001b[0m\n",
            "[ 78%] \u001b[32mBuilding CXX object CMakeFiles/inchworm.dir/src/IRKE.cpp.o\u001b[0m\n",
            "In file included from \u001b[01m\u001b[K/usr/include/c++/7/ext/hash_map:60:0\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/KmerCounter.hpp:53\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/IRKE.hpp:7\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/IRKE.cpp:21\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/c++/7/backward/backward_warning.h:32:2:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K#warning This file includes at least one deprecated or antiquated header which may be removed without further notice at a future date. Please use a non-deprecated interface with equivalent functionality instead. For a listing of replacement headers and interfaces, consult the file backward_warning.h. To disable this warning use -Wno-deprecated. [\u001b[01;35m\u001b[K-Wcpp\u001b[m\u001b[K]\n",
            " #\u001b[01;35m\u001b[Kwarning\u001b[m\u001b[K \\\n",
            "  \u001b[01;35m\u001b[K^~~~~~~\u001b[m\u001b[K\n",
            "[ 82%] \u001b[32mBuilding CXX object CMakeFiles/inchworm.dir/src/KmerCounter.cpp.o\u001b[0m\n",
            "In file included from \u001b[01m\u001b[K/usr/include/c++/7/ext/hash_map:60:0\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/KmerCounter.hpp:53\u001b[m\u001b[K,\n",
            "                 from \u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Inchworm/src/KmerCounter.cpp:1\u001b[m\u001b[K:\n",
            "\u001b[01m\u001b[K/usr/include/c++/7/backward/backward_warning.h:32:2:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K#warning This file includes at least one deprecated or antiquated header which may be removed without further notice at a future date. Please use a non-deprecated interface with equivalent functionality instead. For a listing of replacement headers and interfaces, consult the file backward_warning.h. To disable this warning use -Wno-deprecated. [\u001b[01;35m\u001b[K-Wcpp\u001b[m\u001b[K]\n",
            " #\u001b[01;35m\u001b[Kwarning\u001b[m\u001b[K \\\n",
            "  \u001b[01;35m\u001b[K^~~~~~~\u001b[m\u001b[K\n",
            "[ 85%] \u001b[32mBuilding CXX object CMakeFiles/inchworm.dir/src/string_util.cpp.o\u001b[0m\n",
            "[ 89%] \u001b[32mBuilding CXX object CMakeFiles/inchworm.dir/src/Fasta_reader.cpp.o\u001b[0m\n",
            "[ 92%] \u001b[32mBuilding CXX object CMakeFiles/inchworm.dir/src/stacktrace.cpp.o\u001b[0m\n",
            "[ 96%] \u001b[32mBuilding CXX object CMakeFiles/inchworm.dir/src/argProcessor.cpp.o\u001b[0m\n",
            "[100%] \u001b[32m\u001b[1mLinking CXX executable inchworm\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Inchworm/build'\n",
            "[100%] Built target inchworm\n",
            "make[3]: Leaving directory '/content/trinityrnaseq-v2.13.2/Inchworm/build'\n",
            "\u001b[36mInstall the project...\u001b[0m\n",
            "-- Install configuration: \"Release\"\n",
            "-- Installing: ../bin/inchworm\n",
            "-- Installing: ../bin/FastaToDeBruijn\n",
            "-- Installing: ../bin/fastaToKmerCoverageStats\n",
            "make[2]: Leaving directory '/content/trinityrnaseq-v2.13.2/Inchworm/build'\n",
            "make[1]: Leaving directory '/content/trinityrnaseq-v2.13.2/Inchworm'\n",
            "cd Chrysalis && make\n",
            "make[1]: Entering directory '/content/trinityrnaseq-v2.13.2/Chrysalis'\n",
            "mkdir -p build\n",
            "cd build && cmake -DCMAKE_INSTALL_PREFIX=\"\" ../ && make DESTDIR=../ install\n",
            "-- The C compiler identification is GNU 7.5.0\n",
            "-- The CXX compiler identification is GNU 7.5.0\n",
            "-- Check for working C compiler: /usr/bin/gcc\n",
            "-- Check for working C compiler: /usr/bin/gcc -- works\n",
            "-- Detecting C compiler ABI info\n",
            "-- Detecting C compiler ABI info - done\n",
            "-- Detecting C compile features\n",
            "-- Detecting C compile features - done\n",
            "-- Check for working CXX compiler: /usr/bin/g++\n",
            "-- Check for working CXX compiler: /usr/bin/g++ -- works\n",
            "-- Detecting CXX compiler ABI info\n",
            "-- Detecting CXX compiler ABI info - done\n",
            "-- Detecting CXX compile features\n",
            "-- Detecting CXX compile features - done\n",
            "-- No build type specified so building in RELEASE mode.\n",
            "-- system: Linux\n",
            "-- Configuring done\n",
            "-- Generating done\n",
            "-- Build files have been written to: /content/trinityrnaseq-v2.13.2/Chrysalis/build\n",
            "make[2]: Entering directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "make[3]: Entering directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "\u001b[35m\u001b[1mScanning dependencies of target GraphFromFasta\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "[  1%] \u001b[32mBuilding CXX object CMakeFiles/GraphFromFasta.dir/aligns/KmerAlignCore.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:2:0:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K\"NDEBUG\" redefined\n",
            " #define NDEBUG\n",
            " \n",
            "\u001b[01m\u001b[K<command-line>:0:0:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kthis is the location of the previous definition\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerAlignCore::AddData(const vecDNAVector&, const vecNumVector&, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:79:49:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kt\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "                 KmerAlignCoreRecordStoreTable & \u001b[01;35m\u001b[Kt\u001b[m\u001b[K = m_table; // not used?\n",
            "                                                 \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:49:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j, k;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerAlignCore::AddData(const DNAVector&, int, int, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:128:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j, k;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:128:12:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kj\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int i, \u001b[01;35m\u001b[Kj\u001b[m\u001b[K, k;\n",
            "            \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerAlignCore::SortAll()\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:148:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kconst svec<KmerAlignCoreRecord>& KmerAlignCore::GetMatchesDirectly(const DNAVector&, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:158:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ksize\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ksize\u001b[m\u001b[K = m_pTrans->GetSize();\n",
            "         \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:159:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:159:12:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kj\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int i, \u001b[01;35m\u001b[Kj\u001b[m\u001b[K;\n",
            "            \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerAlignCore::MergeSortFilter(svec<KmerAlignCoreRecord>&, const svec<KmerAlignCoreRecord>&, const svec<KmerAlignCoreRecord>&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:291:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[  2%] \u001b[32mBuilding CXX object CMakeFiles/GraphFromFasta.dir/analysis/AACodons.cc.o\u001b[0m\n",
            "[  4%] \u001b[32mBuilding CXX object CMakeFiles/GraphFromFasta.dir/analysis/DNAVector.cc.o\u001b[0m\n",
            "[  5%] \u001b[32mBuilding CXX object CMakeFiles/GraphFromFasta.dir/analysis/DeBruijnGraph.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid DeBruijnKmer::add_next_kmer(kmer_int_type_t, unsigned int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:154:66:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kkmer_length\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void DeBruijnKmer::add_next_kmer(kmer_int_type_t k, unsigned int \u001b[01;35m\u001b[Kkmer_length\u001b[m\u001b[K) {\n",
            "                                                                  \u001b[01;35m\u001b[K^~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kstd::vector<DeBruijnKmer> DeBruijnGraph::deconvolute_DS_mirror_graph()\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:355:22:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[Krdk\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K-Wunused-but-set-variable\u001b[m\u001b[K]\n",
            "         DeBruijnKmer \u001b[01;35m\u001b[Krdk\u001b[m\u001b[K = _kmer_map.find(rk)->second;\n",
            "                      \u001b[01;35m\u001b[K^~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kstd::vector<std::__cxx11::basic_string<char> > DeBruijnGraph::get_candidate_weldmers(kmer_int_type_t, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:473:23:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "     for (int i = 0; \u001b[01;35m\u001b[Ki < left_extensions.size()\u001b[m\u001b[K; i++) {\n",
            "                     \u001b[01;35m\u001b[K~~^~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:476:27:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         for (int j = 0; \u001b[01;35m\u001b[Kj < right_extensions.size()\u001b[m\u001b[K; j++) {\n",
            "                         \u001b[01;35m\u001b[K~~^~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid DeBruijnGraph::recursively_construct_kmer_extensions(kmer_int_type_t, std::vector<char>&, std::vector<std::__cxx11::basic_string<char> >&, char, std::map<long long unsigned int, bool>&, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:513:23:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "     for (int i = 0; \u001b[01;35m\u001b[Ki < adjacent_kmers.size()\u001b[m\u001b[K; i++) {\n",
            "                     \u001b[01;35m\u001b[K~~^~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:527:41:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         if (\u001b[01;35m\u001b[Kkmer_extension_chars.size() == flank_extension_length\u001b[m\u001b[K) {\n",
            "             \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "[  7%] \u001b[32mBuilding CXX object CMakeFiles/GraphFromFasta.dir/analysis/GraphFromFasta.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/GraphFromFasta.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kbool SimpleHalves(const DNAVector&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/GraphFromFasta.cc:272:15:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Ksuggest parentheses around ‘\u001b[01m\u001b[K&&\u001b[m\u001b[K’ within ‘\u001b[01m\u001b[K||\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wparentheses\u001b[m\u001b[K]\n",
            "             ( \u001b[01;35m\u001b[K(! DISABLE_REPEAT_CHECK)\u001b[m\u001b[K\n",
            "               \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            " \u001b[01;35m\u001b[K              &&\u001b[m\u001b[K\n",
            "               \u001b[01;35m\u001b[K^~\u001b[m\u001b[K\n",
            " \u001b[01;35m\u001b[K              is_simple_repeat(left)\u001b[m\u001b[K || is_simple_repeat(right) )\n",
            "               \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/GraphFromFasta.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kfloat align_get_per_id(const DNAVector&, const DNAVector&, int, int, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/GraphFromFasta.cc:351:94:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kk\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " float align_get_per_id(const DNAVector & a, const DNAVector & b, int startA, int startB, int \u001b[01;35m\u001b[Kk\u001b[m\u001b[K)\n",
            "                                                                                              \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/GraphFromFasta.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kbool Welder::Weldable(const DNAVector&, int, const DNAVector&, int, int, std::__cxx11::string&, unsigned int&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/GraphFromFasta.cc:523:13:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "         int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K;\n",
            "             \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/GraphFromFasta.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid report_iworm_graph(std::map<int, Pool>&, std::map<int, bool>&, std::map<int, int>&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/GraphFromFasta.cc:713:31:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "             for (int j = 0; \u001b[01;35m\u001b[Kj < adjacent_nodes.size()\u001b[m\u001b[K; j++) {\n",
            "                             \u001b[01;35m\u001b[K~~^~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/GraphFromFasta.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid add_scaffolds_to_clusters(std::map<int, Pool>&, std::__cxx11::string, vecDNAVector&, int, float)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/GraphFromFasta.cc:961:33:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "             if (\u001b[01;35m\u001b[Kpair_link_count >= minCov\u001b[m\u001b[K) {\n",
            "                 \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/GraphFromFasta.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/GraphFromFasta.cc:1310:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         for (i=0; \u001b[01;35m\u001b[Ki<dna.size()\u001b[m\u001b[K; i++) {\n",
            "                   \u001b[01;35m\u001b[K~^~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/GraphFromFasta.cc:1316:60:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "             if (iworm_counter % 1000 == 0 || \u001b[01;35m\u001b[Kiworm_counter == dna.size()-1\u001b[m\u001b[K) {\n",
            "                                              \u001b[01;35m\u001b[K~~~~~~~~~~~~~~^~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/GraphFromFasta.cc:1442:16:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "     for (i=0; \u001b[01;35m\u001b[Ki<dna.size()\u001b[m\u001b[K; i++) {\n",
            "               \u001b[01;35m\u001b[K~^~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/GraphFromFasta.cc:1454:13:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kcutoff\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "         int \u001b[01;35m\u001b[Kcutoff\u001b[m\u001b[K = 0;\n",
            "             \u001b[01;35m\u001b[K^~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/GraphFromFasta.cc:1268:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ktotal_iworm_contigs\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ktotal_iworm_contigs\u001b[m\u001b[K = dna.size();\n",
            "         \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "[  8%] \u001b[32mBuilding CXX object CMakeFiles/GraphFromFasta.dir/analysis/KmerTable.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/KmerTable.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Klong long int KmerSequence::BasesToNumber(const DNAVector&, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/KmerTable.cc:27:13:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "   long long \u001b[01;35m\u001b[Ki\u001b[m\u001b[K;\n",
            "             \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 10%] \u001b[32mBuilding CXX object CMakeFiles/GraphFromFasta.dir/analysis/NonRedKmerTable.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid NonRedKmerTable::SetUp(const vecDNAVector&, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:46:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         for (j=0; \u001b[01;35m\u001b[Kj<=d.size()-m_k\u001b[m\u001b[K; j++) {\n",
            "                   \u001b[01;35m\u001b[K~^~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:56:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         for (j=0; \u001b[01;35m\u001b[Kj<=d.size()-m_k\u001b[m\u001b[K; j++) {\n",
            "                   \u001b[01;35m\u001b[K~^~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:79:18:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "       for (j=0; \u001b[01;35m\u001b[Kj<=d.size()-m_k\u001b[m\u001b[K; j++) {\n",
            "                 \u001b[01;35m\u001b[K~^~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid NonRedKmerTable::AddData(const vecDNAVector&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:109:16:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "     for (j=0; \u001b[01;35m\u001b[Kj<= d.isize()-m_k\u001b[m\u001b[K; j++) {\n",
            "               \u001b[01;35m\u001b[K~^~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid NonRedKmerTable::AddData(vecDNAVectorStream&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:125:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 11%] \u001b[32mBuilding CXX object CMakeFiles/GraphFromFasta.dir/analysis/sequenceUtil.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/sequenceUtil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kbool contains_non_gatc(std::__cxx11::string)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/sequenceUtil.cc:33:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Karray subscript has type ‘\u001b[01m\u001b[Kchar\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wchar-subscripts\u001b[m\u001b[K]\n",
            "  if (_base_to_int[c\u001b[01;35m\u001b[K]\u001b[m\u001b[K > 3)\n",
            "                    \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/sequenceUtil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kkmer_int_type_t kmer_to_intval(std::__cxx11::string)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/sequenceUtil.cc:264:26:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Karray subscript has type ‘\u001b[01m\u001b[Kchar\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wchar-subscripts\u001b[m\u001b[K]\n",
            "  int val = _base_to_int[c\u001b[01;35m\u001b[K]\u001b[m\u001b[K;\n",
            "                          \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 12%] \u001b[32mBuilding CXX object CMakeFiles/GraphFromFasta.dir/analysis/stacktrace.cc.o\u001b[0m\n",
            "[ 14%] \u001b[32mBuilding CXX object CMakeFiles/GraphFromFasta.dir/base/ErrorHandling.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid print_trace(FILE*, const char*, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:24:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kout\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *\u001b[01;35m\u001b[Kout\u001b[m\u001b[K, const char *file, int line)\n",
            "                        \u001b[01;35m\u001b[K^~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:41:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kfile\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *out, const char *\u001b[01;35m\u001b[Kfile\u001b[m\u001b[K, int line)\n",
            "                                         \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:51:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kline\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *out, const char *file, int \u001b[01;35m\u001b[Kline\u001b[m\u001b[K)\n",
            "                                                   \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "[ 15%] \u001b[32mBuilding CXX object CMakeFiles/GraphFromFasta.dir/base/FileParser.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/FileParser.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kbool StringParser::IsString(int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/FileParser.cc:50:33:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kindex\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool StringParser::IsString(int \u001b[01;35m\u001b[Kindex\u001b[m\u001b[K)\n",
            "                                 \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "[ 17%] \u001b[32mBuilding CXX object CMakeFiles/GraphFromFasta.dir/base/StringUtil.cc.o\u001b[0m\n",
            "[ 18%] \u001b[32mBuilding CXX object CMakeFiles/GraphFromFasta.dir/util/mutil.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:2:0:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K\"NDEBUG\" redefined\n",
            " #define NDEBUG\n",
            " \n",
            "\u001b[01m\u001b[K<command-line>:0:0:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kthis is the location of the previous definition\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvirtual bool CMAsciiReadFileStream::ReadSimpleType(void*, long int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:303:51:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KpData\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiReadFileStream::ReadSimpleType(void * \u001b[01;35m\u001b[KpData\u001b[m\u001b[K, long lenInBytes)\n",
            "                                                   \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:303:63:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KlenInBytes\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiReadFileStream::ReadSimpleType(void * pData, long \u001b[01;35m\u001b[KlenInBytes\u001b[m\u001b[K)\n",
            "                                                               \u001b[01;35m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvirtual bool CMAsciiWriteFileStream::WriteBlob(const void*, long int, long int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:53:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KpData\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * \u001b[01;35m\u001b[KpData\u001b[m\u001b[K, long lenInElements, long elSize)\n",
            "                                                     \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:65:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KlenInElements\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * pData, long \u001b[01;35m\u001b[KlenInElements\u001b[m\u001b[K, long elSize)\n",
            "                                                                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:85:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KelSize\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * pData, long lenInElements, long \u001b[01;35m\u001b[KelSize\u001b[m\u001b[K)\n",
            "                                                                                     \u001b[01;35m\u001b[K^~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:738:53:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                     \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, long int, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:761:63:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, long val, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                               \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, const MCL_TCHAR*, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:787:80:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, const MCL_TCHAR * szText2, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                                                \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MCLSetUTF8Encode(bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:1288:39:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kb\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MCLSetUTF8Encode(bool \u001b[01;35m\u001b[Kb\u001b[m\u001b[K)\n",
            "                                       \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kbool AddUTF8Sig(CMString&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:1688:16:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kp\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "   const char * \u001b[01;35m\u001b[Kp\u001b[m\u001b[K = (const char*)string;\n",
            "                \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 20%] \u001b[32m\u001b[1mLinking CXX executable GraphFromFasta\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "[ 20%] Built target GraphFromFasta\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "\u001b[35m\u001b[1mScanning dependencies of target ReadsToTranscripts\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "[ 21%] \u001b[32mBuilding CXX object CMakeFiles/ReadsToTranscripts.dir/analysis/AACodons.cc.o\u001b[0m\n",
            "[ 22%] \u001b[32mBuilding CXX object CMakeFiles/ReadsToTranscripts.dir/analysis/DNAVector.cc.o\u001b[0m\n",
            "[ 24%] \u001b[32mBuilding CXX object CMakeFiles/ReadsToTranscripts.dir/analysis/NonRedKmerTable.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid NonRedKmerTable::SetUp(const vecDNAVector&, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:46:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         for (j=0; \u001b[01;35m\u001b[Kj<=d.size()-m_k\u001b[m\u001b[K; j++) {\n",
            "                   \u001b[01;35m\u001b[K~^~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:56:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         for (j=0; \u001b[01;35m\u001b[Kj<=d.size()-m_k\u001b[m\u001b[K; j++) {\n",
            "                   \u001b[01;35m\u001b[K~^~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:79:18:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "       for (j=0; \u001b[01;35m\u001b[Kj<=d.size()-m_k\u001b[m\u001b[K; j++) {\n",
            "                 \u001b[01;35m\u001b[K~^~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid NonRedKmerTable::AddData(const vecDNAVector&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:109:16:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "     for (j=0; \u001b[01;35m\u001b[Kj<= d.isize()-m_k\u001b[m\u001b[K; j++) {\n",
            "               \u001b[01;35m\u001b[K~^~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid NonRedKmerTable::AddData(vecDNAVectorStream&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:125:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 25%] \u001b[32mBuilding CXX object CMakeFiles/ReadsToTranscripts.dir/analysis/ReadsToTranscripts.cc.o\u001b[0m\n",
            "[ 27%] \u001b[32mBuilding CXX object CMakeFiles/ReadsToTranscripts.dir/analysis/sequenceUtil.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/sequenceUtil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kbool contains_non_gatc(std::__cxx11::string)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/sequenceUtil.cc:33:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Karray subscript has type ‘\u001b[01m\u001b[Kchar\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wchar-subscripts\u001b[m\u001b[K]\n",
            "  if (_base_to_int[c\u001b[01;35m\u001b[K]\u001b[m\u001b[K > 3)\n",
            "                    \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/sequenceUtil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kkmer_int_type_t kmer_to_intval(std::__cxx11::string)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/sequenceUtil.cc:264:26:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Karray subscript has type ‘\u001b[01m\u001b[Kchar\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wchar-subscripts\u001b[m\u001b[K]\n",
            "  int val = _base_to_int[c\u001b[01;35m\u001b[K]\u001b[m\u001b[K;\n",
            "                          \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 28%] \u001b[32mBuilding CXX object CMakeFiles/ReadsToTranscripts.dir/analysis/stacktrace.cc.o\u001b[0m\n",
            "[ 30%] \u001b[32mBuilding CXX object CMakeFiles/ReadsToTranscripts.dir/base/ErrorHandling.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid print_trace(FILE*, const char*, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:24:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kout\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *\u001b[01;35m\u001b[Kout\u001b[m\u001b[K, const char *file, int line)\n",
            "                        \u001b[01;35m\u001b[K^~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:41:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kfile\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *out, const char *\u001b[01;35m\u001b[Kfile\u001b[m\u001b[K, int line)\n",
            "                                         \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:51:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kline\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *out, const char *file, int \u001b[01;35m\u001b[Kline\u001b[m\u001b[K)\n",
            "                                                   \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "[ 31%] \u001b[32mBuilding CXX object CMakeFiles/ReadsToTranscripts.dir/base/FileParser.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/FileParser.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kbool StringParser::IsString(int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/FileParser.cc:50:33:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kindex\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool StringParser::IsString(int \u001b[01;35m\u001b[Kindex\u001b[m\u001b[K)\n",
            "                                 \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "[ 32%] \u001b[32mBuilding CXX object CMakeFiles/ReadsToTranscripts.dir/base/StringUtil.cc.o\u001b[0m\n",
            "[ 34%] \u001b[32mBuilding CXX object CMakeFiles/ReadsToTranscripts.dir/util/mutil.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:2:0:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K\"NDEBUG\" redefined\n",
            " #define NDEBUG\n",
            " \n",
            "\u001b[01m\u001b[K<command-line>:0:0:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kthis is the location of the previous definition\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvirtual bool CMAsciiReadFileStream::ReadSimpleType(void*, long int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:303:51:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KpData\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiReadFileStream::ReadSimpleType(void * \u001b[01;35m\u001b[KpData\u001b[m\u001b[K, long lenInBytes)\n",
            "                                                   \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:303:63:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KlenInBytes\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiReadFileStream::ReadSimpleType(void * pData, long \u001b[01;35m\u001b[KlenInBytes\u001b[m\u001b[K)\n",
            "                                                               \u001b[01;35m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvirtual bool CMAsciiWriteFileStream::WriteBlob(const void*, long int, long int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:53:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KpData\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * \u001b[01;35m\u001b[KpData\u001b[m\u001b[K, long lenInElements, long elSize)\n",
            "                                                     \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:65:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KlenInElements\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * pData, long \u001b[01;35m\u001b[KlenInElements\u001b[m\u001b[K, long elSize)\n",
            "                                                                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:85:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KelSize\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * pData, long lenInElements, long \u001b[01;35m\u001b[KelSize\u001b[m\u001b[K)\n",
            "                                                                                     \u001b[01;35m\u001b[K^~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:738:53:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                     \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, long int, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:761:63:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, long val, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                               \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, const MCL_TCHAR*, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:787:80:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, const MCL_TCHAR * szText2, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                                                \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MCLSetUTF8Encode(bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:1288:39:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kb\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MCLSetUTF8Encode(bool \u001b[01;35m\u001b[Kb\u001b[m\u001b[K)\n",
            "                                       \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kbool AddUTF8Sig(CMString&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:1688:16:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kp\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "   const char * \u001b[01;35m\u001b[Kp\u001b[m\u001b[K = (const char*)string;\n",
            "                \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 35%] \u001b[32m\u001b[1mLinking CXX executable ReadsToTranscripts\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "[ 35%] Built target ReadsToTranscripts\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "\u001b[35m\u001b[1mScanning dependencies of target CreateIwormFastaBundle\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "[ 37%] \u001b[32mBuilding CXX object CMakeFiles/CreateIwormFastaBundle.dir/analysis/AACodons.cc.o\u001b[0m\n",
            "[ 38%] \u001b[32mBuilding CXX object CMakeFiles/CreateIwormFastaBundle.dir/analysis/CreateIwormFastaBundle.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/CreateIwormFastaBundle.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/CreateIwormFastaBundle.cc:181:17:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Knum_iworm_contigs\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "             int \u001b[01;35m\u001b[Knum_iworm_contigs\u001b[m\u001b[K = parser.AsInt(2);\n",
            "                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/CreateIwormFastaBundle.cc:143:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[KDEBUG\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     bool \u001b[01;35m\u001b[KDEBUG\u001b[m\u001b[K = P.GetBoolValueFor(debugCmmd);\n",
            "          \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/CreateIwormFastaBundle.cc:166:12:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[KpOut\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     FILE * \u001b[01;35m\u001b[KpOut\u001b[m\u001b[K = NULL;\n",
            "            \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "[ 40%] \u001b[32mBuilding CXX object CMakeFiles/CreateIwormFastaBundle.dir/analysis/DNAVector.cc.o\u001b[0m\n",
            "[ 41%] \u001b[32mBuilding CXX object CMakeFiles/CreateIwormFastaBundle.dir/base/ErrorHandling.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid print_trace(FILE*, const char*, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:24:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kout\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *\u001b[01;35m\u001b[Kout\u001b[m\u001b[K, const char *file, int line)\n",
            "                        \u001b[01;35m\u001b[K^~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:41:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kfile\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *out, const char *\u001b[01;35m\u001b[Kfile\u001b[m\u001b[K, int line)\n",
            "                                         \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:51:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kline\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *out, const char *file, int \u001b[01;35m\u001b[Kline\u001b[m\u001b[K)\n",
            "                                                   \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "[ 42%] \u001b[32mBuilding CXX object CMakeFiles/CreateIwormFastaBundle.dir/base/FileParser.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/FileParser.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kbool StringParser::IsString(int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/FileParser.cc:50:33:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kindex\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool StringParser::IsString(int \u001b[01;35m\u001b[Kindex\u001b[m\u001b[K)\n",
            "                                 \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "[ 44%] \u001b[32mBuilding CXX object CMakeFiles/CreateIwormFastaBundle.dir/base/StringUtil.cc.o\u001b[0m\n",
            "[ 45%] \u001b[32mBuilding CXX object CMakeFiles/CreateIwormFastaBundle.dir/util/mutil.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:2:0:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K\"NDEBUG\" redefined\n",
            " #define NDEBUG\n",
            " \n",
            "\u001b[01m\u001b[K<command-line>:0:0:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kthis is the location of the previous definition\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvirtual bool CMAsciiReadFileStream::ReadSimpleType(void*, long int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:303:51:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KpData\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiReadFileStream::ReadSimpleType(void * \u001b[01;35m\u001b[KpData\u001b[m\u001b[K, long lenInBytes)\n",
            "                                                   \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:303:63:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KlenInBytes\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiReadFileStream::ReadSimpleType(void * pData, long \u001b[01;35m\u001b[KlenInBytes\u001b[m\u001b[K)\n",
            "                                                               \u001b[01;35m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvirtual bool CMAsciiWriteFileStream::WriteBlob(const void*, long int, long int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:53:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KpData\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * \u001b[01;35m\u001b[KpData\u001b[m\u001b[K, long lenInElements, long elSize)\n",
            "                                                     \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:65:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KlenInElements\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * pData, long \u001b[01;35m\u001b[KlenInElements\u001b[m\u001b[K, long elSize)\n",
            "                                                                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:85:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KelSize\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * pData, long lenInElements, long \u001b[01;35m\u001b[KelSize\u001b[m\u001b[K)\n",
            "                                                                                     \u001b[01;35m\u001b[K^~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:738:53:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                     \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, long int, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:761:63:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, long val, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                               \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, const MCL_TCHAR*, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:787:80:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, const MCL_TCHAR * szText2, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                                                \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MCLSetUTF8Encode(bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:1288:39:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kb\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MCLSetUTF8Encode(bool \u001b[01;35m\u001b[Kb\u001b[m\u001b[K)\n",
            "                                       \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kbool AddUTF8Sig(CMString&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:1688:16:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kp\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "   const char * \u001b[01;35m\u001b[Kp\u001b[m\u001b[K = (const char*)string;\n",
            "                \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 47%] \u001b[32m\u001b[1mLinking CXX executable CreateIwormFastaBundle\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "[ 47%] Built target CreateIwormFastaBundle\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "\u001b[35m\u001b[1mScanning dependencies of target QuantifyGraph\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "[ 48%] \u001b[32mBuilding CXX object CMakeFiles/QuantifyGraph.dir/aligns/KmerAlignCore.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:2:0:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K\"NDEBUG\" redefined\n",
            " #define NDEBUG\n",
            " \n",
            "\u001b[01m\u001b[K<command-line>:0:0:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kthis is the location of the previous definition\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerAlignCore::AddData(const vecDNAVector&, const vecNumVector&, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:79:49:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kt\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "                 KmerAlignCoreRecordStoreTable & \u001b[01;35m\u001b[Kt\u001b[m\u001b[K = m_table; // not used?\n",
            "                                                 \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:49:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j, k;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerAlignCore::AddData(const DNAVector&, int, int, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:128:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j, k;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:128:12:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kj\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int i, \u001b[01;35m\u001b[Kj\u001b[m\u001b[K, k;\n",
            "            \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerAlignCore::SortAll()\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:148:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kconst svec<KmerAlignCoreRecord>& KmerAlignCore::GetMatchesDirectly(const DNAVector&, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:158:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ksize\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ksize\u001b[m\u001b[K = m_pTrans->GetSize();\n",
            "         \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:159:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:159:12:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kj\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int i, \u001b[01;35m\u001b[Kj\u001b[m\u001b[K;\n",
            "            \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerAlignCore::MergeSortFilter(svec<KmerAlignCoreRecord>&, const svec<KmerAlignCoreRecord>&, const svec<KmerAlignCoreRecord>&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:291:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 50%] \u001b[32mBuilding CXX object CMakeFiles/QuantifyGraph.dir/analysis/AACodons.cc.o\u001b[0m\n",
            "[ 51%] \u001b[32mBuilding CXX object CMakeFiles/QuantifyGraph.dir/analysis/DNAVector.cc.o\u001b[0m\n",
            "[ 52%] \u001b[32mBuilding CXX object CMakeFiles/QuantifyGraph.dir/analysis/KmerTable.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/KmerTable.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Klong long int KmerSequence::BasesToNumber(const DNAVector&, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/KmerTable.cc:27:13:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "   long long \u001b[01;35m\u001b[Ki\u001b[m\u001b[K;\n",
            "             \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 54%] \u001b[32mBuilding CXX object CMakeFiles/QuantifyGraph.dir/analysis/NonRedKmerTable.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid NonRedKmerTable::SetUp(const vecDNAVector&, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:46:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         for (j=0; \u001b[01;35m\u001b[Kj<=d.size()-m_k\u001b[m\u001b[K; j++) {\n",
            "                   \u001b[01;35m\u001b[K~^~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:56:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         for (j=0; \u001b[01;35m\u001b[Kj<=d.size()-m_k\u001b[m\u001b[K; j++) {\n",
            "                   \u001b[01;35m\u001b[K~^~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:79:18:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "       for (j=0; \u001b[01;35m\u001b[Kj<=d.size()-m_k\u001b[m\u001b[K; j++) {\n",
            "                 \u001b[01;35m\u001b[K~^~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid NonRedKmerTable::AddData(const vecDNAVector&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:109:16:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "     for (j=0; \u001b[01;35m\u001b[Kj<= d.isize()-m_k\u001b[m\u001b[K; j++) {\n",
            "               \u001b[01;35m\u001b[K~^~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid NonRedKmerTable::AddData(vecDNAVectorStream&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:125:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 55%] \u001b[32mBuilding CXX object CMakeFiles/QuantifyGraph.dir/analysis/QuantifyGraph.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/QuantifyGraph.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Klong long int BasesToNumberCountPlus(const std::vector<KmerEntry, std::allocator<KmerEntry> >&, svec<IDS>&, long long int&, const DNAVector&, int, const vecDNAVector&, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/QuantifyGraph.cc:250:34:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[KISO C++ forbids variable length array ‘\u001b[01m\u001b[Kkmerseq\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wvla\u001b[m\u001b[K]\n",
            "     char kmerseq [kmer_length + 1\u001b[01;35m\u001b[K]\u001b[m\u001b[K;\n",
            "                                  \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/QuantifyGraph.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/QuantifyGraph.cc:372:8:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[KprevNode\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "    int \u001b[01;35m\u001b[KprevNode\u001b[m\u001b[K = parser.AsInt(1);\n",
            "        \u001b[01;35m\u001b[K^~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/QuantifyGraph.cc:409:13:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Knode\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "         int \u001b[01;35m\u001b[Knode\u001b[m\u001b[K = parser.AsInt(0);\n",
            "             \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/QuantifyGraph.cc:412:16:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kp2\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "   const char * \u001b[01;35m\u001b[Kp2\u001b[m\u001b[K = s.c_str();\n",
            "                \u001b[01;35m\u001b[K^~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/QuantifyGraph.cc:337:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kj\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "  int i, \u001b[01;35m\u001b[Kj\u001b[m\u001b[K;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/QuantifyGraph.cc:350:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Km\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "  size_t \u001b[01;35m\u001b[Km\u001b[m\u001b[K = kmers.size();\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 57%] \u001b[32mBuilding CXX object CMakeFiles/QuantifyGraph.dir/base/ErrorHandling.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid print_trace(FILE*, const char*, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:24:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kout\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *\u001b[01;35m\u001b[Kout\u001b[m\u001b[K, const char *file, int line)\n",
            "                        \u001b[01;35m\u001b[K^~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:41:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kfile\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *out, const char *\u001b[01;35m\u001b[Kfile\u001b[m\u001b[K, int line)\n",
            "                                         \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:51:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kline\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *out, const char *file, int \u001b[01;35m\u001b[Kline\u001b[m\u001b[K)\n",
            "                                                   \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "[ 58%] \u001b[32mBuilding CXX object CMakeFiles/QuantifyGraph.dir/base/FileParser.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/FileParser.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kbool StringParser::IsString(int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/FileParser.cc:50:33:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kindex\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool StringParser::IsString(int \u001b[01;35m\u001b[Kindex\u001b[m\u001b[K)\n",
            "                                 \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "[ 60%] \u001b[32mBuilding CXX object CMakeFiles/QuantifyGraph.dir/base/StringUtil.cc.o\u001b[0m\n",
            "[ 61%] \u001b[32mBuilding CXX object CMakeFiles/QuantifyGraph.dir/analysis/sequenceUtil.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/sequenceUtil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kbool contains_non_gatc(std::__cxx11::string)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/sequenceUtil.cc:33:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Karray subscript has type ‘\u001b[01m\u001b[Kchar\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wchar-subscripts\u001b[m\u001b[K]\n",
            "  if (_base_to_int[c\u001b[01;35m\u001b[K]\u001b[m\u001b[K > 3)\n",
            "                    \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/sequenceUtil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kkmer_int_type_t kmer_to_intval(std::__cxx11::string)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/sequenceUtil.cc:264:26:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Karray subscript has type ‘\u001b[01m\u001b[Kchar\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wchar-subscripts\u001b[m\u001b[K]\n",
            "  int val = _base_to_int[c\u001b[01;35m\u001b[K]\u001b[m\u001b[K;\n",
            "                          \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 62%] \u001b[32mBuilding CXX object CMakeFiles/QuantifyGraph.dir/analysis/stacktrace.cc.o\u001b[0m\n",
            "[ 64%] \u001b[32mBuilding CXX object CMakeFiles/QuantifyGraph.dir/util/mutil.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:2:0:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K\"NDEBUG\" redefined\n",
            " #define NDEBUG\n",
            " \n",
            "\u001b[01m\u001b[K<command-line>:0:0:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kthis is the location of the previous definition\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvirtual bool CMAsciiReadFileStream::ReadSimpleType(void*, long int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:303:51:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KpData\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiReadFileStream::ReadSimpleType(void * \u001b[01;35m\u001b[KpData\u001b[m\u001b[K, long lenInBytes)\n",
            "                                                   \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:303:63:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KlenInBytes\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiReadFileStream::ReadSimpleType(void * pData, long \u001b[01;35m\u001b[KlenInBytes\u001b[m\u001b[K)\n",
            "                                                               \u001b[01;35m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvirtual bool CMAsciiWriteFileStream::WriteBlob(const void*, long int, long int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:53:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KpData\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * \u001b[01;35m\u001b[KpData\u001b[m\u001b[K, long lenInElements, long elSize)\n",
            "                                                     \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:65:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KlenInElements\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * pData, long \u001b[01;35m\u001b[KlenInElements\u001b[m\u001b[K, long elSize)\n",
            "                                                                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:85:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KelSize\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * pData, long lenInElements, long \u001b[01;35m\u001b[KelSize\u001b[m\u001b[K)\n",
            "                                                                                     \u001b[01;35m\u001b[K^~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:738:53:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                     \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, long int, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:761:63:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, long val, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                               \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, const MCL_TCHAR*, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:787:80:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, const MCL_TCHAR * szText2, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                                                \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MCLSetUTF8Encode(bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:1288:39:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kb\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MCLSetUTF8Encode(bool \u001b[01;35m\u001b[Kb\u001b[m\u001b[K)\n",
            "                                       \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kbool AddUTF8Sig(CMString&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:1688:16:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kp\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "   const char * \u001b[01;35m\u001b[Kp\u001b[m\u001b[K = (const char*)string;\n",
            "                \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 65%] \u001b[32m\u001b[1mLinking CXX executable QuantifyGraph\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "[ 65%] Built target QuantifyGraph\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "\u001b[35m\u001b[1mScanning dependencies of target BubbleUpClustering\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "[ 67%] \u001b[32mBuilding CXX object CMakeFiles/BubbleUpClustering.dir/aligns/KmerAlignCore.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:2:0:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K\"NDEBUG\" redefined\n",
            " #define NDEBUG\n",
            " \n",
            "\u001b[01m\u001b[K<command-line>:0:0:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kthis is the location of the previous definition\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerAlignCore::AddData(const vecDNAVector&, const vecNumVector&, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:79:49:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kt\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "                 KmerAlignCoreRecordStoreTable & \u001b[01;35m\u001b[Kt\u001b[m\u001b[K = m_table; // not used?\n",
            "                                                 \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:49:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j, k;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerAlignCore::AddData(const DNAVector&, int, int, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:128:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j, k;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:128:12:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kj\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int i, \u001b[01;35m\u001b[Kj\u001b[m\u001b[K, k;\n",
            "            \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerAlignCore::SortAll()\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:148:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kconst svec<KmerAlignCoreRecord>& KmerAlignCore::GetMatchesDirectly(const DNAVector&, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:158:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ksize\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ksize\u001b[m\u001b[K = m_pTrans->GetSize();\n",
            "         \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:159:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:159:12:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kj\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int i, \u001b[01;35m\u001b[Kj\u001b[m\u001b[K;\n",
            "            \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerAlignCore::MergeSortFilter(svec<KmerAlignCoreRecord>&, const svec<KmerAlignCoreRecord>&, const svec<KmerAlignCoreRecord>&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:291:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 68%] \u001b[32mBuilding CXX object CMakeFiles/BubbleUpClustering.dir/analysis/AACodons.cc.o\u001b[0m\n",
            "[ 70%] \u001b[32mBuilding CXX object CMakeFiles/BubbleUpClustering.dir/analysis/BubbleUpClustering.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/BubbleUpClustering.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Ksvec<Pool> grow_prioritized_clusters(std::__cxx11::string&, std::map<int, Pool>&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/BubbleUpClustering.cc:111:78:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kweld_reinforced_iworm_clusters\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " svec<Pool> grow_prioritized_clusters(string& weld_graph_file, map<int,Pool>& \u001b[01;35m\u001b[Kweld_reinforced_iworm_clusters\u001b[m\u001b[K) {\n",
            "                                                                              \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/BubbleUpClustering.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/BubbleUpClustering.cc:628:30:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         for (size_t j = 0; \u001b[01;35m\u001b[Kj < p.size()\u001b[m\u001b[K; j++) {\n",
            "                            \u001b[01;35m\u001b[K~~^~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/BubbleUpClustering.cc:641:30:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         for (size_t j = 0; \u001b[01;35m\u001b[Kj < p.size()\u001b[m\u001b[K; j++) {\n",
            "                            \u001b[01;35m\u001b[K~~^~~~~~~~~~\u001b[m\u001b[K\n",
            "[ 71%] \u001b[32mBuilding CXX object CMakeFiles/BubbleUpClustering.dir/analysis/DNAVector.cc.o\u001b[0m\n",
            "[ 72%] \u001b[32mBuilding CXX object CMakeFiles/BubbleUpClustering.dir/analysis/DeBruijnGraph.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid DeBruijnKmer::add_next_kmer(kmer_int_type_t, unsigned int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:154:66:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kkmer_length\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void DeBruijnKmer::add_next_kmer(kmer_int_type_t k, unsigned int \u001b[01;35m\u001b[Kkmer_length\u001b[m\u001b[K) {\n",
            "                                                                  \u001b[01;35m\u001b[K^~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kstd::vector<DeBruijnKmer> DeBruijnGraph::deconvolute_DS_mirror_graph()\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:355:22:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[Krdk\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K-Wunused-but-set-variable\u001b[m\u001b[K]\n",
            "         DeBruijnKmer \u001b[01;35m\u001b[Krdk\u001b[m\u001b[K = _kmer_map.find(rk)->second;\n",
            "                      \u001b[01;35m\u001b[K^~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kstd::vector<std::__cxx11::basic_string<char> > DeBruijnGraph::get_candidate_weldmers(kmer_int_type_t, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:473:23:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "     for (int i = 0; \u001b[01;35m\u001b[Ki < left_extensions.size()\u001b[m\u001b[K; i++) {\n",
            "                     \u001b[01;35m\u001b[K~~^~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:476:27:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         for (int j = 0; \u001b[01;35m\u001b[Kj < right_extensions.size()\u001b[m\u001b[K; j++) {\n",
            "                         \u001b[01;35m\u001b[K~~^~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid DeBruijnGraph::recursively_construct_kmer_extensions(kmer_int_type_t, std::vector<char>&, std::vector<std::__cxx11::basic_string<char> >&, char, std::map<long long unsigned int, bool>&, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:513:23:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "     for (int i = 0; \u001b[01;35m\u001b[Ki < adjacent_kmers.size()\u001b[m\u001b[K; i++) {\n",
            "                     \u001b[01;35m\u001b[K~~^~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/DeBruijnGraph.cc:527:41:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         if (\u001b[01;35m\u001b[Kkmer_extension_chars.size() == flank_extension_length\u001b[m\u001b[K) {\n",
            "             \u001b[01;35m\u001b[K~~~~~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "[ 74%] \u001b[32mBuilding CXX object CMakeFiles/BubbleUpClustering.dir/analysis/KmerTable.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/KmerTable.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Klong long int KmerSequence::BasesToNumber(const DNAVector&, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/KmerTable.cc:27:13:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "   long long \u001b[01;35m\u001b[Ki\u001b[m\u001b[K;\n",
            "             \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 75%] \u001b[32mBuilding CXX object CMakeFiles/BubbleUpClustering.dir/analysis/NonRedKmerTable.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid NonRedKmerTable::SetUp(const vecDNAVector&, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:46:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         for (j=0; \u001b[01;35m\u001b[Kj<=d.size()-m_k\u001b[m\u001b[K; j++) {\n",
            "                   \u001b[01;35m\u001b[K~^~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:56:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         for (j=0; \u001b[01;35m\u001b[Kj<=d.size()-m_k\u001b[m\u001b[K; j++) {\n",
            "                   \u001b[01;35m\u001b[K~^~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:79:18:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "       for (j=0; \u001b[01;35m\u001b[Kj<=d.size()-m_k\u001b[m\u001b[K; j++) {\n",
            "                 \u001b[01;35m\u001b[K~^~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid NonRedKmerTable::AddData(const vecDNAVector&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:109:16:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "     for (j=0; \u001b[01;35m\u001b[Kj<= d.isize()-m_k\u001b[m\u001b[K; j++) {\n",
            "               \u001b[01;35m\u001b[K~^~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid NonRedKmerTable::AddData(vecDNAVectorStream&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/NonRedKmerTable.cc:125:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 77%] \u001b[32mBuilding CXX object CMakeFiles/BubbleUpClustering.dir/analysis/sequenceUtil.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/sequenceUtil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kbool contains_non_gatc(std::__cxx11::string)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/sequenceUtil.cc:33:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Karray subscript has type ‘\u001b[01m\u001b[Kchar\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wchar-subscripts\u001b[m\u001b[K]\n",
            "  if (_base_to_int[c\u001b[01;35m\u001b[K]\u001b[m\u001b[K > 3)\n",
            "                    \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/sequenceUtil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kkmer_int_type_t kmer_to_intval(std::__cxx11::string)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/sequenceUtil.cc:264:26:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Karray subscript has type ‘\u001b[01m\u001b[Kchar\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wchar-subscripts\u001b[m\u001b[K]\n",
            "  int val = _base_to_int[c\u001b[01;35m\u001b[K]\u001b[m\u001b[K;\n",
            "                          \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 78%] \u001b[32mBuilding CXX object CMakeFiles/BubbleUpClustering.dir/analysis/stacktrace.cc.o\u001b[0m\n",
            "[ 80%] \u001b[32mBuilding CXX object CMakeFiles/BubbleUpClustering.dir/base/ErrorHandling.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid print_trace(FILE*, const char*, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:24:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kout\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *\u001b[01;35m\u001b[Kout\u001b[m\u001b[K, const char *file, int line)\n",
            "                        \u001b[01;35m\u001b[K^~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:41:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kfile\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *out, const char *\u001b[01;35m\u001b[Kfile\u001b[m\u001b[K, int line)\n",
            "                                         \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:51:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kline\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *out, const char *file, int \u001b[01;35m\u001b[Kline\u001b[m\u001b[K)\n",
            "                                                   \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "[ 81%] \u001b[32mBuilding CXX object CMakeFiles/BubbleUpClustering.dir/base/FileParser.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/FileParser.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kbool StringParser::IsString(int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/FileParser.cc:50:33:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kindex\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool StringParser::IsString(int \u001b[01;35m\u001b[Kindex\u001b[m\u001b[K)\n",
            "                                 \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "[ 82%] \u001b[32mBuilding CXX object CMakeFiles/BubbleUpClustering.dir/base/StringUtil.cc.o\u001b[0m\n",
            "[ 84%] \u001b[32mBuilding CXX object CMakeFiles/BubbleUpClustering.dir/util/mutil.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:2:0:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K\"NDEBUG\" redefined\n",
            " #define NDEBUG\n",
            " \n",
            "\u001b[01m\u001b[K<command-line>:0:0:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kthis is the location of the previous definition\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvirtual bool CMAsciiReadFileStream::ReadSimpleType(void*, long int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:303:51:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KpData\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiReadFileStream::ReadSimpleType(void * \u001b[01;35m\u001b[KpData\u001b[m\u001b[K, long lenInBytes)\n",
            "                                                   \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:303:63:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KlenInBytes\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiReadFileStream::ReadSimpleType(void * pData, long \u001b[01;35m\u001b[KlenInBytes\u001b[m\u001b[K)\n",
            "                                                               \u001b[01;35m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvirtual bool CMAsciiWriteFileStream::WriteBlob(const void*, long int, long int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:53:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KpData\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * \u001b[01;35m\u001b[KpData\u001b[m\u001b[K, long lenInElements, long elSize)\n",
            "                                                     \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:65:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KlenInElements\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * pData, long \u001b[01;35m\u001b[KlenInElements\u001b[m\u001b[K, long elSize)\n",
            "                                                                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:85:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KelSize\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * pData, long lenInElements, long \u001b[01;35m\u001b[KelSize\u001b[m\u001b[K)\n",
            "                                                                                     \u001b[01;35m\u001b[K^~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:738:53:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                     \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, long int, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:761:63:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, long val, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                               \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, const MCL_TCHAR*, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:787:80:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, const MCL_TCHAR * szText2, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                                                \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MCLSetUTF8Encode(bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:1288:39:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kb\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MCLSetUTF8Encode(bool \u001b[01;35m\u001b[Kb\u001b[m\u001b[K)\n",
            "                                       \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kbool AddUTF8Sig(CMString&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:1688:16:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kp\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "   const char * \u001b[01;35m\u001b[Kp\u001b[m\u001b[K = (const char*)string;\n",
            "                \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 85%] \u001b[32m\u001b[1mLinking CXX executable BubbleUpClustering\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "[ 85%] Built target BubbleUpClustering\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "\u001b[35m\u001b[1mScanning dependencies of target Chrysalis\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "[ 87%] \u001b[32mBuilding CXX object CMakeFiles/Chrysalis.dir/aligns/KmerAlignCore.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:2:0:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K\"NDEBUG\" redefined\n",
            " #define NDEBUG\n",
            " \n",
            "\u001b[01m\u001b[K<command-line>:0:0:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kthis is the location of the previous definition\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerAlignCore::AddData(const vecDNAVector&, const vecNumVector&, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:79:49:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kt\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "                 KmerAlignCoreRecordStoreTable & \u001b[01;35m\u001b[Kt\u001b[m\u001b[K = m_table; // not used?\n",
            "                                                 \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:49:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j, k;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerAlignCore::AddData(const DNAVector&, int, int, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:128:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j, k;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:128:12:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kj\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int i, \u001b[01;35m\u001b[Kj\u001b[m\u001b[K, k;\n",
            "            \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerAlignCore::SortAll()\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:148:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kconst svec<KmerAlignCoreRecord>& KmerAlignCore::GetMatchesDirectly(const DNAVector&, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:158:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ksize\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ksize\u001b[m\u001b[K = m_pTrans->GetSize();\n",
            "         \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:159:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K, j;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:159:12:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kj\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int i, \u001b[01;35m\u001b[Kj\u001b[m\u001b[K;\n",
            "            \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerAlignCore::MergeSortFilter(svec<KmerAlignCoreRecord>&, const svec<KmerAlignCoreRecord>&, const svec<KmerAlignCoreRecord>&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/aligns/KmerAlignCore.cc:291:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[Ki\u001b[m\u001b[K;\n",
            "         \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[ 88%] \u001b[32mBuilding CXX object CMakeFiles/Chrysalis.dir/analysis/AACodons.cc.o\u001b[0m\n",
            "[ 90%] \u001b[32mBuilding CXX object CMakeFiles/Chrysalis.dir/analysis/Chrysalis.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/Chrysalis.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint main(int, char**)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/Chrysalis.cc:582:21:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Knum_iworm_contigs\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "                 int \u001b[01;35m\u001b[Knum_iworm_contigs\u001b[m\u001b[K = parser.AsInt(2);\n",
            "                     \u001b[01;35m\u001b[K^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/Chrysalis.cc:570:16:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[KpOut\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "         FILE * \u001b[01;35m\u001b[KpOut\u001b[m\u001b[K = NULL;\n",
            "                \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/Chrysalis.cc:342:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[KbSkip\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     bool \u001b[01;35m\u001b[KbSkip\u001b[m\u001b[K = P.GetBoolValueFor(skipCmmd);\n",
            "          \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/Chrysalis.cc:347:9:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[KpairDist\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     int \u001b[01;35m\u001b[KpairDist\u001b[m\u001b[K = P.GetIntValueFor(distCmmd);\n",
            "         \u001b[01;35m\u001b[K^~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/Chrysalis.cc:354:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kvariable ‘\u001b[01m\u001b[KbBreak\u001b[m\u001b[K’ set but not used [\u001b[01;35m\u001b[K-Wunused-but-set-variable\u001b[m\u001b[K]\n",
            "     bool \u001b[01;35m\u001b[KbBreak\u001b[m\u001b[K = true;\n",
            "          \u001b[01;35m\u001b[K^~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/Chrysalis.cc:359:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[KbButt\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     bool \u001b[01;35m\u001b[KbButt\u001b[m\u001b[K = P.GetBoolValueFor(buttCmmd);\n",
            "          \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/Chrysalis.cc:361:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kmax_reads\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     long \u001b[01;35m\u001b[Kmax_reads\u001b[m\u001b[K = P.GetLongValueFor(maxReadsCmd);\n",
            "          \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/Chrysalis.cc:372:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[KDEBUG\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     bool \u001b[01;35m\u001b[KDEBUG\u001b[m\u001b[K = P.GetBoolValueFor(debugCmmd);\n",
            "          \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/Chrysalis.cc:384:11:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kignoring return value of ‘\u001b[01m\u001b[Kint system(const char*)\u001b[m\u001b[K’, declared with attribute warn_unused_result [\u001b[01;35m\u001b[K-Wunused-result\u001b[m\u001b[K]\n",
            "     \u001b[01;35m\u001b[Ksystem(command.c_str())\u001b[m\u001b[K;\n",
            "     \u001b[01;35m\u001b[K~~~~~~^~~~~~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "[ 91%] \u001b[32mBuilding CXX object CMakeFiles/Chrysalis.dir/analysis/DNAVector.cc.o\u001b[0m\n",
            "[ 92%] \u001b[32mBuilding CXX object CMakeFiles/Chrysalis.dir/analysis/TranscriptomeGraph.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/TranscriptomeGraph.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerSequence::Setup()\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/TranscriptomeGraph.cc:177:19:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Ki\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "         long long \u001b[01;35m\u001b[Ki\u001b[m\u001b[K;\n",
            "                   \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/TranscriptomeGraph.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvoid KmerSearch::Extend(long long int, DNAVector&, const svec<long long int>&, DNAVector&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/TranscriptomeGraph.cc:547:16:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kplusminus\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     static int \u001b[01;35m\u001b[Kplusminus\u001b[m\u001b[K = 0;\n",
            "                \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/TranscriptomeGraph.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kint TranscriptomeGraph(vecDNAVector&, FILE*, int, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/TranscriptomeGraph.cc:668:20:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kcomparison between signed and unsigned integer expressions [\u001b[01;35m\u001b[K-Wsign-compare\u001b[m\u001b[K]\n",
            "         for (i=0; \u001b[01;35m\u001b[Ki<=d.isize()-k\u001b[m\u001b[K; i++) {\n",
            "                   \u001b[01;35m\u001b[K~^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/TranscriptomeGraph.cc:647:10:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[KbAppend\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     bool \u001b[01;35m\u001b[KbAppend\u001b[m\u001b[K = true;\n",
            "          \u001b[01;35m\u001b[K^~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/TranscriptomeGraph.cc:\u001b[m\u001b[K At global scope:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/analysis/TranscriptomeGraph.cc:547:16:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K‘\u001b[01m\u001b[Kplusminus\u001b[m\u001b[K’ defined but not used [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "     static int \u001b[01;35m\u001b[Kplusminus\u001b[m\u001b[K = 0;\n",
            "                \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "[ 94%] \u001b[32mBuilding CXX object CMakeFiles/Chrysalis.dir/base/ErrorHandling.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid print_trace(FILE*, const char*, int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:24:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kout\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *\u001b[01;35m\u001b[Kout\u001b[m\u001b[K, const char *file, int line)\n",
            "                        \u001b[01;35m\u001b[K^~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:41:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kfile\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *out, const char *\u001b[01;35m\u001b[Kfile\u001b[m\u001b[K, int line)\n",
            "                                         \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/ErrorHandling.cc:7:51:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kline\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " void print_trace(FILE *out, const char *file, int \u001b[01;35m\u001b[Kline\u001b[m\u001b[K)\n",
            "                                                   \u001b[01;35m\u001b[K^~~~\u001b[m\u001b[K\n",
            "[ 95%] \u001b[32mBuilding CXX object CMakeFiles/Chrysalis.dir/base/FileParser.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/FileParser.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kbool StringParser::IsString(int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/base/FileParser.cc:50:33:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kindex\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool StringParser::IsString(int \u001b[01;35m\u001b[Kindex\u001b[m\u001b[K)\n",
            "                                 \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "[ 97%] \u001b[32mBuilding CXX object CMakeFiles/Chrysalis.dir/base/StringUtil.cc.o\u001b[0m\n",
            "[ 98%] \u001b[32mBuilding CXX object CMakeFiles/Chrysalis.dir/util/mutil.cc.o\u001b[0m\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:2:0:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[K\"NDEBUG\" redefined\n",
            " #define NDEBUG\n",
            " \n",
            "\u001b[01m\u001b[K<command-line>:0:0:\u001b[m\u001b[K \u001b[01;36m\u001b[Knote: \u001b[m\u001b[Kthis is the location of the previous definition\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvirtual bool CMAsciiReadFileStream::ReadSimpleType(void*, long int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:303:51:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KpData\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiReadFileStream::ReadSimpleType(void * \u001b[01;35m\u001b[KpData\u001b[m\u001b[K, long lenInBytes)\n",
            "                                                   \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:303:63:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KlenInBytes\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiReadFileStream::ReadSimpleType(void * pData, long \u001b[01;35m\u001b[KlenInBytes\u001b[m\u001b[K)\n",
            "                                                               \u001b[01;35m\u001b[K^~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In member function ‘\u001b[01m\u001b[Kvirtual bool CMAsciiWriteFileStream::WriteBlob(const void*, long int, long int)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:53:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KpData\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * \u001b[01;35m\u001b[KpData\u001b[m\u001b[K, long lenInElements, long elSize)\n",
            "                                                     \u001b[01;35m\u001b[K^~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:65:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KlenInElements\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * pData, long \u001b[01;35m\u001b[KlenInElements\u001b[m\u001b[K, long elSize)\n",
            "                                                                 \u001b[01;35m\u001b[K^~~~~~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:526:85:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KelSize\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " bool CMAsciiWriteFileStream::WriteBlob(const void * pData, long lenInElements, long \u001b[01;35m\u001b[KelSize\u001b[m\u001b[K)\n",
            "                                                                                     \u001b[01;35m\u001b[K^~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:738:53:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                     \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, long int, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:761:63:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, long val, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                               \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MLog(const MCL_TCHAR*, const MCL_TCHAR*, bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:787:80:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[KbLineFeed\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MLog(const MCL_TCHAR * szText, const MCL_TCHAR * szText2, bool \u001b[01;35m\u001b[KbLineFeed\u001b[m\u001b[K)\n",
            "                                                                                \u001b[01;35m\u001b[K^~~~~~~~~\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kvoid MCLSetUTF8Encode(bool)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:1288:39:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused parameter ‘\u001b[01m\u001b[Kb\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-parameter\u001b[m\u001b[K]\n",
            " MDLLEXPORT void MCLSetUTF8Encode(bool \u001b[01;35m\u001b[Kb\u001b[m\u001b[K)\n",
            "                                       \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:\u001b[m\u001b[K In function ‘\u001b[01m\u001b[Kbool AddUTF8Sig(CMString&)\u001b[m\u001b[K’:\n",
            "\u001b[01m\u001b[K/content/trinityrnaseq-v2.13.2/Chrysalis/util/mutil.cc:1688:16:\u001b[m\u001b[K \u001b[01;35m\u001b[Kwarning: \u001b[m\u001b[Kunused variable ‘\u001b[01m\u001b[Kp\u001b[m\u001b[K’ [\u001b[01;35m\u001b[K-Wunused-variable\u001b[m\u001b[K]\n",
            "   const char * \u001b[01;35m\u001b[Kp\u001b[m\u001b[K = (const char*)string;\n",
            "                \u001b[01;35m\u001b[K^\u001b[m\u001b[K\n",
            "[100%] \u001b[32m\u001b[1mLinking CXX executable Chrysalis\u001b[0m\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "[100%] Built target Chrysalis\n",
            "make[3]: Leaving directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "\u001b[36mInstall the project...\u001b[0m\n",
            "-- Install configuration: \"Release\"\n",
            "-- Installing: ../bin/Chrysalis\n",
            "-- Installing: ../bin/GraphFromFasta\n",
            "-- Installing: ../bin/QuantifyGraph\n",
            "-- Installing: ../bin/ReadsToTranscripts\n",
            "-- Installing: ../bin/BubbleUpClustering\n",
            "-- Installing: ../bin/CreateIwormFastaBundle\n",
            "make[2]: Leaving directory '/content/trinityrnaseq-v2.13.2/Chrysalis/build'\n",
            "make[1]: Leaving directory '/content/trinityrnaseq-v2.13.2/Chrysalis'\n",
            "cd trinity-plugins && make trinity_essentials\n",
            "make[1]: Entering directory '/content/trinityrnaseq-v2.13.2/trinity-plugins'\n",
            "cd seqtk-trinity && make CXX=g++ CC=gcc\n",
            "make[2]: Entering directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/seqtk-trinity'\n",
            "gcc -g -Wall -O2 -Wno-unused-function  seqtk.c -o seqtk-trinity -lz -lm\n",
            "make[2]: Leaving directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/seqtk-trinity'\n",
            "mv seqtk-trinity/seqtk-trinity ./BIN/.\n",
            "cd \"ParaFly\" && sh ./configure --prefix=`pwd` CXX=g++ CC=gcc CFLAGS=\"-fopenmp\" CXXFLAGS=\"-fopenmp\" && make install && \\\n",
            "cp bin/ParaFly ../BIN/\n",
            "checking for a BSD-compatible install... /usr/bin/install -c\n",
            "checking whether build environment is sane... yes\n",
            "checking for a thread-safe mkdir -p... /bin/mkdir -p\n",
            "checking for gawk... no\n",
            "checking for mawk... mawk\n",
            "checking whether make sets $(MAKE)... yes\n",
            "checking for C++ compiler default output file name... a.out\n",
            "checking whether the C++ compiler works... yes\n",
            "checking whether we are cross compiling... no\n",
            "checking for suffix of executables... \n",
            "checking for suffix of object files... o\n",
            "checking whether we are using the GNU C++ compiler... yes\n",
            "checking whether g++ accepts -g... yes\n",
            "checking for style of include used by make... GNU\n",
            "checking dependency style of g++... gcc3\n",
            "checking for library containing cos... none required\n",
            "configure: creating ./config.status\n",
            "config.status: creating Makefile\n",
            "config.status: creating src/Makefile\n",
            "config.status: creating config.h\n",
            "config.status: config.h is unchanged\n",
            "config.status: executing depfiles commands\n",
            "make[2]: Entering directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/ParaFly'\n",
            "Making install in src\n",
            "make[3]: Entering directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/ParaFly/src'\n",
            "g++ -DHAVE_CONFIG_H -I. -I..    -pedantic -fopenmp -Wall -Wextra -Wno-long-long -Wno-deprecated -m64 -fopenmp -MT ParaFly.o -MD -MP -MF .deps/ParaFly.Tpo -c -o ParaFly.o ParaFly.cpp\n",
            "mv -f .deps/ParaFly.Tpo .deps/ParaFly.Po\n",
            "g++ -DHAVE_CONFIG_H -I. -I..    -pedantic -fopenmp -Wall -Wextra -Wno-long-long -Wno-deprecated -m64 -fopenmp -MT argProcessor.o -MD -MP -MF .deps/argProcessor.Tpo -c -o argProcessor.o argProcessor.cpp\n",
            "mv -f .deps/argProcessor.Tpo .deps/argProcessor.Po\n",
            "g++ -pedantic -fopenmp -Wall -Wextra -Wno-long-long -Wno-deprecated -m64 -fopenmp   -o ParaFly ParaFly.o argProcessor.o  \n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/ParaFly/src'\n",
            "test -z \"/content/trinityrnaseq-v2.13.2/trinity-plugins/ParaFly/bin\" || /bin/mkdir -p \"/content/trinityrnaseq-v2.13.2/trinity-plugins/ParaFly/bin\"\n",
            "  /usr/bin/install -c ParaFly '/content/trinityrnaseq-v2.13.2/trinity-plugins/ParaFly/bin'\n",
            "make[4]: Nothing to be done for 'install-data-am'.\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/ParaFly/src'\n",
            "make[3]: Leaving directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/ParaFly/src'\n",
            "make[3]: Entering directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/ParaFly'\n",
            "make[4]: Entering directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/ParaFly'\n",
            "make[4]: Nothing to be done for 'install-exec-am'.\n",
            "make[4]: Nothing to be done for 'install-data-am'.\n",
            "make[4]: Leaving directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/ParaFly'\n",
            "make[3]: Leaving directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/ParaFly'\n",
            "make[2]: Leaving directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/ParaFly'\n",
            "ln -sf Trimmomatic-0.36 Trimmomatic\n",
            "cd bamsifter && make\n",
            "make[2]: Entering directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/bamsifter'\n",
            "./build_htslib.sh\n",
            "\n",
            "cd htslib\n",
            "mkdir -p build\n",
            "autoheader\n",
            "./build_htslib.sh: line 7: autoheader: command not found\n",
            "Makefile:10: recipe for target 'htslib/version.h' failed\n",
            "make[2]: *** [htslib/version.h] Error 127\n",
            "make[2]: Leaving directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/bamsifter'\n",
            "Makefile:31: recipe for target 'bamsifter_target' failed\n",
            "make[1]: *** [bamsifter_target] Error 2\n",
            "make[1]: Leaving directory '/content/trinityrnaseq-v2.13.2/trinity-plugins'\n",
            "Makefile:39: recipe for target 'trinity_essentials' failed\n",
            "make: *** [trinity_essentials] Error 2\n",
            "cd trinity-plugins && make plugins\n",
            "make[1]: Entering directory '/content/trinityrnaseq-v2.13.2/trinity-plugins'\n",
            "cd slclust && make CXX=g++ CC=gcc install\n",
            "make[2]: Entering directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/slclust'\n",
            "X=`pwd`; \\\n",
            "for i in src; \\\n",
            "do echo '<<<' $i '>>>'; cd $X/$i; make install; done\n",
            "<<< src >>>\n",
            "make[3]: Entering directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/slclust/src'\n",
            "g++ -I../include -Wall    -c -o slcluster.o slcluster.cpp\n",
            "g++ -I../include -Wall    -c -o graph.o graph.cpp\n",
            "g++ -I../include -Wall    -c -o graphnode.o graphnode.cpp\n",
            "g++ -I../include -Wall    -c -o cmd_line_opts.o cmd_line_opts.cpp\n",
            "g++  slcluster.o graph.o graphnode.o cmd_line_opts.o  -o slclust\n",
            "chmod 755 slclust\n",
            "mv slclust ../bin/\n",
            "make[3]: Leaving directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/slclust/src'\n",
            "make[2]: Leaving directory '/content/trinityrnaseq-v2.13.2/trinity-plugins/slclust'\n",
            "cd COLLECTL && tar xvf \"collectl-4.1.0\".src.tar.gz && ln -sf \"collectl-4.1.0\" collectl\n",
            "collectl-4.1.0/\n",
            "collectl-4.1.0/service/\n",
            "collectl-4.1.0/service/collectl.service\n",
            "collectl-4.1.0/docs/\n",
            "collectl-4.1.0/docs/TheMath.html\n",
            "collectl-4.1.0/docs/RunningAsAService.html\n",
            "collectl-4.1.0/docs/Ganglia3.jpg\n",
            "collectl-4.1.0/docs/SlowProc.html\n",
            "collectl-4.1.0/docs/Tutorial-Lustre.html\n",
            "collectl-4.1.0/docs/Playback.html\n",
            "collectl-4.1.0/docs/Memory.html\n",
            "collectl-4.1.0/docs/WhySummary.html\n",
            "collectl-4.1.0/docs/Matrix.html\n",
            "collectl-4.1.0/docs/Data.html\n",
            "collectl-4.1.0/docs/Infiniband.html\n",
            "collectl-4.1.0/docs/Interrupts.html\n",
            "collectl-4.1.0/docs/style.css\n",
            "collectl-4.1.0/docs/SlabInfo.html\n",
            "collectl-4.1.0/docs/OutputFiles.html\n",
            "collectl-4.1.0/docs/Data-brief.html\n",
            "collectl-4.1.0/docs/Ganglia.jpg\n",
            "collectl-4.1.0/docs/Tutorial.html\n",
            "collectl-4.1.0/docs/Examples.html\n",
            "collectl-4.1.0/docs/colmux.html\n",
            "collectl-4.1.0/docs/Gexpr.html\n",
            "collectl-4.1.0/docs/markseger,collectl@web.sourceforge.net\n",
            "collectl-4.1.0/docs/Inodes.html\n",
            "collectl-4.1.0/docs/OperationalModes.html\n",
            "collectl-4.1.0/docs/Data-verbose.html\n",
            "collectl-4.1.0/docs/Plotfiles.html\n",
            "collectl-4.1.0/docs/Ganglia2.jpg\n",
            "collectl-4.1.0/docs/NfsInfo.html\n",
            "collectl-4.1.0/docs/InputFiles.html\n",
            "collectl-4.1.0/docs/Messages.html\n",
            "collectl-4.1.0/docs/Disks.html\n",
            "collectl-4.1.0/docs/Hello.html\n",
            "collectl-4.1.0/docs/HiResTime.html\n",
            "collectl-4.1.0/docs/Startup.html\n",
            "collectl-4.1.0/docs/Network.html\n",
            "collectl-4.1.0/docs/FAQ-collectl.html\n",
            "collectl-4.1.0/docs/Misc.html\n",
            "collectl-4.1.0/docs/slub.jpg\n",
            "collectl-4.1.0/docs/Graphite.html\n",
            "collectl-4.1.0/docs/OutputFormats.html\n",
            "collectl-4.1.0/docs/Data-detail.html\n",
            "collectl-4.1.0/docs/Sockets.html\n",
            "collectl-4.1.0/docs/Process.html\n",
            "collectl-4.1.0/docs/ColmuxCPU.jpg\n",
            "collectl-4.1.0/docs/Environmental.html\n",
            "collectl-4.1.0/docs/Socket.html\n",
            "collectl-4.1.0/docs/Export.html\n",
            "collectl-4.1.0/docs/FileNaming.html\n",
            "collectl-4.1.0/docs/Lustre.html\n",
            "collectl-4.1.0/docs/Import.html\n",
            "collectl-4.1.0/docs/Performance.html\n",
            "collectl-4.1.0/docs/Exceptions.html\n",
            "collectl-4.1.0/docs/CPUs.html\n",
            "collectl-4.1.0/docs/Logging.html\n",
            "collectl-4.1.0/docs/ColmuxLustre.jpg\n",
            "collectl-4.1.0/docs/Documentation.html\n",
            "collectl-4.1.0/docs/OpenStack.html\n",
            "collectl-4.1.0/docs/BuddyInfo.html\n",
            "collectl-4.1.0/ARTISTIC\n",
            "collectl-4.1.0/envrules.std\n",
            "collectl-4.1.0/COPYING\n",
            "collectl-4.1.0/vmsum.ph\n",
            "collectl-4.1.0/vnet.ph\n",
            "collectl-4.1.0/gexpr.ph\n",
            "collectl-4.1.0/GPL\n",
            "collectl-4.1.0/README-WINDOWS\n",
            "collectl-4.1.0/client.pl\n",
            "collectl-4.1.0/man1/\n",
            "collectl-4.1.0/man1/colmux.1\n",
            "collectl-4.1.0/man1/collectl.1\n",
            "collectl-4.1.0/vmstat.ph\n",
            "collectl-4.1.0/misc.ph\n",
            "collectl-4.1.0/initd/\n",
            "collectl-4.1.0/initd/collectl-suse\n",
            "collectl-4.1.0/initd/collectl\n",
            "collectl-4.1.0/initd/collectl-debian\n",
            "collectl-4.1.0/initd/collectl-generic\n",
            "collectl-4.1.0/collectl.conf\n",
            "collectl-4.1.0/collectl\n",
            "collectl-4.1.0/colmux\n",
            "collectl-4.1.0/INSTALL\n",
            "collectl-4.1.0/graphite.ph\n",
            "collectl-4.1.0/proctree.ph\n",
            "collectl-4.1.0/README\n",
            "collectl-4.1.0/UNINSTALL\n",
            "collectl-4.1.0/RELEASE-collectl\n",
            "collectl-4.1.0/hello.ph\n",
            "collectl-4.1.0/formatit.ph\n",
            "collectl-4.1.0/lexpr.ph\n",
            "collectl-4.1.0/statsd.ph\n",
            "collectl-4.1.0/col2tlviz.pl\n",
            "\n",
            "\n",
            "** Done building plugins **\n",
            "\n",
            "\n",
            "make[1]: Leaving directory '/content/trinityrnaseq-v2.13.2/trinity-plugins'\n",
            "sh ./util/support_scripts/plugin_install_tests.sh\n",
            "## Checking plugin installations:\n",
            "\n",
            "slclust:                 has been Installed Properly\n",
            "collectl:                has been Installed Properly\n",
            "util/support_scripts/trinity_installer.py\n",
            "Trinity package dir: /content/trinityrnaseq-v2.13.2\n",
            "Trinity package name: trinityrnaseq-v2.13.2\n",
            "sending incremental file list\n",
            "./\n",
            "Changelog.txt\n",
            "LICENSE\n",
            "LICENSE.txt\n",
            "Makefile\n",
            "README\n",
            "README.md\n",
            "Trinity\n",
            "__pull_trinity_ext_sample_data.sh\n",
            "developer.notes\n",
            "make.macOSX.sh\n",
            "Analysis/\n",
            "Analysis/DifferentialExpression/\n",
            "Analysis/DifferentialExpression/DE_graph_to_dot.pl\n",
            "Analysis/DifferentialExpression/DTE_to_DTU.pl\n",
            "Analysis/DifferentialExpression/GOplot.Rscript\n",
            "Analysis/DifferentialExpression/Glimma.Trinity.Rscript\n",
            "Analysis/DifferentialExpression/PtR\n",
            "Analysis/DifferentialExpression/ROKU.pl\n",
            "Analysis/DifferentialExpression/add_annot_to_trans_id.pl\n",
            "Analysis/DifferentialExpression/add_annotations_to_GO_and_lengths_file.R\n",
            "Analysis/DifferentialExpression/add_blastx_hit_to_trinity_id.pl\n",
            "Analysis/DifferentialExpression/analyze_diff_expr.pl\n",
            "Analysis/DifferentialExpression/assign_tissue_specific.pl\n",
            "Analysis/DifferentialExpression/compare_gene_trans_DE_ranks.pl\n",
            "Analysis/DifferentialExpression/cut_tree_into_clusters.pl\n",
            "Analysis/DifferentialExpression/define_clusters_by_cutting_tree.pl\n",
            "Analysis/DifferentialExpression/diff_expr_analysis_to_heatmap_html.pl\n",
            "Analysis/DifferentialExpression/diff_express.cgi\n",
            "Analysis/DifferentialExpression/downsample_count_matrix.pl\n",
            "Analysis/DifferentialExpression/extract_GO_enriched_genes.pl\n",
            "Analysis/DifferentialExpression/filter_diff_expr.pl\n",
            "Analysis/DifferentialExpression/filter_matrix_min_sum_rowcounts.pl\n",
            "Analysis/DifferentialExpression/get_tissue_enriched_DE_one_vs_all.pl\n",
            "Analysis/DifferentialExpression/get_transcript_lengths.pl\n",
            "Analysis/DifferentialExpression/identify_diff_isoform_splicing.pl\n",
            "Analysis/DifferentialExpression/log2_transform_matrix.pl\n",
            "Analysis/DifferentialExpression/log2_transform_median_center_fpkm_matrix.pl\n",
            "Analysis/DifferentialExpression/matrix_to_gene_plots.pl\n",
            "Analysis/DifferentialExpression/merge_matrices.pl\n",
            "Analysis/DifferentialExpression/merge_subclusters.pl\n",
            "Analysis/DifferentialExpression/plot_all_DE_MAplots.Rscript\n",
            "Analysis/DifferentialExpression/plot_all_DE_volcanos.Rscript\n",
            "Analysis/DifferentialExpression/plot_expression_patterns.pl\n",
            "Analysis/DifferentialExpression/plot_log2FC_hist.Rscript\n",
            "Analysis/DifferentialExpression/prune_isoforms_fasta.pl\n",
            "Analysis/DifferentialExpression/prune_isoforms_gtf.pl\n",
            "Analysis/DifferentialExpression/rank_roku_by_expr.pl\n",
            "Analysis/DifferentialExpression/remove_batch_effects_from_count_matrix.pl\n",
            "Analysis/DifferentialExpression/rename_matrix_column_labels.pl\n",
            "Analysis/DifferentialExpression/rename_matrix_feature_identifiers.pl\n",
            "Analysis/DifferentialExpression/replicates_to_sample_averages_matrix.pl\n",
            "Analysis/DifferentialExpression/run_DE_analysis.pl\n",
            "Analysis/DifferentialExpression/run_GOseq.pl\n",
            "Analysis/DifferentialExpression/run_TMM_normalization_write_FPKM_matrix.pl\n",
            "Analysis/DifferentialExpression/stratify_diff_expression.pl\n",
            "Analysis/DifferentialExpression/subcluster_to_canvasXpress_html.make_index_html.pl\n",
            "Analysis/DifferentialExpression/subcluster_to_canvasXpress_html.pl\n",
            "Analysis/DifferentialExpression/summarize_diff_expr_across_min_threshold_ranges.pl\n",
            "Analysis/DifferentialExpression/validate_UP_subset.Rscript\n",
            "Analysis/DifferentialExpression/R/\n",
            "Analysis/DifferentialExpression/R/edgeR.TMM.minimal.R\n",
            "Analysis/DifferentialExpression/R/edgeR_funcs.R\n",
            "Analysis/DifferentialExpression/R/get_cluster_info.R\n",
            "Analysis/DifferentialExpression/R/heatmap.3.R\n",
            "Analysis/DifferentialExpression/R/jaccard_distance.R\n",
            "Analysis/DifferentialExpression/R/manually_define_clusters.R\n",
            "Analysis/DifferentialExpression/R/misc_rnaseq_funcs.R\n",
            "Analysis/DifferentialExpression/R/pairs3.R\n",
            "Analysis/DifferentialExpression/R/rnaseq_plot_funcs.R\n",
            "Analysis/DifferentialExpression/R/test.heatmap.3.R\n",
            "Analysis/DifferentialExpression/R/vioplot2.R\n",
            "Analysis/DifferentialExpression/R/tests/\n",
            "Analysis/DifferentialExpression/R/tests/test_heatmap_w_pca.R\n",
            "Analysis/DifferentialExpression/TissueEnrichment/\n",
            "Analysis/DifferentialExpression/TissueEnrichment/DE_graph_to_dot.pl\n",
            "Analysis/DifferentialExpression/TissueEnrichment/DE_results_to_pairwise_summary.pl\n",
            "Analysis/DifferentialExpression/TissueEnrichment/README.md\n",
            "Analysis/DifferentialExpression/TissueEnrichment/group_isoforms_by_tissue_enrichment.pl\n",
            "Analysis/DifferentialExpression/TissueEnrichment/pairwise_DE_summary_to_DE_classification.pl\n",
            "Analysis/DifferentialExpression/cluster_sample_data/\n",
            "Analysis/DifferentialExpression/cluster_sample_data/MLF_ESC_NPC.cuff.genes.fpkm.matrix.gz\n",
            "Analysis/DifferentialExpression/cluster_sample_data/cleanme.pl\n",
            "Analysis/DifferentialExpression/cluster_sample_data/orig.samples.txt\n",
            "Analysis/DifferentialExpression/cluster_sample_data/runMe.sh\n",
            "Analysis/DifferentialExpression/cluster_sample_data/samples.txt\n",
            "Analysis/DifferentialExpression/cluster_sample_data/Islam_scde_data/\n",
            "Analysis/DifferentialExpression/cluster_sample_data/Islam_scde_data/es.mef.fpkm.matrix\n",
            "Analysis/DifferentialExpression/deprecated/\n",
            "Analysis/DifferentialExpression/deprecated/prep_n_run_GOplot.pl\n",
            "Analysis/DifferentialExpression/pairwise_summaries/\n",
            "Analysis/DifferentialExpression/pairwise_summaries/DE_pair_counts_to_matrix.pl\n",
            "Analysis/DifferentialExpression/pairwise_summaries/EBSeq_to_pairwise_summary.pl\n",
            "Analysis/DifferentialExpression/pairwise_summaries/add_counts_to_classes.pl\n",
            "Analysis/DifferentialExpression/pairwise_summaries/class_to_separate_fpkm_matrices.pl\n",
            "Analysis/DifferentialExpression/pairwise_summaries/edgeR_to_pairwise_summary.pl\n",
            "Analysis/DifferentialExpression/pairwise_summaries/examine_rank_correlation.pl\n",
            "Analysis/DifferentialExpression/pairwise_summaries/extract_venn_agree_from_summaries.pl\n",
            "Analysis/DifferentialExpression/pairwise_summaries/mmdiff_to_pairwise_summary.pl\n",
            "Analysis/DifferentialExpression/pairwise_summaries/notes\n",
            "Analysis/DifferentialExpression/pairwise_summaries/pairwise_DE_summary_to_DE_classification.pl\n",
            "Analysis/DifferentialExpression/pairwise_summaries/venn_pairwise_summaries.pair_stats.pl\n",
            "Analysis/DifferentialExpression/pairwise_summaries/venn_pairwise_summaries.pl\n",
            "Analysis/FL_reconstruction_analysis/\n",
            "Analysis/FL_reconstruction_analysis/FL_trans_analysis_pipeline.pl\n",
            "Analysis/FL_reconstruction_analysis/compute_oracle.pl\n",
            "Analysis/FL_reconstruction_analysis/count_by_expression_quintile.pl\n",
            "Analysis/FL_reconstruction_analysis/fusion_comparisons_via_maps_files.pl\n",
            "Analysis/FL_reconstruction_analysis/get_genes_from_maps_file.pl\n",
            "Analysis/FL_reconstruction_analysis/maps_file_to_paralog_representation.pl\n",
            "Analysis/FL_reconstruction_analysis/oracle_counter.pl\n",
            "Analysis/FL_reconstruction_analysis/tier_gene_trans_alignments.pl\n",
            "Analysis/FL_reconstruction_analysis/tier_gene_trans_alignments.tiers_to_boxplot.pl\n",
            "Analysis/FL_reconstruction_analysis/R/\n",
            "Analysis/FL_reconstruction_analysis/R/boot.tree.R\n",
            "Analysis/FL_reconstruction_analysis/util/\n",
            "Analysis/FL_reconstruction_analysis/util/blat_full_length_mappings.pl\n",
            "Analysis/FL_reconstruction_analysis/util/blat_map_filter_with_isoforms.pl\n",
            "Analysis/FL_reconstruction_analysis/util/blat_psl_to_align_summary_stats.pl\n",
            "Analysis/FL_reconstruction_analysis/util/blat_query_top_hit_extractor.pl\n",
            "Analysis/FL_reconstruction_analysis/util/blat_top_tier_genes.pl\n",
            "Analysis/GenomeViewPlugin/\n",
            "Analysis/GenomeViewPlugin/test.html\n",
            "Analysis/GenomeViewPlugin/test_session.html\n",
            "Analysis/GenomeViewPlugin/trinity_to_genomeview_html.pl\n",
            "Analysis/SuperTranscripts/\n",
            "Analysis/SuperTranscripts/README.md\n",
            "Analysis/SuperTranscripts/Trinity_gene_splice_modeler.py\n",
            "Analysis/SuperTranscripts/extract_supertranscript_from_reference.py\n",
            "Analysis/SuperTranscripts/AllelicVariants/\n",
            "Analysis/SuperTranscripts/AllelicVariants/VCF_to_annotated_SNP_report.pl\n",
            "Analysis/SuperTranscripts/AllelicVariants/run_variant_calling.py\n",
            "Analysis/SuperTranscripts/AllelicVariants/util/\n",
            "Analysis/SuperTranscripts/AllelicVariants/util/clean_bam.pl\n",
            "Analysis/SuperTranscripts/DTU/\n",
            "Analysis/SuperTranscripts/DTU/README.md\n",
            "Analysis/SuperTranscripts/DTU/dexseq_wrapper.pl\n",
            "Analysis/SuperTranscripts/DTU/util/\n",
            "Analysis/SuperTranscripts/DTU/util/reformat_featureCounts.pl\n",
            "Analysis/SuperTranscripts/_misc/\n",
            "Analysis/SuperTranscripts/_misc/aln_before_after.pl\n",
            "Analysis/SuperTranscripts/pylib/\n",
            "Analysis/SuperTranscripts/pylib/Compact_graph_partial.py\n",
            "Analysis/SuperTranscripts/pylib/Compact_graph_pruner.py\n",
            "Analysis/SuperTranscripts/pylib/Compact_graph_whole.py\n",
            "Analysis/SuperTranscripts/pylib/DP_matrix.py\n",
            "Analysis/SuperTranscripts/pylib/Gene_splice_modeler.py\n",
            "Analysis/SuperTranscripts/pylib/GraphCycleException.py\n",
            "Analysis/SuperTranscripts/pylib/Node_alignment.py\n",
            "Analysis/SuperTranscripts/pylib/Node_path.py\n",
            "Analysis/SuperTranscripts/pylib/Splice_model_refiner.py\n",
            "Analysis/SuperTranscripts/pylib/TGLOBALS.py\n",
            "Analysis/SuperTranscripts/pylib/TGraph.py\n",
            "Analysis/SuperTranscripts/pylib/TNode.py\n",
            "Analysis/SuperTranscripts/pylib/Topological_sort.py\n",
            "Analysis/SuperTranscripts/pylib/Trinity_fasta_parser.py\n",
            "Analysis/SuperTranscripts/pylib/Trinity_util.py\n",
            "Analysis/SuperTranscripts/pylib/__init__.py\n",
            "Butterfly/\n",
            "Butterfly/Butterfly.jar\n",
            "Butterfly/Butterfly/\n",
            "Butterfly/Butterfly/Butterfly.jar\n",
            "Butterfly/Butterfly/README.md\n",
            "Butterfly/Butterfly/src/\n",
            "Butterfly/Butterfly/src/README\n",
            "Butterfly/Butterfly/src/_t_seqA.fa\n",
            "Butterfly/Butterfly/src/_t_seqB.fa\n",
            "Butterfly/Butterfly/src/build.xml\n",
            "Butterfly/Butterfly/src/runExample.sh\n",
            "Butterfly/Butterfly/src/run_NWalign.sh\n",
            "Butterfly/Butterfly/src/run_butterfly_nonJar.sh\n",
            "Butterfly/Butterfly/src/lib/\n",
            "Butterfly/Butterfly/src/lib/Jaligner.jar\n",
            "Butterfly/Butterfly/src/lib/collections-generic-4.01.jar\n",
            "Butterfly/Butterfly/src/lib/java-getopt-1.0.13.jar\n",
            "Butterfly/Butterfly/src/lib/jung-algorithms-2.0.1.jar\n",
            "Butterfly/Butterfly/src/lib/jung-api-2.0.1.jar\n",
            "Butterfly/Butterfly/src/lib/jung-graph-impl-2.0.1.jar\n",
            "Butterfly/Butterfly/src/sample_data/\n",
            "Butterfly/Butterfly/src/sample_data/c1.graph.out\n",
            "Butterfly/Butterfly/src/sample_data/c1.graph.reads\n",
            "Butterfly/Butterfly/src/src/\n",
            "Butterfly/Butterfly/src/src/AlignmentStats.java\n",
            "Butterfly/Butterfly/src/src/BFLY_GLOBALS.java\n",
            "Butterfly/Butterfly/src/src/BipartiteMatching.java\n",
            "Butterfly/Butterfly/src/src/DFS.java\n",
            "Butterfly/Butterfly/src/src/GraphPath.java\n",
            "Butterfly/Butterfly/src/src/IsoformExpressionLearning.java\n",
            "Butterfly/Butterfly/src/src/ListComparator.java\n",
            "Butterfly/Butterfly/src/src/LocInGraph.java\n",
            "Butterfly/Butterfly/src/src/My_DFS.java\n",
            "Butterfly/Butterfly/src/src/NWalign.java\n",
            "Butterfly/Butterfly/src/src/NumPathNodeLoopsEdgeComparator.java\n",
            "Butterfly/Butterfly/src/src/PairPath.java\n",
            "Butterfly/Butterfly/src/src/PairPathWOrig.java\n",
            "Butterfly/Butterfly/src/src/PasaVertex.java\n",
            "Butterfly/Butterfly/src/src/Path.java\n",
            "Butterfly/Butterfly/src/src/PathEndSeqVertexFinishTimeComparator.java\n",
            "Butterfly/Butterfly/src/src/PathExpressionComparator.java\n",
            "Butterfly/Butterfly/src/src/PathOverlap.java\n",
            "Butterfly/Butterfly/src/src/PathReadSupportComparator.java\n",
            "Butterfly/Butterfly/src/src/PathWithOrig.java\n",
            "Butterfly/Butterfly/src/src/Path_n_MM_count.java\n",
            "Butterfly/Butterfly/src/src/Read.java\n",
            "Butterfly/Butterfly/src/src/ScoredPath.java\n",
            "Butterfly/Butterfly/src/src/SeqComparator.java\n",
            "Butterfly/Butterfly/src/src/SeqVertex.java\n",
            "Butterfly/Butterfly/src/src/SeqVertexDepthComparator.java\n",
            "Butterfly/Butterfly/src/src/SeqVertexFinishTimeComparator.java\n",
            "Butterfly/Butterfly/src/src/SeqVertexNodeDepthComparator.java\n",
            "Butterfly/Butterfly/src/src/SimpleEdge.java\n",
            "Butterfly/Butterfly/src/src/SimpleEdgeComparator.java\n",
            "Butterfly/Butterfly/src/src/SimplePathNodeEdge.java\n",
            "Butterfly/Butterfly/src/src/TopologicalSort.java\n",
            "Butterfly/Butterfly/src/src/TransAssembly_allProbPaths.java\n",
            "Butterfly/Butterfly/src/src/UniquePathContentComparator.java\n",
            "Butterfly/Butterfly/src/src/Visitor.java\n",
            "Butterfly/Butterfly/src/src/ZipperAlignment.java\n",
            "Butterfly/Butterfly/src/src/numLoopsEdgeComparator.java\n",
            "Butterfly/Butterfly/src/src/edu/\n",
            "Butterfly/Butterfly/src/src/edu/uci/\n",
            "Butterfly/Butterfly/src/src/edu/uci/ics/\n",
            "Butterfly/Butterfly/src/src/edu/uci/ics/jung/\n",
            "Butterfly/Butterfly/src/src/edu/uci/ics/jung/algorithms/\n",
            "Butterfly/Butterfly/src/src/edu/uci/ics/jung/algorithms/shortestpath/\n",
            "Butterfly/Butterfly/src/src/edu/uci/ics/jung/algorithms/shortestpath/DijkstraDistanceWoVer.java\n",
            "Butterfly/Butterfly/src/src/jaligner/\n",
            "Butterfly/Butterfly/src/src/jaligner/Alignment.java\n",
            "Butterfly/Butterfly/src/src/jaligner/Cell.java\n",
            "Butterfly/Butterfly/src/src/jaligner/Directions.java\n",
            "Butterfly/Butterfly/src/src/jaligner/Markups.java\n",
            "Butterfly/Butterfly/src/src/jaligner/NeedlemanWunsch.java\n",
            "Butterfly/Butterfly/src/src/jaligner/NeedlemanWunschGotoh.java\n",
            "Butterfly/Butterfly/src/src/jaligner/NeedlemanWunschGotohBanded.java\n",
            "Butterfly/Butterfly/src/src/jaligner/Sequence.java\n",
            "Butterfly/Butterfly/src/src/jaligner/SmithWatermanGotoh.java\n",
            "Butterfly/Butterfly/src/src/jaligner/TracebackNode.java\n",
            "Butterfly/Butterfly/src/src/jaligner/package.html\n",
            "Butterfly/Butterfly/src/src/jaligner/example/\n",
            "Butterfly/Butterfly/src/src/jaligner/example/NeedlemanWunschExample.java\n",
            "Butterfly/Butterfly/src/src/jaligner/example/NeedlemanWunschGotohExample.java\n",
            "Butterfly/Butterfly/src/src/jaligner/example/SmithWatermanGotohExample.java\n",
            "Butterfly/Butterfly/src/src/jaligner/example/package.html\n",
            "Butterfly/Butterfly/src/src/jaligner/example/sequences/\n",
            "Butterfly/Butterfly/src/src/jaligner/example/sequences/p53_human.fasta\n",
            "Butterfly/Butterfly/src/src/jaligner/example/sequences/p53_mouse.fasta\n",
            "Butterfly/Butterfly/src/src/jaligner/formats/\n",
            "Butterfly/Butterfly/src/src/jaligner/formats/CLUSTAL.java\n",
            "Butterfly/Butterfly/src/src/jaligner/formats/FASTA.java\n",
            "Butterfly/Butterfly/src/src/jaligner/formats/Format.java\n",
            "Butterfly/Butterfly/src/src/jaligner/formats/FormatFactory.java\n",
            "Butterfly/Butterfly/src/src/jaligner/formats/Pair.java\n",
            "Butterfly/Butterfly/src/src/jaligner/formats/package.html\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/MatricesCompartor.java\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/Matrix.java\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/MatrixGenerator.java\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/MatrixLoader.java\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/MatrixLoaderException.java\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/package.html\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM100\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM30\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM35\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM40\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM45\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM50\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM55\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM60\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM62\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM65\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM70\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM75\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM80\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM85\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUM90\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/BLOSUMN\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/DAYHOFF\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/EDNAFULL\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/GONNET\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/IDENTITY\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/MATCH\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM10\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM100\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM110\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM120\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM130\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM140\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM150\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM160\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM170\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM180\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM190\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM20\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM200\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM210\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM220\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM230\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM240\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM250\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM260\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM270\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM280\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM290\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM30\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM300\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM310\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM320\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM330\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM340\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM350\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM360\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM370\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM380\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM390\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM40\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM400\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM410\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM420\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM430\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM440\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM450\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM460\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM470\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM480\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM490\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM50\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM500\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM60\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM70\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM80\n",
            "Butterfly/Butterfly/src/src/jaligner/matrix/matrices/PAM90\n",
            "Butterfly/Butterfly/src/src/jaligner/test/\n",
            "Butterfly/Butterfly/src/src/jaligner/test/NeedlemanWunschGotohTester.java\n",
            "Butterfly/Butterfly/src/src/jaligner/test/NeedlemanWunschTester.java\n",
            "Butterfly/Butterfly/src/src/jaligner/test/RandomSequenceGenerator.java\n",
            "Butterfly/Butterfly/src/src/jaligner/test/SmithWatermanGotohTester.java\n",
            "Butterfly/Butterfly/src/src/jaligner/test/package.html\n",
            "Butterfly/Butterfly/src/src/jaligner/ui/\n",
            "Butterfly/Butterfly/src/src/jaligner/ui/filechooser/\n",
            "Butterfly/Butterfly/src/src/jaligner/ui/filechooser/NamedInputStream.java\n",
            "Butterfly/Butterfly/src/src/jaligner/util/\n",
            "Butterfly/Butterfly/src/src/jaligner/util/Commons.java\n",
            "Butterfly/Butterfly/src/src/jaligner/util/SequenceParser.java\n",
            "Butterfly/Butterfly/src/src/jaligner/util/SequenceParserException.java\n",
            "Butterfly/Butterfly/src/src/jaligner/util/package.html\n",
            "Chrysalis/\n",
            "Chrysalis/CMakeLists.txt\n",
            "Chrysalis/Makefile\n",
            "Chrysalis/README.md\n",
            "Chrysalis/chrysalis.notes\n",
            "Chrysalis/make.log\n",
            "Chrysalis/aligns/\n",
            "Chrysalis/aligns/KmerAlignCore.cc\n",
            "Chrysalis/aligns/KmerAlignCore.h\n",
            "Chrysalis/analysis/\n",
            "Chrysalis/analysis/AACodons.cc\n",
            "Chrysalis/analysis/AACodons.h\n",
            "Chrysalis/analysis/BreakTransByPairs.cc\n",
            "Chrysalis/analysis/BubbleUpClustering.cc\n",
            "Chrysalis/analysis/Chrysalis.cc\n",
            "Chrysalis/analysis/CompMgr.h\n",
            "Chrysalis/analysis/CreateIwormFastaBundle.cc\n",
            "Chrysalis/analysis/DNAVector.cc\n",
            "Chrysalis/analysis/DNAVector.h\n",
            "Chrysalis/analysis/DeBruijnGraph.cc\n",
            "Chrysalis/analysis/DeBruijnGraph.h\n",
            "Chrysalis/analysis/GraphFromFasta.cc\n",
            "Chrysalis/analysis/IsoformAugment.cc\n",
            "Chrysalis/analysis/JoinTransByPairs.cc\n",
            "Chrysalis/analysis/KmerTable.cc\n",
            "Chrysalis/analysis/KmerTable.h\n",
            "Chrysalis/analysis/NonRedKmerTable.cc\n",
            "Chrysalis/analysis/NonRedKmerTable.h\n",
            "Chrysalis/analysis/Pool.h\n",
            "Chrysalis/analysis/QuantifyGraph.cc\n",
            "Chrysalis/analysis/ReadsToComponents.pl\n",
            "Chrysalis/analysis/ReadsToTranscripts.cc\n",
            "Chrysalis/analysis/RunButterfly.cc\n",
            "Chrysalis/analysis/TranscriptomeFromVaryK.cc\n",
            "Chrysalis/analysis/TranscriptomeGraph.cc\n",
            "Chrysalis/analysis/TranscriptomeGraph.h\n",
            "Chrysalis/analysis/sequenceUtil.cc\n",
            "Chrysalis/analysis/sequenceUtil.h\n",
            "Chrysalis/analysis/stacktrace.cc\n",
            "Chrysalis/analysis/stacktrace.h\n",
            "Chrysalis/analysis/string_util.h\n",
            "Chrysalis/base/\n",
            "Chrysalis/base/CommandLineParser.h\n",
            "Chrysalis/base/ErrorHandling.cc\n",
            "Chrysalis/base/ErrorHandling.h\n",
            "Chrysalis/base/FileParser.cc\n",
            "Chrysalis/base/FileParser.h\n",
            "Chrysalis/base/SVector.h\n",
            "Chrysalis/base/StringUtil.cc\n",
            "Chrysalis/base/StringUtil.h\n",
            "Chrysalis/bin/\n",
            "Chrysalis/bin/BubbleUpClustering\n",
            "Chrysalis/bin/Chrysalis\n",
            "Chrysalis/bin/CreateIwormFastaBundle\n",
            "Chrysalis/bin/GraphFromFasta\n",
            "Chrysalis/bin/QuantifyGraph\n",
            "Chrysalis/bin/ReadsToTranscripts\n",
            "Chrysalis/build/\n",
            "Chrysalis/build/BubbleUpClustering\n",
            "Chrysalis/build/CMakeCache.txt\n",
            "Chrysalis/build/Chrysalis\n",
            "Chrysalis/build/CreateIwormFastaBundle\n",
            "Chrysalis/build/GraphFromFasta\n",
            "Chrysalis/build/Makefile\n",
            "Chrysalis/build/QuantifyGraph\n",
            "Chrysalis/build/ReadsToTranscripts\n",
            "Chrysalis/build/cmake_install.cmake\n",
            "Chrysalis/build/install_manifest.txt\n",
            "Chrysalis/build/CMakeFiles/\n",
            "Chrysalis/build/CMakeFiles/CMakeDirectoryInformation.cmake\n",
            "Chrysalis/build/CMakeFiles/CMakeOutput.log\n",
            "Chrysalis/build/CMakeFiles/Makefile.cmake\n",
            "Chrysalis/build/CMakeFiles/Makefile2\n",
            "Chrysalis/build/CMakeFiles/TargetDirectories.txt\n",
            "Chrysalis/build/CMakeFiles/cmake.check_cache\n",
            "Chrysalis/build/CMakeFiles/feature_tests.bin\n",
            "Chrysalis/build/CMakeFiles/feature_tests.c\n",
            "Chrysalis/build/CMakeFiles/feature_tests.cxx\n",
            "Chrysalis/build/CMakeFiles/progress.marks\n",
            "Chrysalis/build/CMakeFiles/3.12.0/\n",
            "Chrysalis/build/CMakeFiles/3.12.0/CMakeCCompiler.cmake\n",
            "Chrysalis/build/CMakeFiles/3.12.0/CMakeCXXCompiler.cmake\n",
            "Chrysalis/build/CMakeFiles/3.12.0/CMakeDetermineCompilerABI_C.bin\n",
            "Chrysalis/build/CMakeFiles/3.12.0/CMakeDetermineCompilerABI_CXX.bin\n",
            "Chrysalis/build/CMakeFiles/3.12.0/CMakeSystem.cmake\n",
            "Chrysalis/build/CMakeFiles/3.12.0/CompilerIdC/\n",
            "Chrysalis/build/CMakeFiles/3.12.0/CompilerIdC/CMakeCCompilerId.c\n",
            "Chrysalis/build/CMakeFiles/3.12.0/CompilerIdC/a.out\n",
            "Chrysalis/build/CMakeFiles/3.12.0/CompilerIdC/tmp/\n",
            "Chrysalis/build/CMakeFiles/3.12.0/CompilerIdCXX/\n",
            "Chrysalis/build/CMakeFiles/3.12.0/CompilerIdCXX/CMakeCXXCompilerId.cpp\n",
            "Chrysalis/build/CMakeFiles/3.12.0/CompilerIdCXX/a.out\n",
            "Chrysalis/build/CMakeFiles/3.12.0/CompilerIdCXX/tmp/\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/CXX.includecache\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/DependInfo.cmake\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/build.make\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/cmake_clean.cmake\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/depend.internal\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/depend.make\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/flags.make\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/link.txt\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/progress.make\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/aligns/\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/aligns/KmerAlignCore.cc.o\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/analysis/\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/analysis/AACodons.cc.o\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/analysis/BubbleUpClustering.cc.o\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/analysis/DNAVector.cc.o\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/analysis/DeBruijnGraph.cc.o\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/analysis/KmerTable.cc.o\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/analysis/NonRedKmerTable.cc.o\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/analysis/sequenceUtil.cc.o\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/analysis/stacktrace.cc.o\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/base/\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/base/ErrorHandling.cc.o\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/base/FileParser.cc.o\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/base/StringUtil.cc.o\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/util/\n",
            "Chrysalis/build/CMakeFiles/BubbleUpClustering.dir/util/mutil.cc.o\n",
            "Chrysalis/build/CMakeFiles/CMakeTmp/\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/CXX.includecache\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/DependInfo.cmake\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/build.make\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/cmake_clean.cmake\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/depend.internal\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/depend.make\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/flags.make\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/link.txt\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/progress.make\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/aligns/\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/aligns/KmerAlignCore.cc.o\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/analysis/\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/analysis/AACodons.cc.o\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/analysis/Chrysalis.cc.o\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/analysis/DNAVector.cc.o\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/analysis/TranscriptomeGraph.cc.o\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/base/\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/base/ErrorHandling.cc.o\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/base/FileParser.cc.o\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/base/StringUtil.cc.o\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/util/\n",
            "Chrysalis/build/CMakeFiles/Chrysalis.dir/util/mutil.cc.o\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/CXX.includecache\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/DependInfo.cmake\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/build.make\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/cmake_clean.cmake\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/depend.internal\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/depend.make\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/flags.make\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/link.txt\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/progress.make\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/analysis/\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/analysis/AACodons.cc.o\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/analysis/CreateIwormFastaBundle.cc.o\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/analysis/DNAVector.cc.o\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/base/\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/base/ErrorHandling.cc.o\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/base/FileParser.cc.o\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/base/StringUtil.cc.o\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/util/\n",
            "Chrysalis/build/CMakeFiles/CreateIwormFastaBundle.dir/util/mutil.cc.o\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/CXX.includecache\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/DependInfo.cmake\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/build.make\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/cmake_clean.cmake\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/depend.internal\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/depend.make\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/flags.make\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/link.txt\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/progress.make\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/aligns/\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/aligns/KmerAlignCore.cc.o\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/analysis/\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/analysis/AACodons.cc.o\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/analysis/DNAVector.cc.o\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/analysis/DeBruijnGraph.cc.o\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/analysis/GraphFromFasta.cc.o\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/analysis/KmerTable.cc.o\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/analysis/NonRedKmerTable.cc.o\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/analysis/sequenceUtil.cc.o\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/analysis/stacktrace.cc.o\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/base/\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/base/ErrorHandling.cc.o\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/base/FileParser.cc.o\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/base/StringUtil.cc.o\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/util/\n",
            "Chrysalis/build/CMakeFiles/GraphFromFasta.dir/util/mutil.cc.o\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/CXX.includecache\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/DependInfo.cmake\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/build.make\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/cmake_clean.cmake\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/depend.internal\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/depend.make\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/flags.make\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/link.txt\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/progress.make\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/aligns/\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/aligns/KmerAlignCore.cc.o\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/analysis/\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/analysis/AACodons.cc.o\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/analysis/DNAVector.cc.o\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/analysis/KmerTable.cc.o\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/analysis/NonRedKmerTable.cc.o\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/analysis/QuantifyGraph.cc.o\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/analysis/sequenceUtil.cc.o\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/analysis/stacktrace.cc.o\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/base/\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/base/ErrorHandling.cc.o\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/base/FileParser.cc.o\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/base/StringUtil.cc.o\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/util/\n",
            "Chrysalis/build/CMakeFiles/QuantifyGraph.dir/util/mutil.cc.o\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/CXX.includecache\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/DependInfo.cmake\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/build.make\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/cmake_clean.cmake\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/depend.internal\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/depend.make\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/flags.make\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/link.txt\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/progress.make\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/analysis/\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/analysis/AACodons.cc.o\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/analysis/DNAVector.cc.o\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/analysis/NonRedKmerTable.cc.o\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/analysis/ReadsToTranscripts.cc.o\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/analysis/sequenceUtil.cc.o\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/analysis/stacktrace.cc.o\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/base/\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/base/ErrorHandling.cc.o\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/base/FileParser.cc.o\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/base/StringUtil.cc.o\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/util/\n",
            "Chrysalis/build/CMakeFiles/ReadsToTranscripts.dir/util/mutil.cc.o\n",
            "Chrysalis/system/\n",
            "Chrysalis/system/BigFileDefines.h\n",
            "Chrysalis/util/\n",
            "Chrysalis/util/checkLock.c\n",
            "Chrysalis/util/mutil.cc\n",
            "Chrysalis/util/mutil.h\n",
            "Docker/\n",
            "Docker/Dockerfile\n",
            "Docker/README.md\n",
            "Docker/VERSION.txt\n",
            "Docker/build_docker.sh\n",
            "Docker/cleanMe.sh\n",
            "Docker/make_simg.sh\n",
            "Docker/notes.txt\n",
            "Docker/push_docker.sh\n",
            "Docker/run_ext_test_docker.sh\n",
            "Docker/run_ext_test_singularity.sh\n",
            "Docker/run_test_docker.sh\n",
            "Docker/test_data/\n",
            "Docker/test_data/reads.left.fq.gz\n",
            "Docker/test_data/reads.right.fq.gz\n",
            "Inchworm/\n",
            "Inchworm/CMakeLists.txt\n",
            "Inchworm/LICENSE\n",
            "Inchworm/Makefile\n",
            "Inchworm/README\n",
            "Inchworm/bin/\n",
            "Inchworm/bin/FastaToDeBruijn\n",
            "Inchworm/bin/fastaToKmerCoverageStats\n",
            "Inchworm/bin/inchworm\n",
            "Inchworm/build/\n",
            "Inchworm/build/CMakeCache.txt\n",
            "Inchworm/build/FastaToDeBruijn\n",
            "Inchworm/build/Makefile\n",
            "Inchworm/build/cmake_install.cmake\n",
            "Inchworm/build/fastaToKmerCoverageStats\n",
            "Inchworm/build/inchworm\n",
            "Inchworm/build/install_manifest.txt\n",
            "Inchworm/build/CMakeFiles/\n",
            "Inchworm/build/CMakeFiles/CMakeDirectoryInformation.cmake\n",
            "Inchworm/build/CMakeFiles/CMakeOutput.log\n",
            "Inchworm/build/CMakeFiles/Makefile.cmake\n",
            "Inchworm/build/CMakeFiles/Makefile2\n",
            "Inchworm/build/CMakeFiles/TargetDirectories.txt\n",
            "Inchworm/build/CMakeFiles/cmake.check_cache\n",
            "Inchworm/build/CMakeFiles/feature_tests.bin\n",
            "Inchworm/build/CMakeFiles/feature_tests.c\n",
            "Inchworm/build/CMakeFiles/feature_tests.cxx\n",
            "Inchworm/build/CMakeFiles/progress.marks\n",
            "Inchworm/build/CMakeFiles/3.12.0/\n",
            "Inchworm/build/CMakeFiles/3.12.0/CMakeCCompiler.cmake\n",
            "Inchworm/build/CMakeFiles/3.12.0/CMakeCXXCompiler.cmake\n",
            "Inchworm/build/CMakeFiles/3.12.0/CMakeDetermineCompilerABI_C.bin\n",
            "Inchworm/build/CMakeFiles/3.12.0/CMakeDetermineCompilerABI_CXX.bin\n",
            "Inchworm/build/CMakeFiles/3.12.0/CMakeSystem.cmake\n",
            "Inchworm/build/CMakeFiles/3.12.0/CompilerIdC/\n",
            "Inchworm/build/CMakeFiles/3.12.0/CompilerIdC/CMakeCCompilerId.c\n",
            "Inchworm/build/CMakeFiles/3.12.0/CompilerIdC/a.out\n",
            "Inchworm/build/CMakeFiles/3.12.0/CompilerIdC/tmp/\n",
            "Inchworm/build/CMakeFiles/3.12.0/CompilerIdCXX/\n",
            "Inchworm/build/CMakeFiles/3.12.0/CompilerIdCXX/CMakeCXXCompilerId.cpp\n",
            "Inchworm/build/CMakeFiles/3.12.0/CompilerIdCXX/a.out\n",
            "Inchworm/build/CMakeFiles/3.12.0/CompilerIdCXX/tmp/\n",
            "Inchworm/build/CMakeFiles/CMakeTmp/\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/CXX.includecache\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/DependInfo.cmake\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/build.make\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/cmake_clean.cmake\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/depend.internal\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/depend.make\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/flags.make\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/link.txt\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/progress.make\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/src/\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/src/DeBruijnGraph.cpp.o\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/src/FastaToDeBruijn.cpp.o\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/src/Fasta_entry.cpp.o\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/src/Fasta_reader.cpp.o\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/src/argProcessor.cpp.o\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/src/sequenceUtil.cpp.o\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/src/stacktrace.cpp.o\n",
            "Inchworm/build/CMakeFiles/FastaToDeBruijn.dir/src/string_util.cpp.o\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/CXX.includecache\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/DependInfo.cmake\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/build.make\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/cmake_clean.cmake\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/depend.internal\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/depend.make\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/flags.make\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/link.txt\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/progress.make\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/src/\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/src/Fasta_entry.cpp.o\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/src/Fasta_reader.cpp.o\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/src/KmerCounter.cpp.o\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/src/argProcessor.cpp.o\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/src/fastaToKmerCoverageStats.cpp.o\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/src/sequenceUtil.cpp.o\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/src/stacktrace.cpp.o\n",
            "Inchworm/build/CMakeFiles/fastaToKmerCoverageStats.dir/src/string_util.cpp.o\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/CXX.includecache\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/DependInfo.cmake\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/build.make\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/cmake_clean.cmake\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/depend.internal\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/depend.make\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/flags.make\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/link.txt\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/progress.make\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/src/\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/src/Fasta_entry.cpp.o\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/src/Fasta_reader.cpp.o\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/src/IRKE.cpp.o\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/src/IRKE_run.cpp.o\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/src/KmerCounter.cpp.o\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/src/argProcessor.cpp.o\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/src/sequenceUtil.cpp.o\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/src/stacktrace.cpp.o\n",
            "Inchworm/build/CMakeFiles/inchworm.dir/src/string_util.cpp.o\n",
            "Inchworm/src/\n",
            "Inchworm/src/Cigar.cpp\n",
            "Inchworm/src/Cigar.hpp\n",
            "Inchworm/src/DeBruijnGraph.cpp\n",
            "Inchworm/src/DeBruijnGraph.hpp\n",
            "Inchworm/src/FastaToDeBruijn\n",
            "Inchworm/src/FastaToDeBruijn.cpp\n",
            "Inchworm/src/Fasta_entry.cpp\n",
            "Inchworm/src/Fasta_entry.hpp\n",
            "Inchworm/src/Fasta_reader.cpp\n",
            "Inchworm/src/Fasta_reader.hpp\n",
            "Inchworm/src/IRKE.cpp\n",
            "Inchworm/src/IRKE.hpp\n",
            "Inchworm/src/IRKE_run.cpp\n",
            "Inchworm/src/KmerCounter.cpp\n",
            "Inchworm/src/KmerCounter.hpp\n",
            "Inchworm/src/Ktree.cpp\n",
            "Inchworm/src/Ktree.hpp\n",
            "Inchworm/src/KtreeNode.cpp\n",
            "Inchworm/src/KtreeNode.hpp\n",
            "Inchworm/src/MaskedKmerCounter.cpp\n",
            "Inchworm/src/MaskedKmerCounter.hpp\n",
            "Inchworm/src/SAM_entry.cpp\n",
            "Inchworm/src/SAM_entry.hpp\n",
            "Inchworm/src/SAM_reader.cpp\n",
            "Inchworm/src/SAM_reader.hpp\n",
            "Inchworm/src/SNP_stomper\n",
            "Inchworm/src/SNP_stomper.cpp\n",
            "Inchworm/src/argProcessor.cpp\n",
            "Inchworm/src/argProcessor.hpp\n",
            "Inchworm/src/cigar_tweaker\n",
            "Inchworm/src/cigar_tweaker.cpp\n",
            "Inchworm/src/common.hpp\n",
            "Inchworm/src/fastaToKmerCoverageStats\n",
            "Inchworm/src/fastaToKmerCoverageStats.cpp\n",
            "Inchworm/src/inchworm\n",
            "Inchworm/src/irke_common.hpp\n",
            "Inchworm/src/prev.Makefile.am\n",
            "Inchworm/src/pull_reads_with_kmers\n",
            "Inchworm/src/pull_reads_with_kmers.cpp\n",
            "Inchworm/src/sequenceUtil.cpp\n",
            "Inchworm/src/sequenceUtil.hpp\n",
            "Inchworm/src/stacktrace.cpp\n",
            "Inchworm/src/stacktrace.hpp\n",
            "Inchworm/src/string_util.cpp\n",
            "Inchworm/src/string_util.hpp\n",
            "Inchworm/src/mpi_deprecated/\n",
            "Inchworm/src/mpi_deprecated/MPI_InchwormMakefile\n",
            "Inchworm/src/mpi_deprecated/MPI_cray.Makefile\n",
            "Inchworm/src/mpi_deprecated/MPIinchworm.cpp\n",
            "PerlLib/\n",
            "PerlLib/Ascii_genome_illustrator.pm\n",
            "PerlLib/BED_utils.pm\n",
            "PerlLib/BHStats.pm\n",
            "PerlLib/CIGAR.pm\n",
            "PerlLib/CMD_processor.pm\n",
            "PerlLib/COMMON.pm\n",
            "PerlLib/ColorGradient.pm\n",
            "PerlLib/DelimParser.pm\n",
            "PerlLib/EM.pm\n",
            "PerlLib/Exons_to_geneobj.pm\n",
            "PerlLib/Fasta_reader.pm\n",
            "PerlLib/Fasta_retriever.pm\n",
            "PerlLib/Fastq_reader.pm\n",
            "PerlLib/GFF3_alignment_utils.pm\n",
            "PerlLib/GFF3_utils.pm\n",
            "PerlLib/GFF_maker.pm\n",
            "PerlLib/GTF.pm\n",
            "PerlLib/GTF_utils.pm\n",
            "PerlLib/Gene_obj.pm\n",
            "PerlLib/Gene_obj_indexer.pm\n",
            "PerlLib/Ktree.pm\n",
            "PerlLib/Longest_orf.pm\n",
            "PerlLib/Nuc_translator.pm\n",
            "PerlLib/Overlap_info.pm\n",
            "PerlLib/Overlap_piler.pm\n",
            "PerlLib/PSL_parser.pm\n",
            "PerlLib/Pipeliner.pm\n",
            "PerlLib/Process_cmd.pm\n",
            "PerlLib/SAM_entry.pm\n",
            "PerlLib/SAM_reader.pm\n",
            "PerlLib/SingleLinkageClusterer.pm\n",
            "PerlLib/Thread_helper.pm\n",
            "PerlLib/TiedHash.pm\n",
            "PerlLib/VCF_parser.pm\n",
            "PerlLib/WigParser.pm\n",
            "PerlLib/overlapping_nucs.ph\n",
            "PerlLib/test_Fasta_retriever.pl\n",
            "PerlLib/test_htc_gridrunner_LSF.pl\n",
            "PerlLib/test_htc_gridrunner_SGE.pl\n",
            "PerlLib/CDNA/\n",
            "PerlLib/CDNA/Alignment_segment.pm\n",
            "PerlLib/CDNA/Alternative_splice_comparer.pm\n",
            "PerlLib/CDNA/CDNA_alignment.pm\n",
            "PerlLib/CDNA/CDNA_stitcher.pm\n",
            "PerlLib/CDNA/Gene_obj_alignment_assembler.pm\n",
            "PerlLib/CDNA/Genome_based_cDNA_assembler.pm\n",
            "PerlLib/CDNA/Genome_based_cDNA_graph_assembler.pm\n",
            "PerlLib/CDNA/Overlap_assembler.pm\n",
            "PerlLib/CDNA/PASA_alignment_assembler.pm\n",
            "PerlLib/CDNA/Splice_graph_assembler.pm\n",
            "PerlLib/CanvasXpress/\n",
            "PerlLib/CanvasXpress/Heatmap.pm\n",
            "PerlLib/KmerGraphLib/\n",
            "PerlLib/KmerGraphLib/AlignGraph.pm\n",
            "PerlLib/KmerGraphLib/AlignNode.pm\n",
            "PerlLib/KmerGraphLib/GenericGraph.pm\n",
            "PerlLib/KmerGraphLib/GenericNode.pm\n",
            "PerlLib/KmerGraphLib/KmerGraph.pm\n",
            "PerlLib/KmerGraphLib/KmerNode.pm\n",
            "PerlLib/KmerGraphLib/ReadCoverageGraph.pm\n",
            "PerlLib/KmerGraphLib/ReadCoverageNode.pm\n",
            "PerlLib/KmerGraphLib/ReadManager.pm\n",
            "PerlLib/KmerGraphLib/ReadTracker.pm\n",
            "PerlLib/KmerGraphLib/SAM_entry.pm\n",
            "PerlLib/KmerGraphLib/SAM_reader.pm\n",
            "PerlLib/KmerGraphLib/SAM_to_AlignGraph.pm\n",
            "PerlLib/KmerGraphLib/StringGraph.pm\n",
            "PerlLib/KmerGraphLib/StringNode.pm\n",
            "PerlLib/Simulate/\n",
            "PerlLib/Simulate/Uniform_Read_Generator.pm\n",
            "PyLib/\n",
            "PyLib/Pipeliner.py\n",
            "WDL/\n",
            "WDL/Dockerfile\n",
            "WDL/build.sh\n",
            "WDL/parse_commands.py\n",
            "WDL/trinity_abundance_estimates_to_matrix.wdl\n",
            "WDL/trinity_differential_expression.wdl\n",
            "WDL/trinityrnaseq.wdl\n",
            "WDL/minimap2/\n",
            "WDL/minimap2/Dockerfile\n",
            "WDL/minimap2/build.sh\n",
            "WDL/minimap2/minimap2_align_to_reference.wdl\n",
            "sample_data/\n",
            "sample_data/Makefile\n",
            "sample_data/test_Trinity_Assembly/\n",
            "sample_data/test_Trinity_Assembly/Makefile\n",
            "sample_data/test_Trinity_Assembly/README\n",
            "sample_data/test_Trinity_Assembly/cleanme.pl\n",
            "sample_data/test_Trinity_Assembly/longReads.fa\n",
            "sample_data/test_Trinity_Assembly/reads.left.fa.gz\n",
            "sample_data/test_Trinity_Assembly/reads.left.fq.gz\n",
            "sample_data/test_Trinity_Assembly/reads.right.fa.gz\n",
            "sample_data/test_Trinity_Assembly/reads.right.fq.gz\n",
            "sample_data/test_Trinity_Assembly/reads2.left.fq.gz\n",
            "sample_data/test_Trinity_Assembly/reads2.right.fq.gz\n",
            "sample_data/test_Trinity_Assembly/runMe.sh\n",
            "sample_data/test_Trinity_Assembly/samples.PE.txt\n",
            "sample_data/test_Trinity_Assembly/samples.SE.txt\n",
            "sample_data/test_Trinity_Assembly/test_FL.sh\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/align_reads_via_bowtie.sh\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/bowtie2.sam.aligned.bam\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/refSeqs.fa\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/refSeqs.fa.gene_trans_map\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex01/\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex01/ex01.IGV.png\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex01/ex01.reads.left.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex01/ex01.reads.right.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex01/ex01.refSeq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex02/\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex02/ex02.refSeq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex02/ex2.reads.left.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex02/ex2.reads.right.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex02/example2.png\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex02/region2.sam\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex02/region2.sam.adj\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex03/\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex03/ex03.refSeq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex03/ex3.reads.left.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex03/ex3.reads.right.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex03/example3.png\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex03/region3.sam\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex03/region3.sam.adj\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex04/\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex04/ex04.refSeq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex04/ex4.reads.left.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex04/ex4.reads.right.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex04/example4.png\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex04/region4.sam\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex04/region4.sam.adj\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/clean.left.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/clean.right.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/ex05.refSeq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/ex5.reads.left.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/ex5.reads.right.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/example5.png\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/isoform_pair.fasta\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/isoform_pair.fasta.Bfly.dot\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/isoform_pair.fasta.gmap.bed\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/isoform_pair.fasta.gmap.gff\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/region5.sam\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/region5.sam.adj\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/runMe.clean.sh\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex05/runMe.sh\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex06/\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex06/ex06.refSeq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex06/ex6.reads.left.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex06/ex6.reads.right.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex06/example6.png\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex06/region6.sam\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex06/region6.sam.adj\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex07/\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex07/ex07.refSeq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex07/ex7.reads.left.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex07/ex7.reads.right.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex07/example7.png\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex07/region7.sam\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex07/region7.sam.adj\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex08/\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex08/ex08.refSeq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex08/ex8.reads.left.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex08/ex8.reads.right.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex08/example8.png\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex08/region8.sam\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex08/region8.sam.adj\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex09/\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex09/ex09.refSeq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex09/ex9.reads.left.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex09/ex9.reads.right.fq\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex09/example9.png\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex09/region9.sam\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex09/region9.sam.adj\n",
            "sample_data/test_Trinity_Assembly/__indiv_ex_sample_derived/ex09/runMe.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe.diff_kmer_length.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe.noSeqtk.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_CuffFly.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_PasaFly.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_as_DS.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_bowtie_components.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_docker.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_full_cleanup.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_include_long_reads.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_no_cleanup.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_no_para_iworm.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_no_salmon.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_no_salmon_no_cleanup.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_no_symlinks.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_parallel_iworm_assembly.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_piecemeal.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_trin_complete.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_use_bowtie2.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_use_my_ramdisk.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_use_workdir.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_using_Grid_LSF.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_using_Grid_SGE.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_using_Grid_uger.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_wSuperReads.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_w_monitoring.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_with_qual_trimming.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_with_qual_trimming_and_normalization.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__runMe_with_qual_trimming_and_normalize_libs_separately.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__run_PE_samples_file.min_kmer_cov_3.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__run_PE_samples_file.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__run_SE_samples_file.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__run_abundance_estimation_include_antisense.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__test_runMe.singleFQ.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__test_runMe_NO_normalization.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__test_runMe_docker.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__test_runMe_include_normalization.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__test_runMe_lenient_path_extension.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__test_runMe_singularity.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/__test_runMe_with_jaccard_clip.sh\n",
            "sample_data/test_Trinity_Assembly/misc_run_tests/runMe_no_qualTrim.sh\n",
            "trinity-plugins/\n",
            "trinity-plugins/Makefile\n",
            "trinity-plugins/README\n",
            "trinity-plugins/Trimmomatic -> Trimmomatic-0.36\n",
            "trinity-plugins/BIN/\n",
            "trinity-plugins/BIN/ParaFly\n",
            "trinity-plugins/BIN/README\n",
            "trinity-plugins/BIN/seqtk-trinity\n",
            "trinity-plugins/COLLECTL/\n",
            "trinity-plugins/COLLECTL/collectl -> collectl-4.1.0\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0.src.tar.gz\n",
            "trinity-plugins/COLLECTL/examine_resource_usage_profiling.pl\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/ARTISTIC\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/COPYING\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/GPL\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/INSTALL\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/README\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/README-WINDOWS\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/RELEASE-collectl\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/UNINSTALL\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/client.pl\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/col2tlviz.pl\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/collectl\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/collectl.conf\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/colmux\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/envrules.std\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/formatit.ph\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/gexpr.ph\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/graphite.ph\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/hello.ph\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/lexpr.ph\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/misc.ph\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/proctree.ph\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/statsd.ph\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/vmstat.ph\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/vmsum.ph\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/vnet.ph\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/BuddyInfo.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/CPUs.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/ColmuxCPU.jpg\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/ColmuxLustre.jpg\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Data-brief.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Data-detail.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Data-verbose.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Data.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Disks.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Documentation.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Environmental.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Examples.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Exceptions.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Export.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/FAQ-collectl.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/FileNaming.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Ganglia.jpg\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Ganglia2.jpg\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Ganglia3.jpg\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Gexpr.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Graphite.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Hello.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/HiResTime.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Import.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Infiniband.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Inodes.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/InputFiles.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Interrupts.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Logging.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Lustre.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Matrix.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Memory.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Messages.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Misc.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Network.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/NfsInfo.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/OpenStack.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/OperationalModes.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/OutputFiles.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/OutputFormats.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Performance.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Playback.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Plotfiles.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Process.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/RunningAsAService.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/SlabInfo.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/SlowProc.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Socket.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Sockets.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Startup.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/TheMath.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Tutorial-Lustre.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/Tutorial.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/WhySummary.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/colmux.html\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/markseger,collectl@web.sourceforge.net\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/slub.jpg\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/docs/style.css\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/initd/\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/initd/collectl\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/initd/collectl-debian\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/initd/collectl-generic\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/initd/collectl-suse\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/man1/\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/man1/collectl.1\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/man1/colmux.1\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/service/\n",
            "trinity-plugins/COLLECTL/collectl-4.1.0/service/collectl.service\n",
            "trinity-plugins/COLLECTL/util/\n",
            "trinity-plugins/COLLECTL/util/collectl_dat_to_time_matrix.py\n",
            "trinity-plugins/COLLECTL/util/plot_time_vs_resource.Rscript\n",
            "trinity-plugins/DEXseq_util/\n",
            "trinity-plugins/DEXseq_util/README.txt\n",
            "trinity-plugins/DEXseq_util/dexseq_prepare_annotation.py\n",
            "trinity-plugins/ParaFly/\n",
            "trinity-plugins/ParaFly/LICENSE\n",
            "trinity-plugins/ParaFly/Makefile\n",
            "trinity-plugins/ParaFly/Makefile.am\n",
            "trinity-plugins/ParaFly/Makefile.in\n",
            "trinity-plugins/ParaFly/README.md\n",
            "trinity-plugins/ParaFly/aclocal.m4\n",
            "trinity-plugins/ParaFly/config.h\n",
            "trinity-plugins/ParaFly/config.h.in\n",
            "trinity-plugins/ParaFly/config.log\n",
            "trinity-plugins/ParaFly/config.status\n",
            "trinity-plugins/ParaFly/configure\n",
            "trinity-plugins/ParaFly/configure.ac\n",
            "trinity-plugins/ParaFly/depcomp\n",
            "trinity-plugins/ParaFly/install-sh\n",
            "trinity-plugins/ParaFly/missing\n",
            "trinity-plugins/ParaFly/stamp-h1\n",
            "trinity-plugins/ParaFly/bin/\n",
            "trinity-plugins/ParaFly/bin/ParaFly\n",
            "trinity-plugins/ParaFly/src/\n",
            "trinity-plugins/ParaFly/src/Makefile\n",
            "trinity-plugins/ParaFly/src/Makefile.am\n",
            "trinity-plugins/ParaFly/src/Makefile.in\n",
            "trinity-plugins/ParaFly/src/ParaFly\n",
            "trinity-plugins/ParaFly/src/ParaFly.cpp\n",
            "trinity-plugins/ParaFly/src/ParaFly.o\n",
            "trinity-plugins/ParaFly/src/argProcessor.cpp\n",
            "trinity-plugins/ParaFly/src/argProcessor.hpp\n",
            "trinity-plugins/ParaFly/src/argProcessor.o\n",
            "trinity-plugins/ParaFly/test/\n",
            "trinity-plugins/ParaFly/test/Makefile\n",
            "trinity-plugins/ParaFly/test/cmds.txt\n",
            "trinity-plugins/ParaFly/test/rand_exit.pl\n",
            "trinity-plugins/Trimmomatic-0.36/\n",
            "trinity-plugins/Trimmomatic-0.36/LICENSE\n",
            "trinity-plugins/Trimmomatic-0.36/trimmomatic-0.36.jar\n",
            "trinity-plugins/Trimmomatic-0.36/trimmomatic.jar -> trimmomatic-0.36.jar\n",
            "trinity-plugins/Trimmomatic-0.36/adapters/\n",
            "trinity-plugins/Trimmomatic-0.36/adapters/NexteraPE-PE.fa\n",
            "trinity-plugins/Trimmomatic-0.36/adapters/TruSeq2-PE.fa\n",
            "trinity-plugins/Trimmomatic-0.36/adapters/TruSeq2-SE.fa\n",
            "trinity-plugins/Trimmomatic-0.36/adapters/TruSeq3-PE-2.fa\n",
            "trinity-plugins/Trimmomatic-0.36/adapters/TruSeq3-PE.fa\n",
            "trinity-plugins/Trimmomatic-0.36/adapters/TruSeq3-SE.fa\n",
            "trinity-plugins/bamsifter/\n",
            "trinity-plugins/bamsifter/Makefile\n",
            "trinity-plugins/bamsifter/README.md\n",
            "trinity-plugins/bamsifter/bamsifter\n",
            "trinity-plugins/bamsifter/build_htslib.sh\n",
            "trinity-plugins/bamsifter/sift_bam_max_cov.cpp\n",
            "trinity-plugins/bamsifter/htslib/\n",
            "trinity-plugins/bamsifter/htslib/INSTALL\n",
            "trinity-plugins/bamsifter/htslib/LICENSE\n",
            "trinity-plugins/bamsifter/htslib/Makefile\n",
            "trinity-plugins/bamsifter/htslib/NEWS\n",
            "trinity-plugins/bamsifter/htslib/README\n",
            "trinity-plugins/bamsifter/htslib/README.large_positions.md\n",
            "trinity-plugins/bamsifter/htslib/README.md\n",
            "trinity-plugins/bamsifter/htslib/aclocal.m4\n",
            "trinity-plugins/bamsifter/htslib/bcf_sr_sort.c\n",
            "trinity-plugins/bamsifter/htslib/bcf_sr_sort.h\n",
            "trinity-plugins/bamsifter/htslib/bgzf.c\n",
            "trinity-plugins/bamsifter/htslib/bgzip.1\n",
            "trinity-plugins/bamsifter/htslib/bgzip.c\n",
            "trinity-plugins/bamsifter/htslib/config.mk.in\n",
            "trinity-plugins/bamsifter/htslib/configure.ac\n",
            "trinity-plugins/bamsifter/htslib/errmod.c\n",
            "trinity-plugins/bamsifter/htslib/faidx.5\n",
            "trinity-plugins/bamsifter/htslib/faidx.c\n",
            "trinity-plugins/bamsifter/htslib/header.c\n",
            "trinity-plugins/bamsifter/htslib/header.h\n",
            "trinity-plugins/bamsifter/htslib/hfile.c\n",
            "trinity-plugins/bamsifter/htslib/hfile_gcs.c\n",
            "trinity-plugins/bamsifter/htslib/hfile_internal.h\n",
            "trinity-plugins/bamsifter/htslib/hfile_libcurl.c\n",
            "trinity-plugins/bamsifter/htslib/hfile_net.c\n",
            "trinity-plugins/bamsifter/htslib/hfile_s3.c\n",
            "trinity-plugins/bamsifter/htslib/hfile_s3_write.c\n",
            "trinity-plugins/bamsifter/htslib/hts.c\n",
            "trinity-plugins/bamsifter/htslib/hts_internal.h\n",
            "trinity-plugins/bamsifter/htslib/hts_os.c\n",
            "trinity-plugins/bamsifter/htslib/htsfile.1\n",
            "trinity-plugins/bamsifter/htslib/htsfile.c\n",
            "trinity-plugins/bamsifter/htslib/htslib-s3-plugin.7\n",
            "trinity-plugins/bamsifter/htslib/htslib.mk\n",
            "trinity-plugins/bamsifter/htslib/htslib.pc.in\n",
            "trinity-plugins/bamsifter/htslib/htslib_vars.mk\n",
            "trinity-plugins/bamsifter/htslib/kfunc.c\n",
            "trinity-plugins/bamsifter/htslib/knetfile.c\n",
            "trinity-plugins/bamsifter/htslib/kstring.c\n",
            "trinity-plugins/bamsifter/htslib/md5.c\n",
            "trinity-plugins/bamsifter/htslib/multipart.c\n",
            "trinity-plugins/bamsifter/htslib/plugin.c\n",
            "trinity-plugins/bamsifter/htslib/probaln.c\n",
            "trinity-plugins/bamsifter/htslib/realn.c\n",
            "trinity-plugins/bamsifter/htslib/regidx.c\n",
            "trinity-plugins/bamsifter/htslib/region.c\n",
            "trinity-plugins/bamsifter/htslib/sam.5\n",
            "trinity-plugins/bamsifter/htslib/sam.c\n",
            "trinity-plugins/bamsifter/htslib/sam_internal.h\n",
            "trinity-plugins/bamsifter/htslib/synced_bcf_reader.c\n",
            "trinity-plugins/bamsifter/htslib/tabix.1\n",
            "trinity-plugins/bamsifter/htslib/tabix.c\n",
            "trinity-plugins/bamsifter/htslib/tbx.c\n",
            "trinity-plugins/bamsifter/htslib/textutils.c\n",
            "trinity-plugins/bamsifter/htslib/textutils_internal.h\n",
            "trinity-plugins/bamsifter/htslib/thread_pool.c\n",
            "trinity-plugins/bamsifter/htslib/thread_pool_internal.h\n",
            "trinity-plugins/bamsifter/htslib/vcf.5\n",
            "trinity-plugins/bamsifter/htslib/vcf.c\n",
            "trinity-plugins/bamsifter/htslib/vcf_sweep.c\n",
            "trinity-plugins/bamsifter/htslib/vcfutils.c\n",
            "trinity-plugins/bamsifter/htslib/version.sh\n",
            "trinity-plugins/bamsifter/htslib/build/\n",
            "trinity-plugins/bamsifter/htslib/cram/\n",
            "trinity-plugins/bamsifter/htslib/cram/cram.h\n",
            "trinity-plugins/bamsifter/htslib/cram/cram_codecs.c\n",
            "trinity-plugins/bamsifter/htslib/cram/cram_codecs.h\n",
            "trinity-plugins/bamsifter/htslib/cram/cram_decode.c\n",
            "trinity-plugins/bamsifter/htslib/cram/cram_decode.h\n",
            "trinity-plugins/bamsifter/htslib/cram/cram_encode.c\n",
            "trinity-plugins/bamsifter/htslib/cram/cram_encode.h\n",
            "trinity-plugins/bamsifter/htslib/cram/cram_external.c\n",
            "trinity-plugins/bamsifter/htslib/cram/cram_index.c\n",
            "trinity-plugins/bamsifter/htslib/cram/cram_index.h\n",
            "trinity-plugins/bamsifter/htslib/cram/cram_io.c\n",
            "trinity-plugins/bamsifter/htslib/cram/cram_io.h\n",
            "trinity-plugins/bamsifter/htslib/cram/cram_samtools.c\n",
            "trinity-plugins/bamsifter/htslib/cram/cram_samtools.h\n",
            "trinity-plugins/bamsifter/htslib/cram/cram_stats.c\n",
            "trinity-plugins/bamsifter/htslib/cram/cram_stats.h\n",
            "trinity-plugins/bamsifter/htslib/cram/cram_structs.h\n",
            "trinity-plugins/bamsifter/htslib/cram/mFILE.c\n",
            "trinity-plugins/bamsifter/htslib/cram/mFILE.h\n",
            "trinity-plugins/bamsifter/htslib/cram/misc.h\n",
            "trinity-plugins/bamsifter/htslib/cram/open_trace_file.c\n",
            "trinity-plugins/bamsifter/htslib/cram/open_trace_file.h\n",
            "trinity-plugins/bamsifter/htslib/cram/os.h\n",
            "trinity-plugins/bamsifter/htslib/cram/pooled_alloc.c\n",
            "trinity-plugins/bamsifter/htslib/cram/pooled_alloc.h\n",
            "trinity-plugins/bamsifter/htslib/cram/rANS_byte.h\n",
            "trinity-plugins/bamsifter/htslib/cram/rANS_static.c\n",
            "trinity-plugins/bamsifter/htslib/cram/rANS_static.h\n",
            "trinity-plugins/bamsifter/htslib/cram/string_alloc.c\n",
            "trinity-plugins/bamsifter/htslib/cram/string_alloc.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/\n",
            "trinity-plugins/bamsifter/htslib/htslib/bgzf.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/cram.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/faidx.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/hfile.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/hts.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/hts_defs.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/hts_endian.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/hts_log.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/hts_os.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/kbitset.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/kfunc.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/khash.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/khash_str2int.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/klist.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/knetfile.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/kseq.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/ksort.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/kstring.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/regidx.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/sam.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/synced_bcf_reader.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/tbx.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/thread_pool.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/vcf.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/vcf_sweep.h\n",
            "trinity-plugins/bamsifter/htslib/htslib/vcfutils.h\n",
            "trinity-plugins/bamsifter/htslib/m4/\n",
            "trinity-plugins/bamsifter/htslib/m4/hts_hide_dynamic_syms.m4\n",
            "trinity-plugins/bamsifter/htslib/m4/hts_prog_cc_warnings.m4\n",
            "trinity-plugins/bamsifter/htslib/os/\n",
            "trinity-plugins/bamsifter/htslib/os/lzma_stub.h\n",
            "trinity-plugins/bamsifter/htslib/os/rand.c\n",
            "trinity-plugins/bamsifter/htslib/test/\n",
            "trinity-plugins/bamsifter/htslib/test/auxf#values.sam\n",
            "trinity-plugins/bamsifter/htslib/test/auxf#values_java.cram\n",
            "trinity-plugins/bamsifter/htslib/test/auxf.fa\n",
            "trinity-plugins/bamsifter/htslib/test/auxf.fa.fai\n",
            "trinity-plugins/bamsifter/htslib/test/bgziptest.txt\n",
            "trinity-plugins/bamsifter/htslib/test/bgziptest.txt.gz\n",
            "trinity-plugins/bamsifter/htslib/test/bgziptest.txt.gz.gzi\n",
            "trinity-plugins/bamsifter/htslib/test/c1#bounds.sam\n",
            "trinity-plugins/bamsifter/htslib/test/c1#clip.sam\n",
            "trinity-plugins/bamsifter/htslib/test/c1#noseq.sam\n",
            "trinity-plugins/bamsifter/htslib/test/c1#pad1.sam\n",
            "trinity-plugins/bamsifter/htslib/test/c1#pad2.sam\n",
            "trinity-plugins/bamsifter/htslib/test/c1#pad3.sam\n",
            "trinity-plugins/bamsifter/htslib/test/c1#unknown.sam\n",
            "trinity-plugins/bamsifter/htslib/test/c1.fa\n",
            "trinity-plugins/bamsifter/htslib/test/c1.fa.fai\n",
            "trinity-plugins/bamsifter/htslib/test/c2#pad.sam\n",
            "trinity-plugins/bamsifter/htslib/test/c2.fa\n",
            "trinity-plugins/bamsifter/htslib/test/c2.fa.fai\n",
            "trinity-plugins/bamsifter/htslib/test/ce#1.sam\n",
            "trinity-plugins/bamsifter/htslib/test/ce#1000.sam\n",
            "trinity-plugins/bamsifter/htslib/test/ce#2.sam\n",
            "trinity-plugins/bamsifter/htslib/test/ce#5.sam\n",
            "trinity-plugins/bamsifter/htslib/test/ce#5b.sam\n",
            "trinity-plugins/bamsifter/htslib/test/ce#5b_java.cram\n",
            "trinity-plugins/bamsifter/htslib/test/ce#large_seq.sam\n",
            "trinity-plugins/bamsifter/htslib/test/ce#supp.sam\n",
            "trinity-plugins/bamsifter/htslib/test/ce#tag_depadded.sam\n",
            "trinity-plugins/bamsifter/htslib/test/ce#tag_padded.sam\n",
            "trinity-plugins/bamsifter/htslib/test/ce#unmap.sam\n",
            "trinity-plugins/bamsifter/htslib/test/ce#unmap1.sam\n",
            "trinity-plugins/bamsifter/htslib/test/ce#unmap2.sam\n",
            "trinity-plugins/bamsifter/htslib/test/ce.fa\n",
            "trinity-plugins/bamsifter/htslib/test/ce.fa.fai\n",
            "trinity-plugins/bamsifter/htslib/test/colons.bam\n",
            "trinity-plugins/bamsifter/htslib/test/colons.bam.bai\n",
            "trinity-plugins/bamsifter/htslib/test/compare_sam.pl\n",
            "trinity-plugins/bamsifter/htslib/test/cross_validate.sh\n",
            "trinity-plugins/bamsifter/htslib/test/emptyfile\n",
            "trinity-plugins/bamsifter/htslib/test/faidx.fa\n",
            "trinity-plugins/bamsifter/htslib/test/fastqs.fq\n",
            "trinity-plugins/bamsifter/htslib/test/fastqs_README.txt\n",
            "trinity-plugins/bamsifter/htslib/test/fieldarith.c\n",
            "trinity-plugins/bamsifter/htslib/test/fieldarith.sam\n",
            "trinity-plugins/bamsifter/htslib/test/formatcols.vcf\n",
            "trinity-plugins/bamsifter/htslib/test/formatmissing-out.vcf\n",
            "trinity-plugins/bamsifter/htslib/test/formatmissing.vcf\n",
            "trinity-plugins/bamsifter/htslib/test/header_syms.pl\n",
            "trinity-plugins/bamsifter/htslib/test/hfile.c\n",
            "trinity-plugins/bamsifter/htslib/test/hts_endian.c\n",
            "trinity-plugins/bamsifter/htslib/test/index.bam.bai\n",
            "trinity-plugins/bamsifter/htslib/test/index.bam.csi\n",
            "trinity-plugins/bamsifter/htslib/test/index.bcf.csi\n",
            "trinity-plugins/bamsifter/htslib/test/index.cram.crai\n",
            "trinity-plugins/bamsifter/htslib/test/index.sam\n",
            "trinity-plugins/bamsifter/htslib/test/index.sam.gz.bai\n",
            "trinity-plugins/bamsifter/htslib/test/index.sam.gz.csi\n",
            "trinity-plugins/bamsifter/htslib/test/index.vcf\n",
            "trinity-plugins/bamsifter/htslib/test/index.vcf.gz.csi\n",
            "trinity-plugins/bamsifter/htslib/test/index.vcf.gz.tbi\n",
            "trinity-plugins/bamsifter/htslib/test/md#1.sam\n",
            "trinity-plugins/bamsifter/htslib/test/md.fa\n",
            "trinity-plugins/bamsifter/htslib/test/md.fa.fai\n",
            "trinity-plugins/bamsifter/htslib/test/no_hdr_sq_1.bam\n",
            "trinity-plugins/bamsifter/htslib/test/no_hdr_sq_1.bam.csi\n",
            "trinity-plugins/bamsifter/htslib/test/no_hdr_sq_1.expected.sam\n",
            "trinity-plugins/bamsifter/htslib/test/noroundtrip-out.vcf\n",
            "trinity-plugins/bamsifter/htslib/test/noroundtrip.vcf\n",
            "trinity-plugins/bamsifter/htslib/test/pileup.c\n",
            "trinity-plugins/bamsifter/htslib/test/range.bam\n",
            "trinity-plugins/bamsifter/htslib/test/range.bam.bai\n",
            "trinity-plugins/bamsifter/htslib/test/range.cram\n",
            "trinity-plugins/bamsifter/htslib/test/range.cram.crai\n",
            "trinity-plugins/bamsifter/htslib/test/range.out\n",
            "trinity-plugins/bamsifter/htslib/test/realn01.fa\n",
            "trinity-plugins/bamsifter/htslib/test/realn01.fa.fai\n",
            "trinity-plugins/bamsifter/htslib/test/realn01.sam\n",
            "trinity-plugins/bamsifter/htslib/test/realn01_exp-a.sam\n",
            "trinity-plugins/bamsifter/htslib/test/realn01_exp-e.sam\n",
            "trinity-plugins/bamsifter/htslib/test/realn01_exp.sam\n",
            "trinity-plugins/bamsifter/htslib/test/realn02-r.sam\n",
            "trinity-plugins/bamsifter/htslib/test/realn02.fa\n",
            "trinity-plugins/bamsifter/htslib/test/realn02.fa.fai\n",
            "trinity-plugins/bamsifter/htslib/test/realn02.sam\n",
            "trinity-plugins/bamsifter/htslib/test/realn02_exp-a.sam\n",
            "trinity-plugins/bamsifter/htslib/test/realn02_exp-e.sam\n",
            "trinity-plugins/bamsifter/htslib/test/realn02_exp.sam\n",
            "trinity-plugins/bamsifter/htslib/test/sam.c\n",
            "trinity-plugins/bamsifter/htslib/test/simple_test_driver.sh\n",
            "trinity-plugins/bamsifter/htslib/test/tabix.out\n",
            "trinity-plugins/bamsifter/htslib/test/test-bcf-sr.c\n",
            "trinity-plugins/bamsifter/htslib/test/test-bcf-sr.pl\n",
            "trinity-plugins/bamsifter/htslib/test/test-bcf-translate.c\n",
            "trinity-plugins/bamsifter/htslib/test/test-bcf-translate.out\n",
            "trinity-plugins/bamsifter/htslib/test/test-logging.pl\n",
            "trinity-plugins/bamsifter/htslib/test/test-parse-reg.c\n",
            "trinity-plugins/bamsifter/htslib/test/test-regidx.c\n",
            "trinity-plugins/bamsifter/htslib/test/test-vcf-api.c\n",
            "trinity-plugins/bamsifter/htslib/test/test-vcf-api.out\n",
            "trinity-plugins/bamsifter/htslib/test/test-vcf-hdr-in.vcf\n",
            "trinity-plugins/bamsifter/htslib/test/test-vcf-hdr.out\n",
            "trinity-plugins/bamsifter/htslib/test/test-vcf-sweep.c\n",
            "trinity-plugins/bamsifter/htslib/test/test-vcf-sweep.out\n",
            "trinity-plugins/bamsifter/htslib/test/test.pl\n",
            "trinity-plugins/bamsifter/htslib/test/test_bgzf.c\n",
            "trinity-plugins/bamsifter/htslib/test/test_index.c\n",
            "trinity-plugins/bamsifter/htslib/test/test_kstring.c\n",
            "trinity-plugins/bamsifter/htslib/test/test_realn.c\n",
            "trinity-plugins/bamsifter/htslib/test/test_str2int.c\n",
            "trinity-plugins/bamsifter/htslib/test/test_view.c\n",
            "trinity-plugins/bamsifter/htslib/test/thrash_threads1.c\n",
            "trinity-plugins/bamsifter/htslib/test/thrash_threads2.c\n",
            "trinity-plugins/bamsifter/htslib/test/thrash_threads3.c\n",
            "trinity-plugins/bamsifter/htslib/test/thrash_threads4.c\n",
            "trinity-plugins/bamsifter/htslib/test/thrash_threads5.c\n",
            "trinity-plugins/bamsifter/htslib/test/thrash_threads6.c\n",
            "trinity-plugins/bamsifter/htslib/test/thrash_threads7.c\n",
            "trinity-plugins/bamsifter/htslib/test/thread_pool.md\n",
            "trinity-plugins/bamsifter/htslib/test/xx#MD.sam\n",
            "trinity-plugins/bamsifter/htslib/test/xx#MD2.sam\n",
            "trinity-plugins/bamsifter/htslib/test/xx#blank.sam\n",
            "trinity-plugins/bamsifter/htslib/test/xx#large_aux.sam\n",
            "trinity-plugins/bamsifter/htslib/test/xx#large_aux2.sam\n",
            "trinity-plugins/bamsifter/htslib/test/xx#large_aux_java.cram\n",
            "trinity-plugins/bamsifter/htslib/test/xx#minimal.sam\n",
            "trinity-plugins/bamsifter/htslib/test/xx#pair.sam\n",
            "trinity-plugins/bamsifter/htslib/test/xx#repeated.sam\n",
            "trinity-plugins/bamsifter/htslib/test/xx#rg.sam\n",
            "trinity-plugins/bamsifter/htslib/test/xx#tlen.sam\n",
            "trinity-plugins/bamsifter/htslib/test/xx#tlen2.sam\n",
            "trinity-plugins/bamsifter/htslib/test/xx#triplet.sam\n",
            "trinity-plugins/bamsifter/htslib/test/xx#unsorted.sam\n",
            "trinity-plugins/bamsifter/htslib/test/xx.fa\n",
            "trinity-plugins/bamsifter/htslib/test/xx.fa.fai\n",
            "trinity-plugins/bamsifter/htslib/test/fuzz/\n",
            "trinity-plugins/bamsifter/htslib/test/fuzz/hts_open_fuzzer.c\n",
            "trinity-plugins/bamsifter/htslib/test/longrefs/\n",
            "trinity-plugins/bamsifter/htslib/test/longrefs/index.expected1.vcf\n",
            "trinity-plugins/bamsifter/htslib/test/longrefs/index.expected2.vcf\n",
            "trinity-plugins/bamsifter/htslib/test/longrefs/index.vcf\n",
            "trinity-plugins/bamsifter/htslib/test/longrefs/longref.sam\n",
            "trinity-plugins/bamsifter/htslib/test/longrefs/longref_itr.expected.sam\n",
            "trinity-plugins/bamsifter/htslib/test/longrefs/longref_multi.expected.sam\n",
            "trinity-plugins/bamsifter/htslib/test/maintainer/\n",
            "trinity-plugins/bamsifter/htslib/test/maintainer/check_copyright.pl\n",
            "trinity-plugins/bamsifter/htslib/test/maintainer/check_spaces.pl\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/c1#pad1.out\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/c1#pad1.sam\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/c1#pad2.out\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/c1#pad2.sam\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/c1#pad3.out\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/c1#pad3.sam\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/mp_D.out\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/mp_D.sam\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/mp_DI.out\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/mp_DI.sam\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/mp_I.out\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/mp_I.sam\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/mp_ID.out\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/mp_ID.sam\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/mp_N.out\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/mp_N.sam\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/mp_N2.out\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/mp_N2.sam\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/mp_P.out\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/mp_P.sam\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/mpileup.tst\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/small.bam\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/small.out\n",
            "trinity-plugins/bamsifter/htslib/test/mpileup/test-pileup.sh\n",
            "trinity-plugins/bamsifter/htslib/test/tabix/\n",
            "trinity-plugins/bamsifter/htslib/test/tabix/bed_file.Y.100200.out\n",
            "trinity-plugins/bamsifter/htslib/test/tabix/bed_file.bed\n",
            "trinity-plugins/bamsifter/htslib/test/tabix/gff_file.X.2934832.2935190.out\n",
            "trinity-plugins/bamsifter/htslib/test/tabix/gff_file.gff\n",
            "trinity-plugins/bamsifter/htslib/test/tabix/large_chr.20.1.2147483647.out\n",
            "trinity-plugins/bamsifter/htslib/test/tabix/large_chr.vcf\n",
            "trinity-plugins/bamsifter/htslib/test/tabix/tabix.tst\n",
            "trinity-plugins/bamsifter/htslib/test/tabix/test-tabix.sh\n",
            "trinity-plugins/bamsifter/htslib/test/tabix/vcf_file.1.3000151.out\n",
            "trinity-plugins/bamsifter/htslib/test/tabix/vcf_file.2.3199812.out\n",
            "trinity-plugins/bamsifter/htslib/test/tabix/vcf_file.vcf\n",
            "trinity-plugins/scaffold_iworm_contigs/\n",
            "trinity-plugins/scaffold_iworm_contigs/Makefile\n",
            "trinity-plugins/scaffold_iworm_contigs/ScaffoldIwormContigs.cpp\n",
            "trinity-plugins/scaffold_iworm_contigs/error_checker.cpp\n",
            "trinity-plugins/scaffold_iworm_contigs/error_checker.h\n",
            "trinity-plugins/seqtk-trinity/\n",
            "trinity-plugins/seqtk-trinity/LICENSE\n",
            "trinity-plugins/seqtk-trinity/Makefile\n",
            "trinity-plugins/seqtk-trinity/README.md\n",
            "trinity-plugins/seqtk-trinity/khash.h\n",
            "trinity-plugins/seqtk-trinity/kseq.h\n",
            "trinity-plugins/seqtk-trinity/seqtk.c\n",
            "trinity-plugins/seqtk-trinity/testing/\n",
            "trinity-plugins/seqtk-trinity/testing/Makefile\n",
            "trinity-plugins/seqtk-trinity/testing/corrupt_1.fq\n",
            "trinity-plugins/seqtk-trinity/testing/forward_reverse_notation_1.fq\n",
            "trinity-plugins/seqtk-trinity/testing/forward_reverse_notation_2.fq\n",
            "trinity-plugins/seqtk-trinity/testing/newformat_1.fq\n",
            "trinity-plugins/seqtk-trinity/testing/newformat_2.fq\n",
            "trinity-plugins/seqtk-trinity/testing/oldformat_1.fq\n",
            "trinity-plugins/seqtk-trinity/testing/oldformat_2.fq\n",
            "trinity-plugins/seqtk-trinity/testing/stripped.fq\n",
            "trinity-plugins/seqtk-trinity/testing/test_seqtk_trinity.py\n",
            "trinity-plugins/seqtk-trinity/testing/unequal_seq_n_quals_1.fq\n",
            "trinity-plugins/slclust/\n",
            "trinity-plugins/slclust/Makefile\n",
            "trinity-plugins/slclust/README\n",
            "trinity-plugins/slclust/bin/\n",
            "trinity-plugins/slclust/bin/slclust\n",
            "trinity-plugins/slclust/include/\n",
            "trinity-plugins/slclust/include/cmd_line_opts.h\n",
            "trinity-plugins/slclust/include/common.h\n",
            "trinity-plugins/slclust/include/graph.h\n",
            "trinity-plugins/slclust/include/graphnode.h\n",
            "trinity-plugins/slclust/src/\n",
            "trinity-plugins/slclust/src/Makefile\n",
            "trinity-plugins/slclust/src/cmd_line_opts.cpp\n",
            "trinity-plugins/slclust/src/cmd_line_opts.o\n",
            "trinity-plugins/slclust/src/graph.cpp\n",
            "trinity-plugins/slclust/src/graph.o\n",
            "trinity-plugins/slclust/src/graphnode.cpp\n",
            "trinity-plugins/slclust/src/graphnode.o\n",
            "trinity-plugins/slclust/src/slcluster.cpp\n",
            "trinity-plugins/slclust/src/slcluster.o\n",
            "trinityrnaseq.wiki/\n",
            "trinityrnaseq.wiki/Accessing-Trinity-on-Publicly-Available-Compute-Resources.md\n",
            "trinityrnaseq.wiki/Coding-Region-Identification-in-Trinity-Assemblies.md\n",
            "trinityrnaseq.wiki/Computing-and-Time-requirements.md\n",
            "trinityrnaseq.wiki/Contributing.md\n",
            "trinityrnaseq.wiki/Counting-Full-Length-Trinity-Transcripts.md\n",
            "trinityrnaseq.wiki/DiffTranscriptUsage.md\n",
            "trinityrnaseq.wiki/Errors-during-Trinity-run.md\n",
            "trinityrnaseq.wiki/Examine-Strand-Specificity.md\n",
            "trinityrnaseq.wiki/Functional-Annotation-of-Transcripts.md\n",
            "trinityrnaseq.wiki/Genome-Guided-Trinity-Transcriptome-Assembly.md\n",
            "trinityrnaseq.wiki/Genome-Structure-Annotation-Using-Trinity--and-PASA.md\n",
            "trinityrnaseq.wiki/Home.md\n",
            "trinityrnaseq.wiki/How-can-I-run-this-in-parallel-on-a-computing-grid?.md\n",
            "trinityrnaseq.wiki/How-do-I-combine-reads?.md\n",
            "trinityrnaseq.wiki/How-do-I-identify-the-specific-reads-that-were-incorporated-into-the-transcript-assemblies?.md\n",
            "trinityrnaseq.wiki/How-do-I-use-reads-I-downloaded-from-SRA?.md\n",
            "trinityrnaseq.wiki/Installing-Trinity.md\n",
            "trinityrnaseq.wiki/Navigating-Trinity-DE-features-Using-TM4-MeV.md\n",
            "trinityrnaseq.wiki/Output-of-Trinity-Assembly.md\n",
            "trinityrnaseq.wiki/Post-Transcriptome-Assembly-Downstream-Analyses.md\n",
            "trinityrnaseq.wiki/QC-Samples-and-Biological-Replicates.md\n",
            "trinityrnaseq.wiki/RNA-Seq-Read-Representation-by-Trinity-Assembly.md\n",
            "trinityrnaseq.wiki/Running-GOSeq.md\n",
            "trinityrnaseq.wiki/Running-Trinity.md\n",
            "trinityrnaseq.wiki/Sample-Specific-Expression.md\n",
            "trinityrnaseq.wiki/SuperTranscripts.md\n",
            "trinityrnaseq.wiki/There-are-too-many-transcripts!-What-do-I-do?.md\n",
            "trinityrnaseq.wiki/Transcriptome-Assembly-Quality-Assessment.md\n",
            "trinityrnaseq.wiki/Transcriptome-Contig-Nx-and-ExN50-stats.md\n",
            "trinityrnaseq.wiki/Trinity-Computing-Requirements.md\n",
            "trinityrnaseq.wiki/Trinity-Differential-Expression.md\n",
            "trinityrnaseq.wiki/Trinity-FAQ.md\n",
            "trinityrnaseq.wiki/Trinity-Insilico-Normalization.md\n",
            "trinityrnaseq.wiki/Trinity-On-Terra.md\n",
            "trinityrnaseq.wiki/Trinity-Progress-Monitoring.md\n",
            "trinityrnaseq.wiki/Trinity-Runtime-Profiling.md\n",
            "trinityrnaseq.wiki/Trinity-Tidbits.md\n",
            "trinityrnaseq.wiki/Trinity-Transcript-Quantification.md\n",
            "trinityrnaseq.wiki/Trinity-in-Docker.md\n",
            "trinityrnaseq.wiki/Variant-Calling.md\n",
            "trinityrnaseq.wiki/_Sidebar.md\n",
            "trinityrnaseq.wiki/data/\n",
            "trinityrnaseq.wiki/data/strand_specificity_SS_but_run_DS.dat\n",
            "trinityrnaseq.wiki/data/strand_specificity_analysis_DS.dat\n",
            "trinityrnaseq.wiki/data/strand_specificity_analysis_RF.dat\n",
            "trinityrnaseq.wiki/images/\n",
            "trinityrnaseq.wiki/images/DGE_DTE_DTU.png\n",
            "trinityrnaseq.wiki/images/ExN50_plot.png\n",
            "trinityrnaseq.wiki/images/Ex_vs_N50.png\n",
            "trinityrnaseq.wiki/images/FL_Spombe_saturation.png\n",
            "trinityrnaseq.wiki/images/FL_mouse_saturation.png\n",
            "trinityrnaseq.wiki/images/FusionInspectorsvg.png\n",
            "trinityrnaseq.wiki/images/IGV_Trinity_screenshot.png\n",
            "trinityrnaseq.wiki/images/MA_and_volcano_plot.png\n",
            "trinityrnaseq.wiki/images/MAplotSmearEdgeR.png\n",
            "trinityrnaseq.wiki/images/Nature_Trinity_cov.small.gif\n",
            "trinityrnaseq.wiki/images/PCA.png\n",
            "trinityrnaseq.wiki/images/TrinityCompositeLogo.png\n",
            "trinityrnaseq.wiki/images/TrinityGridExec.png\n",
            "trinityrnaseq.wiki/images/clustered_heatmap.png\n",
            "trinityrnaseq.wiki/images/collectl_monitoring_mouse10M.png\n",
            "trinityrnaseq.wiki/images/collectl_monitoring_mouse50M.png\n",
            "trinityrnaseq.wiki/images/cut_tree_60pct_gene_clusters.png\n",
            "trinityrnaseq.wiki/images/expression_profiles_for_clusters.png\n",
            "trinityrnaseq.wiki/images/extrapolate_gene_count.png\n",
            "trinityrnaseq.wiki/images/gene_count_vs_min_TPM.png\n",
            "trinityrnaseq.wiki/images/genes_vs_minFPKM_plot.png\n",
            "trinityrnaseq.wiki/images/genes_vs_samples_heatmap.png\n",
            "trinityrnaseq.wiki/images/hcl_genes_tree.png\n",
            "trinityrnaseq.wiki/images/igv_view_trinity_reads.png\n",
            "trinityrnaseq.wiki/images/manually_selected_clusters.png\n",
            "trinityrnaseq.wiki/images/moby.svg\n",
            "trinityrnaseq.wiki/images/nbt_cover.gif\n",
            "trinityrnaseq.wiki/images/replicate_pair_MA_plot.png\n",
            "trinityrnaseq.wiki/images/replicate_pair_correlations.png\n",
            "trinityrnaseq.wiki/images/replicate_pair_scatter_plot.png\n",
            "trinityrnaseq.wiki/images/sample_cor_matrix.png\n",
            "trinityrnaseq.wiki/images/sample_correlation_matrix.png\n",
            "trinityrnaseq.wiki/images/strand_specificity.jpg\n",
            "trinityrnaseq.wiki/images/strand_specificity_SS_but_run_DS.png\n",
            "trinityrnaseq.wiki/images/strand_specificity_analysis_DS.png\n",
            "trinityrnaseq.wiki/images/strand_specificity_analysis_RF.png\n",
            "trinityrnaseq.wiki/images/sum_of_mapped_fragments.png\n",
            "trinityrnaseq.wiki/images/superTranscripts_VarCalling.png\n",
            "trinityrnaseq.wiki/images/supertranscript.png\n",
            "trinityrnaseq.wiki/images/tissue_enriched_expression.png\n",
            "trinityrnaseq.wiki/images/trinity_button.jpg\n",
            "trinityrnaseq.wiki/images/MeV_guide/\n",
            "trinityrnaseq.wiki/images/MeV_guide/HCL_analysis_tab.png\n",
            "trinityrnaseq.wiki/images/MeV_guide/HCL_heatmap.png\n",
            "trinityrnaseq.wiki/images/MeV_guide/change_heatmap_element_size.png\n",
            "trinityrnaseq.wiki/images/MeV_guide/click_first_data_cell.png\n",
            "trinityrnaseq.wiki/images/MeV_guide/heatmap_cluster_selected.png\n",
            "trinityrnaseq.wiki/images/MeV_guide/hierarchical_clustering_dialog.png\n",
            "trinityrnaseq.wiki/images/MeV_guide/hierarchical_clustering_menu.png\n",
            "trinityrnaseq.wiki/images/MeV_guide/initial_MeV_loaded_data_view.png\n",
            "trinityrnaseq.wiki/images/MeV_guide/load_data_dialog.png\n",
            "trinityrnaseq.wiki/images/MeV_guide/load_data_menu.png\n",
            "trinityrnaseq.wiki/images/MeV_guide/mean_center_data.png\n",
            "trinityrnaseq.wiki/images/MeV_guide/select_matrix_file.png\n",
            "trinityrnaseq.wiki/images/MeV_guide/set_color_scale_limits_dialog.png\n",
            "trinityrnaseq.wiki/images/MeV_guide/set_color_scale_limits_menu.png\n",
            "trinityrnaseq.wiki/misc/\n",
            "trinityrnaseq.wiki/misc/glimma_MA/\n",
            "trinityrnaseq.wiki/misc/glimma_MA/MA-plot.html\n",
            "trinityrnaseq.wiki/misc/glimma_MA/css/\n",
            "trinityrnaseq.wiki/misc/glimma_MA/css/glimma.min.css\n",
            "trinityrnaseq.wiki/misc/glimma_MA/js/\n",
            "trinityrnaseq.wiki/misc/glimma_MA/js/data.js\n",
            "trinityrnaseq.wiki/misc/glimma_MA/js/glimma.min.js\n",
            "trinityrnaseq.wiki/misc/glimma_volcano/\n",
            "trinityrnaseq.wiki/misc/glimma_volcano/volcano.html\n",
            "trinityrnaseq.wiki/misc/glimma_volcano/css/\n",
            "trinityrnaseq.wiki/misc/glimma_volcano/css/glimma.min.css\n",
            "trinityrnaseq.wiki/misc/glimma_volcano/js/\n",
            "trinityrnaseq.wiki/misc/glimma_volcano/js/data.js\n",
            "trinityrnaseq.wiki/misc/glimma_volcano/js/glimma.min.js\n",
            "util/\n",
            "util/TrinityStats.pl\n",
            "util/abundance_estimates_to_matrix.pl\n",
            "util/align_and_estimate_abundance.pl\n",
            "util/analyze_blastPlus_topHit_coverage.pl\n",
            "util/filter_low_expr_transcripts.pl\n",
            "util/insilico_read_normalization.pl\n",
            "util/retrieve_sequences_from_fasta.pl\n",
            "util/sift_bam_max_cov.pl\n",
            "util/PBS/\n",
            "util/PBS/N50stats.pl\n",
            "util/PBS/README\n",
            "util/PBS/TRINITY.CONFIG.template\n",
            "util/PBS/pbs_check.pl\n",
            "util/PBS/trinity_kill.pl\n",
            "util/PBS/trinity_kill.sh\n",
            "util/PBS/trinity_pbs.cont\n",
            "util/PBS/trinity_pbs.header\n",
            "util/PBS/trinity_pbs.p1\n",
            "util/PBS/trinity_pbs.p2\n",
            "util/PBS/trinity_pbs.p3\n",
            "util/PBS/trinity_pbs.p4a\n",
            "util/PBS/trinity_pbs.p4b\n",
            "util/PBS/trinity_pbs.p5b\n",
            "util/PBS/trinity_pbs.sh\n",
            "util/R/\n",
            "util/R/expression_analysis_lib.R\n",
            "util/R/get_Poisson_conf_intervals.R\n",
            "util/misc/\n",
            "util/misc/ButterflyFastaToGraphDot.pl\n",
            "util/misc/HiCpipe_nameSortedSam_to_raw.pl\n",
            "util/misc/Monarch\n",
            "util/misc/N50.pl\n",
            "util/misc/SAM_coordsorted_max_reads_per_position.pl\n",
            "util/misc/SAM_intron_extractor.pl\n",
            "util/misc/SAM_nameSorted_to_uniq_count_stats.pl\n",
            "util/misc/SAM_pair_to_bed.pl\n",
            "util/misc/SAM_show_alignment.pl\n",
            "util/misc/SAM_show_alignment.summarize_stats.pl\n",
            "util/misc/SAM_sortAny_to_count_stats.pl\n",
            "util/misc/SAM_toString.pl\n",
            "util/misc/SAM_to_bed.pl\n",
            "util/misc/SAM_to_fasta.pl\n",
            "util/misc/SRA_to_fastq.notes\n",
            "util/misc/SRA_to_fastq.pl\n",
            "util/misc/TPM_weighted_gene_length.py\n",
            "util/misc/TophatCufflinksWrapper.pl\n",
            "util/misc/Trinity_genome_aligned_gff3_to_regrouped_genes_gtf.pl\n",
            "util/misc/Trinity_node_seq_extractor.pl\n",
            "util/misc/acc_list_to_fasta_entries.pl\n",
            "util/misc/alexie_analyze_blast.pl\n",
            "util/misc/align_reads_launch_igv.pl\n",
            "util/misc/allele_simulator.pl\n",
            "util/misc/analyze_blastPlus_topHit_coverage.annotate_details_w_FL_info.pl\n",
            "util/misc/analyze_blastPlus_topHit_coverage.by_prioritized_compreh_category.pl\n",
            "util/misc/analyze_blastPlus_topHit_coverage.extract_OS.pl\n",
            "util/misc/analyze_blastPlus_topHit_coverage.org_matrix.pl\n",
            "util/misc/average.pl\n",
            "util/misc/blast_outfmt6_group_segments.pl\n",
            "util/misc/blast_outfmt6_group_segments.to_Markov_Clustering.pl\n",
            "util/misc/blast_outfmt6_group_segments.tophit_coverage.pl\n",
            "util/misc/blastn_wrapper.pl\n",
            "util/misc/capture_orig_n_unmapped_reads.pl\n",
            "util/misc/cat_require_newlines.pl\n",
            "util/misc/cdhit_examine_isoforms.pl\n",
            "util/misc/cdna_fasta_file_to_transcript_gtf.pl\n",
            "util/misc/check_chrysalis_graph_reciprocal_edges.pl\n",
            "util/misc/check_fastQ_pair_ordering.pl\n",
            "util/misc/chrys_graph_to_dot.pl\n",
            "util/misc/collate_fqs.pl\n",
            "util/misc/combined_nameSorted_to_dup_pairs_removed.pl\n",
            "util/misc/compare_FL_stats.pl\n",
            "util/misc/compare_bflies.pl\n",
            "util/misc/component_to_graph_dot.pl\n",
            "util/misc/contig_ExN50_statistic.pl\n",
            "util/misc/convert_fasta_identifiers_for_FL_analysis.pl\n",
            "util/misc/count_N50_given_MIN_FPKM_threshold.pl\n",
            "util/misc/count_features_given_MIN_FPKM_threshold.pl\n",
            "util/misc/count_iso_per_gene_dist.pl\n",
            "util/misc/count_matrix_features_given_MIN_TPM_threshold.pl\n",
            "util/misc/count_number_fasta_seqs.pl\n",
            "util/misc/count_trans_per_component.pl\n",
            "util/misc/decode_SAM_flag_value.pl\n",
            "util/misc/describe_SAM_read_flag_info.pl\n",
            "util/misc/determine_RF_strand_specificity.pl\n",
            "util/misc/ensure_paired_end_bam_file.pl\n",
            "util/misc/examine_iworm_FL_across_threads.pl\n",
            "util/misc/examine_strand_specificity.pl\n",
            "util/misc/examine_weldmer_halves.pl\n",
            "util/misc/extract_fastQ_pairings.pl\n",
            "util/misc/fan_out_fasta_seqs_to_indiv_files.pl\n",
            "util/misc/fastQ_append_acc.pl\n",
            "util/misc/fastQ_rand_subset.SE.reservoir_sampling_reqiures_high_mem.pl\n",
            "util/misc/fastQ_rand_subset.pl\n",
            "util/misc/fastQ_rand_subset.reservoir_sampling_reqiures_high_mem.pl\n",
            "util/misc/fastQ_top_N_records.pl\n",
            "util/misc/fasta_file_reformatter.pl\n",
            "util/misc/fasta_filter_by_min_length.pl\n",
            "util/misc/fasta_remove_duplicates.pl\n",
            "util/misc/fasta_seq_length.pl\n",
            "util/misc/fasta_to_cmd_generator.pl\n",
            "util/misc/fasta_write_sense_n_anti.pl\n",
            "util/misc/fastq_cleaner.pl\n",
            "util/misc/fastq_interleave_pairs.pl\n",
            "util/misc/fastq_merge_sorted_tab_lists.pl\n",
            "util/misc/fastq_stats.pl\n",
            "util/misc/fastq_unweave_pairs.pl\n",
            "util/misc/filter_out_accs_from_fasta.pl\n",
            "util/misc/filter_similar_seqs_expr_and_strand_aware.pl\n",
            "util/misc/flattened_gff_n_genome_to_Trinity_emulator.pl\n",
            "util/misc/frag_boundary_to_wig.pl\n",
            "util/misc/frag_to_bed.pl\n",
            "util/misc/gene_gff3_to_introns.pl\n",
            "util/misc/gene_to_shared_transcript_content.pl\n",
            "util/misc/genome_gff3_to_gene_gff3_partitions.pl\n",
            "util/misc/get_GC_content_dist.pl\n",
            "util/misc/get_longest_isoform_seq_per_trinity_gene.pl\n",
            "util/misc/get_path_nodes_from_fasta.pl\n",
            "util/misc/get_welds_from_chrysals_graphFromFasta_out.pl\n",
            "util/misc/gff3_file_to_cdna.pl\n",
            "util/misc/gff3_file_utr_coverage_trimmer.pl\n",
            "util/misc/gff3_to_genome_feature_base_encoding.parse_SAM.pl\n",
            "util/misc/gff3_to_genome_feature_base_encoding.pl\n",
            "util/misc/gmap_gff3_chimera_jaccard_analyzer.pl\n",
            "util/misc/gmap_gff3_to_percent_length_stats.count_mapped_transcripts.pl\n",
            "util/misc/gmap_gff3_to_percent_length_stats.pl\n",
            "util/misc/gmap_native_to_format_converter.pl\n",
            "util/misc/gtf_to_bed_format.pl\n",
            "util/misc/gtf_to_introns.pl\n",
            "util/misc/hicpipe_raw_converter.pl\n",
            "util/misc/identify_distal_isoform_variations.pl\n",
            "util/misc/illustrate_ref_comparison.pl\n",
            "util/misc/iworm_welds_to_dot.pl\n",
            "util/misc/jaccard_sam_pair_refiner.pl\n",
            "util/misc/join_any.pl\n",
            "util/misc/join_by_left_col.pl\n",
            "util/misc/join_expr_vals_single_table.pl\n",
            "util/misc/kmer_counter.pl\n",
            "util/misc/m8_blastclust.pl\n",
            "util/misc/map_gtf_transcripts_to_genome_annots.pl\n",
            "util/misc/merge_RSEM_output_to_matrix.pl\n",
            "util/misc/merge_blast_n_rsem_results.pl\n",
            "util/misc/merge_rsem_n_express_for_compare.pl\n",
            "util/misc/mpi_iworm_proc_contigs_to_fa.pl\n",
            "util/misc/nameSorted_SAM_to_FastQ.pl\n",
            "util/misc/nameSorted_SAM_to_paired_fastq.pl\n",
            "util/misc/omp_iworm_thread_contigs_to_fa.pl\n",
            "util/misc/organize_data_table_by_trinity_component.pl\n",
            "util/misc/pair_up_fastq_files_1_2.pl\n",
            "util/misc/pair_up_fastq_files_LeftRight.pl\n",
            "util/misc/pair_up_fastq_files_R1_R2.pl\n",
            "util/misc/pairwise_kmer_content_comparer.pl\n",
            "util/misc/plot_ExN50_statistic.Rscript\n",
            "util/misc/plot_expressed_gene_dist.pl\n",
            "util/misc/plot_strand_specificity_dist_by_quantile.Rscript\n",
            "util/misc/print.pl\n",
            "util/misc/print_kmers.pl\n",
            "util/misc/process_GMAP_alignments_gff3_chimeras_ok.pl\n",
            "util/misc/process_minimap2_alignments.pl\n",
            "util/misc/prop_pair_sam_refiner.pl\n",
            "util/misc/randomly_mutate_seqs.pl\n",
            "util/misc/randomly_sample_PE_fastq.pl\n",
            "util/misc/remove_cntrl_chars.pl\n",
            "util/misc/row_to_column.pl\n",
            "util/misc/run_DETONATE.pl\n",
            "util/misc/run_GSNAP.pl\n",
            "util/misc/run_HISAT.pl\n",
            "util/misc/run_HISAT2_via_samples_file.pl\n",
            "util/misc/run_HiCpipe_bowtie.pl\n",
            "util/misc/run_STAR.pl\n",
            "util/misc/run_STAR_via_samples_file.pl\n",
            "util/misc/run_TOPHAT.pl\n",
            "util/misc/run_bowtie2.pl\n",
            "util/misc/run_bwa.pl\n",
            "util/misc/run_bwasw_trinity.pl\n",
            "util/misc/run_jellyfish.pl\n",
            "util/misc/run_read_simulator_per_fasta_entry.pl\n",
            "util/misc/run_read_simulator_per_gene.pl\n",
            "util/misc/run_trimmomatic_qual_trimming.pl\n",
            "util/misc/seqinfo_refseq_to_dot.pl\n",
            "util/misc/shuffle.pl\n",
            "util/misc/simulate_illuminaPE_from_transcripts.pl\n",
            "util/misc/simulate_illuminaPE_from_transcripts.wgsim.pl\n",
            "util/misc/simulate_reads_sam_and_fa.pl\n",
            "util/misc/sixFrameTranslation.pl\n",
            "util/misc/sort_fastq.pl\n",
            "util/misc/strip_fasta_header.pl\n",
            "util/misc/tab_to_fastQ.pl\n",
            "util/misc/tab_to_fasta.pl\n",
            "util/misc/tblastn_wrapper.pl\n",
            "util/misc/testUnlimitStacksize.pl\n",
            "util/misc/transcript_coverage_UTR_trimmer.pl\n",
            "util/misc/transcript_fasta_to_ORF_pics.pl\n",
            "util/misc/transcript_gff3_to_bed.pl\n",
            "util/misc/transdecoder_pep_to_false_fusion_finder.pl\n",
            "util/misc/trinity_component_distribution.pl\n",
            "util/misc/trinity_trans_matrix_to_rep_trans_gene_matrix.pl\n",
            "util/misc/Artemis/\n",
            "util/misc/Artemis/join_multi_wig_to_graph_plot.pl\n",
            "util/misc/Monarch_util/\n",
            "util/misc/Monarch_util/generate_gene_alt_splicing_graphs.pl\n",
            "util/misc/Monarch_util/generate_trans_graphs.pl\n",
            "util/misc/PerlLib/\n",
            "util/misc/PerlLib/SegmentGraph.pm\n",
            "util/misc/TEST_SUPPORT/\n",
            "util/misc/TEST_SUPPORT/BFLY_TESTING/\n",
            "util/misc/TEST_SUPPORT/BFLY_TESTING/graph_out_to_bfly_cmd.pl\n",
            "util/misc/alt_GG_read_partitioning_JCornish/\n",
            "util/misc/alt_GG_read_partitioning_JCornish/genwig.sh\n",
            "util/misc/alt_GG_read_partitioning_JCornish/genwig2.py\n",
            "util/misc/altsplice_simulation_toolkit/\n",
            "util/misc/altsplice_simulation_toolkit/sim_single_bubble.pl\n",
            "util/misc/bam_gene_tests/\n",
            "util/misc/bam_gene_tests/extract_bam_reads_per_target_gene.pl\n",
            "util/misc/bam_gene_tests/extract_bam_reads_per_target_transcript.pl\n",
            "util/misc/bam_gene_tests/harvest_transcripts.pl\n",
            "util/misc/bam_gene_tests/write_trin_cmds.pl\n",
            "util/misc/blat_util/\n",
            "util/misc/blat_util/blat_sam_add_reads2.pl\n",
            "util/misc/blat_util/blat_to_sam.pl\n",
            "util/misc/blat_util/blat_top_hit_extractor.pl\n",
            "util/misc/blat_util/process_BLAT_alignments.pl\n",
            "util/misc/blat_util/pslx_to_gff3.pl\n",
            "util/misc/blat_util/run_BLAT_shortReads.pl\n",
            "util/misc/blat_util/top_blat_sam_extractor.pl\n",
            "util/misc/insilico_norm_kmer_hists/\n",
            "util/misc/insilico_norm_kmer_hists/kmer_histo.NormMaxKCov50.txt\n",
            "util/misc/insilico_norm_kmer_hists/kmer_histo.all.txt\n",
            "util/misc/insilico_norm_kmer_hists/plot_me.R\n",
            "util/misc/insilico_norm_kmer_hists/result.pdf\n",
            "util/misc/iso_reco_analysis/\n",
            "util/misc/iso_reco_analysis/bam_to_cuff.pl\n",
            "util/misc/iso_reco_analysis/cuff_gtf_to_bed.pl\n",
            "util/misc/iso_reco_analysis/gene_gff3_to_bed_cmds.pl\n",
            "util/misc/iso_reco_analysis/gmap_to_ref.pl\n",
            "util/misc/iso_reco_analysis/notes\n",
            "util/misc/iso_reco_analysis/run_trinity_WITH_LR.pl\n",
            "util/misc/iso_reco_analysis/run_trinity_no_LR.pl\n",
            "util/misc/iso_reco_analysis/sim_reads.pl\n",
            "util/misc/iso_reco_analysis/trans_gff3_to_bed_cmds.pl\n",
            "util/misc/sim_test_framework/\n",
            "util/misc/sim_test_framework/audit_summary_stats.pl\n",
            "util/misc/sim_test_framework/audit_summary_stats.reexamine.pl\n",
            "util/misc/sim_test_framework/info_files_to_eval_cmds.pl\n",
            "util/misc/sim_test_framework/partition_target_transcripts.pl\n",
            "util/misc/sim_test_framework/run_Trinity_eval.pl\n",
            "util/misc/sim_test_framework/run_Trinity_eval.sh\n",
            "util/misc/sim_test_framework/run_simulate_reads.wgsim.pl\n",
            "util/misc/sim_test_framework/write_simulate_read_commands.pl\n",
            "util/misc/sim_test_framework/util/\n",
            "util/misc/sim_test_framework/util/find_pruned_edges_shouldve_kept.pl\n",
            "util/misc/splice_path_analysis/\n",
            "util/misc/splice_path_analysis/assess_intron_path_sensitivity.pl\n",
            "util/misc/splice_path_analysis/assess_intron_path_sensitivity.summarizer.pl\n",
            "util/misc/splice_path_analysis/diff_splice_paths.pl\n",
            "util/misc/splice_path_analysis/intron_barcharter.pl\n",
            "util/support_scripts/\n",
            "util/support_scripts/ExitTester.jar\n",
            "util/support_scripts/GG_partitioned_trinity_aggregator.pl\n",
            "util/support_scripts/SAM_coordSorted_fragment_Read_coverage_writer.pl\n",
            "util/support_scripts/SAM_coordSorted_fragment_coverage_writer2.pl\n",
            "util/support_scripts/SAM_extract_properly_mapped_pairs.pl\n",
            "util/support_scripts/SAM_extract_uniquely_mapped_reads.pl\n",
            "util/support_scripts/SAM_filter_out_unmapped_reads.pl\n",
            "util/support_scripts/SAM_ordered_pair_jaccard.pl\n",
            "util/support_scripts/SAM_set_transcribed_orient_info.pl\n",
            "util/support_scripts/SAM_strand_separator.pl\n",
            "util/support_scripts/SAM_to_frag_coords.pl\n",
            "util/support_scripts/add_LR_reads_to_iworm_bundle.pl\n",
            "util/support_scripts/annotate_chrysalis_welds_with_iworm_names.pl\n",
            "util/support_scripts/batch_cmds.pl\n",
            "util/support_scripts/bowtie2_wrapper.pl\n",
            "util/support_scripts/define_SAM_coverage_partitions2.pl\n",
            "util/support_scripts/define_coverage_partitions.pl\n",
            "util/support_scripts/eXpress_trans_to_gene_results.pl\n",
            "util/support_scripts/ensure_coord_sorted_sam.pl\n",
            "util/support_scripts/extract_reads_per_partition.pl\n",
            "util/support_scripts/fastQ_to_fastA.pl\n",
            "util/support_scripts/fastQ_to_tab.pl\n",
            "util/support_scripts/fasta_find_duplicates.pl\n",
            "util/support_scripts/fasta_to_tab.pl\n",
            "util/support_scripts/filter_iworm_by_min_length_or_cov.pl\n",
            "util/support_scripts/filter_transcripts_require_min_cov.pl\n",
            "util/support_scripts/fragment_coverage_writer.pl\n",
            "util/support_scripts/get_Trinity_gene_to_trans_map.pl\n",
            "util/support_scripts/inchworm_transcript_splitter.pl\n",
            "util/support_scripts/iworm_LR_to_scaff_pairs.pl\n",
            "util/support_scripts/jaccard_fasta_clipper.pl\n",
            "util/support_scripts/jaccard_wig_clipper.pl\n",
            "util/support_scripts/join_partitions_within_range.pl\n",
            "util/support_scripts/kallisto_trans_to_gene_results.pl\n",
            "util/support_scripts/merge_pair_and_LR_scaff_links.pl\n",
            "util/support_scripts/nbkc_merge_left_right_stats.pl\n",
            "util/support_scripts/nbkc_normalize.pl\n",
            "util/support_scripts/ordered_fragment_coords_to_jaccard.pl\n",
            "util/support_scripts/outfmt6_add_percent_match_length.pl\n",
            "util/support_scripts/partition_chrysalis_graphs_n_reads.pl\n",
            "util/support_scripts/partitioned_trinity_aggregator.pl\n",
            "util/support_scripts/plugin_install_tests.sh\n",
            "util/support_scripts/prep_rnaseq_alignments_for_genome_assisted_assembly.pl\n",
            "util/support_scripts/print_butterfly_assemblies.pl\n",
            "util/support_scripts/process_GMAP_alignments_gff3_chimeras_ok.pl\n",
            "util/support_scripts/revcomp_fasta.pl\n",
            "util/support_scripts/run_TMM_scale_matrix.pl\n",
            "util/support_scripts/run_UpperQuartileNormalization_matrix.pl\n",
            "util/support_scripts/salmon_runner.pl\n",
            "util/support_scripts/salmon_trans_to_gene_results.pl\n",
            "util/support_scripts/scaffold_iworm_contigs.pl\n",
            "util/support_scripts/segment_GFF_partitions.pl\n",
            "util/support_scripts/trinity_install_tests.sh\n",
            "util/support_scripts/trinity_installer.py\n",
            "util/support_scripts/wig_clip_to_bed.pl\n",
            "util/support_scripts/write_partitioned_trinity_cmds.pl\n",
            "util/support_scripts/tests/\n",
            "util/support_scripts/tests/sample_data_tests.py\n",
            "util/support_scripts/tests/test_prep.py\n",
            "util/support_scripts/tests/tests.py\n",
            "\n",
            "sent 100,791,314 bytes  received 34,300 bytes  67,217,076.00 bytes/sec\n",
            "total size is 100,643,228  speedup is 1.00\n",
            "Trinity package installed at: /usr/local/bin\n",
            "\n",
            "\n",
            "\tFor convenience, set env var TRINITY_HOME=/usr/local/bin\n",
            "\n",
            "\tSimply add:\n",
            "\n",
            "\texport TRINITY_HOME=/usr/local/bin\n",
            "\tto your ~/.bashrc file.\n",
            "\n",
            "\n",
            "\tand run trinity via:  $TRINITY_HOME/Trinity\n",
            "\n",
            "\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "!export TRINITY_HOME=/usr/local/bin"
      ],
      "metadata": {
        "id": "eIwnUEAzubcY"
      },
      "execution_count": 12,
      "outputs": []
    },
    {
      "cell_type": "code",
      "source": [
        "%cd .."
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "H7F1OuHpvZsD",
        "outputId": "fcda46de-657c-4fd6-88de-8217bd0dc18f"
      },
      "execution_count": 13,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "#TEST INSTALLATION\n",
        "!Trinity --help"
      ],
      "metadata": {
        "id": "ARfKsTyyMcf2",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "d17a2f72-e8f4-4ead-da79-c6fa82837abb"
      },
      "execution_count": 14,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\n",
            "\n",
            "###############################################################################\n",
            "#\n",
            "\n",
            "     ______  ____   ____  ____   ____  ______  __ __\n",
            "    |      ||    \\ |    ||    \\ |    ||      ||  |  |\n",
            "    |      ||  D  ) |  | |  _  | |  | |      ||  |  |\n",
            "    |_|  |_||    /  |  | |  |  | |  | |_|  |_||  ~  |\n",
            "      |  |  |    \\  |  | |  |  | |  |   |  |  |___, |\n",
            "      |  |  |  .  \\ |  | |  |  | |  |   |  |  |     |\n",
            "      |__|  |__|\\_||____||__|__||____|  |__|  |____/\n",
            "\n",
            "    Trinity-v2.13.2\n",
            "\n",
            "\n",
            "#\n",
            "#\n",
            "# Required:\n",
            "#\n",
            "#  --seqType <string>      :type of reads: ('fa' or 'fq')\n",
            "#\n",
            "#  --max_memory <string>      :suggested max memory to use by Trinity where limiting can be enabled. (jellyfish, sorting, etc)\n",
            "#                            provided in Gb of RAM, ie.  '--max_memory 10G'\n",
            "#\n",
            "#  If paired reads:\n",
            "#      --left  <string>    :left reads, one or more file names (separated by commas, no spaces)\n",
            "#      --right <string>    :right reads, one or more file names (separated by commas, no spaces)\n",
            "#\n",
            "#  Or, if unpaired reads:\n",
            "#      --single <string>   :single reads, one or more file names, comma-delimited (note, if single file contains pairs, can use flag: --run_as_paired )\n",
            "#\n",
            "#  Or,\n",
            "#      --samples_file <string>         tab-delimited text file indicating biological replicate relationships.\n",
            "#                                   ex.\n",
            "#                                        cond_A    cond_A_rep1    A_rep1_left.fq    A_rep1_right.fq\n",
            "#                                        cond_A    cond_A_rep2    A_rep2_left.fq    A_rep2_right.fq\n",
            "#                                        cond_B    cond_B_rep1    B_rep1_left.fq    B_rep1_right.fq\n",
            "#                                        cond_B    cond_B_rep2    B_rep2_left.fq    B_rep2_right.fq\n",
            "#\n",
            "#                      # if single-end instead of paired-end, then leave the 4th column above empty.\n",
            "#\n",
            "####################################\n",
            "##  Misc:  #########################\n",
            "#\n",
            "#  --include_supertranscripts      :yield supertranscripts fasta and gtf files as outputs.\n",
            "#\n",
            "#  --SS_lib_type <string>          :Strand-specific RNA-Seq read orientation.\n",
            "#                                   if paired: RF or FR,\n",
            "#                                   if single: F or R.   (dUTP method = RF)\n",
            "#                                   See web documentation.\n",
            "#\n",
            "#  --CPU <int>                     :number of CPUs to use, default: 2\n",
            "#  --min_contig_length <int>       :minimum assembled contig length to report\n",
            "#                                   (def=200)\n",
            "#\n",
            "#  --long_reads <string>           :fasta file containing error-corrected or circular consensus (CCS) pac bio reads\n",
            "#                                   (** note: experimental parameter **, this functionality continues to be under development)\n",
            "#\n",
            "#  --genome_guided_bam <string>    :genome guided mode, provide path to coordinate-sorted bam file.\n",
            "#                                   (see genome-guided param section under --show_full_usage_info)\n",
            "#\n",
            "#  --long_reads_bam <string>       :long reads to include for genome-guided Trinity\n",
            "#                                  (bam file consists of error-corrected or circular consensus (CCS) pac bio read aligned to the genome)\n",
            "#\n",
            "#  --jaccard_clip                  :option, set if you have paired reads and\n",
            "#                                   you expect high gene density with UTR\n",
            "#                                   overlap (use FASTQ input file format\n",
            "#                                   for reads).\n",
            "#                                   (note: jaccard_clip is an expensive\n",
            "#                                   operation, so avoid using it unless\n",
            "#                                   necessary due to finding excessive fusion\n",
            "#                                   transcripts w/o it.)\n",
            "#\n",
            "#  --trimmomatic                   :run Trimmomatic to quality trim reads\n",
            "#                                        see '--quality_trimming_params' under full usage info for tailored settings.\n",
            "#\n",
            "#  --output <string>               :name of directory for output (will be\n",
            "#                                   created if it doesn't already exist)\n",
            "#                                   default( your current working directory: \"/content/trinity_out_dir\" \n",
            "#                                    note: must include 'trinity' in the name as a safety precaution! )\n",
            "#  \n",
            "#  --full_cleanup                  :only retain the Trinity fasta file, rename as ${output_dir}.Trinity.fasta\n",
            "#\n",
            "#  --cite                          :show the Trinity literature citation\n",
            "#\n",
            "#  --verbose                       :provide additional job status info during the run.\n",
            "#\n",
            "#  --version                       :reports Trinity version (Trinity-v2.13.2) and exits.\n",
            "#\n",
            "#  --show_full_usage_info          :show the many many more options available for running Trinity (expert usage).\n",
            "#\n",
            "#\n",
            "###############################################################################\n",
            "#\n",
            "#  *Note, a typical Trinity command might be:\n",
            "#\n",
            "#        Trinity --seqType fq --max_memory 50G --left reads_1.fq  --right reads_2.fq --CPU 6\n",
            "#\n",
            "#            (if you have multiple samples, use --samples_file ... see above for details)\n",
            "#\n",
            "#    and for Genome-guided Trinity, provide a coordinate-sorted bam:\n",
            "#\n",
            "#        Trinity --genome_guided_bam rnaseq_alignments.csorted.bam --max_memory 50G\n",
            "#                --genome_guided_max_intron 10000 --CPU 6\n",
            "#\n",
            "#     see: /usr/local/bin/sample_data/test_Trinity_Assembly/\n",
            "#          for sample data and 'runMe.sh' for example Trinity execution\n",
            "#\n",
            "#     For more details, visit: http://trinityrnaseq.github.io\n",
            "#\n",
            "###############################################################################\n",
            "\n",
            "\n",
            "    \n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "**02. Usage Test**"
      ],
      "metadata": {
        "id": "4bdYe-SQYZRi"
      }
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "YM6rler270U7",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "f5926005-7f02-4fc6-e610-596d104c4e3b"
      },
      "source": [
        "#DOWNLOAD EXAMPLE DATA\n",
        "!wget -O TrinityNatureProtocolTutorial.tgz https://sourceforge.net/projects/trinityrnaseq/files/misc/TrinityNatureProtocolTutorial.tgz/download"
      ],
      "execution_count": 15,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "--2021-12-10 03:40:00--  https://sourceforge.net/projects/trinityrnaseq/files/misc/TrinityNatureProtocolTutorial.tgz/download\n",
            "Resolving sourceforge.net (sourceforge.net)... 204.68.111.105\n",
            "Connecting to sourceforge.net (sourceforge.net)|204.68.111.105|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 Found\n",
            "Location: https://downloads.sourceforge.net/project/trinityrnaseq/misc/TrinityNatureProtocolTutorial.tgz?ts=gAAAAABhsswRSh9R5F45BwcpS88iVGVPKuHiUPOG4zkznsy1cgixoPFCXCC5izu2Bdtk-qs5mBRK7AWAM4uUFdAQqVteUzwlJw%3D%3D&use_mirror=phoenixnap&r= [following]\n",
            "--2021-12-10 03:40:01--  https://downloads.sourceforge.net/project/trinityrnaseq/misc/TrinityNatureProtocolTutorial.tgz?ts=gAAAAABhsswRSh9R5F45BwcpS88iVGVPKuHiUPOG4zkznsy1cgixoPFCXCC5izu2Bdtk-qs5mBRK7AWAM4uUFdAQqVteUzwlJw%3D%3D&use_mirror=phoenixnap&r=\n",
            "Resolving downloads.sourceforge.net (downloads.sourceforge.net)... 204.68.111.105\n",
            "Connecting to downloads.sourceforge.net (downloads.sourceforge.net)|204.68.111.105|:443... connected.\n",
            "HTTP request sent, awaiting response... 302 Found\n",
            "Location: https://phoenixnap.dl.sourceforge.net/project/trinityrnaseq/misc/TrinityNatureProtocolTutorial.tgz [following]\n",
            "--2021-12-10 03:40:01--  https://phoenixnap.dl.sourceforge.net/project/trinityrnaseq/misc/TrinityNatureProtocolTutorial.tgz\n",
            "Resolving phoenixnap.dl.sourceforge.net (phoenixnap.dl.sourceforge.net)... 184.164.141.26\n",
            "Connecting to phoenixnap.dl.sourceforge.net (phoenixnap.dl.sourceforge.net)|184.164.141.26|:443... connected.\n",
            "HTTP request sent, awaiting response... 200 OK\n",
            "Length: 474015394 (452M) [application/octet-stream]\n",
            "Saving to: ‘TrinityNatureProtocolTutorial.tgz’\n",
            "\n",
            "TrinityNatureProtoc 100%[===================>] 452.06M  38.4MB/s    in 12s     \n",
            "\n",
            "2021-12-10 03:40:14 (36.6 MB/s) - ‘TrinityNatureProtocolTutorial.tgz’ saved [474015394/474015394]\n",
            "\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "vlCFX1mvB-RZ",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "67a9c0b4-577a-4260-8626-c9d7ec76a79f"
      },
      "source": [
        "#UNPACKING DATA FILE\n",
        "!tar -xvf TrinityNatureProtocolTutorial.tgz"
      ],
      "execution_count": 16,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "TrinityNatureProtocolTutorial/\n",
            "TrinityNatureProtocolTutorial/S_pombe_refTrans.fasta\n",
            "TrinityNatureProtocolTutorial/1M_READS_sample/\n",
            "TrinityNatureProtocolTutorial/1M_READS_sample/Sp.hs.1M.left.fq\n",
            "TrinityNatureProtocolTutorial/1M_READS_sample/Sp.log.1M.right.fq\n",
            "TrinityNatureProtocolTutorial/1M_READS_sample/Sp.ds.1M.right.fq\n",
            "TrinityNatureProtocolTutorial/1M_READS_sample/Sp.ds.1M.left.fq\n",
            "TrinityNatureProtocolTutorial/1M_READS_sample/Sp.log.1M.left.fq\n",
            "TrinityNatureProtocolTutorial/1M_READS_sample/Sp.plat.1M.left.fq\n",
            "TrinityNatureProtocolTutorial/1M_READS_sample/Sp.plat.1M.right.fq\n",
            "TrinityNatureProtocolTutorial/1M_READS_sample/Sp.hs.1M.right.fq\n",
            "TrinityNatureProtocolTutorial/samples_n_reads_described.txt\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "K7SB0C0MEE8F"
      },
      "source": [
        "**03. Example**"
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "%cd TrinityNatureProtocolTutorial/1M_READS_sample "
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "thlsm-v_ya9f",
        "outputId": "a326ddc4-bc61-449a-ec94-6dbd76aa9797"
      },
      "execution_count": 17,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/TrinityNatureProtocolTutorial/1M_READS_sample\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "To assemble, provide Trinity with a list of all the left and right fastq/fasta files to the --left and --right parameters"
      ],
      "metadata": {
        "id": "Jv4ve9-_ZKXb"
      }
    },
    {
      "cell_type": "code",
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "qydcx2iBX5dn",
        "outputId": "f7c3dc56-40fd-41b0-dd53-9ca1ad6f7a09"
      },
      "source": [
        "#RUN TRINITY ASSEMBLY\n",
        "!Trinity --seqType fq --SS_lib_type RF \\\n",
        "           --left Sp.ds.1M.left.fq,Sp.log.1M.left.fq,Sp.hs.1M.left.fq,Sp.plat.1M.left.fq \\\n",
        "           --right Sp.log.1M.right.fq,Sp.hs.1M.right.fq,Sp.ds.1M.right.fq,Sp.plat.1M.right.fq \\\n",
        "           --CPU 10 --max_memory 20G \\\n",
        "           --output /content/TRINITY --no_salmon"
      ],
      "execution_count": 18,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\n",
            "     ______  ____   ____  ____   ____  ______  __ __\n",
            "    |      ||    \\ |    ||    \\ |    ||      ||  |  |\n",
            "    |      ||  D  ) |  | |  _  | |  | |      ||  |  |\n",
            "    |_|  |_||    /  |  | |  |  | |  | |_|  |_||  ~  |\n",
            "      |  |  |    \\  |  | |  |  | |  |   |  |  |___, |\n",
            "      |  |  |  .  \\ |  | |  |  | |  |   |  |  |     |\n",
            "      |__|  |__|\\_||____||__|__||____|  |__|  |____/\n",
            "\n",
            "    Trinity-v2.13.2\n",
            "\n",
            "\n",
            "\n",
            "Left read files: $VAR1 = [\n",
            "          'Sp.ds.1M.left.fq',\n",
            "          'Sp.log.1M.left.fq',\n",
            "          'Sp.hs.1M.left.fq',\n",
            "          'Sp.plat.1M.left.fq'\n",
            "        ];\n",
            "Right read files: $VAR1 = [\n",
            "          'Sp.log.1M.right.fq',\n",
            "          'Sp.hs.1M.right.fq',\n",
            "          'Sp.ds.1M.right.fq',\n",
            "          'Sp.plat.1M.right.fq'\n",
            "        ];\n",
            "Trinity version: Trinity-v2.13.2\n",
            "-currently using the latest production release of Trinity.\n",
            "\n",
            "Friday, December 10, 2021: 03:40:30\tCMD: java -Xmx64m -XX:ParallelGCThreads=2  -jar /usr/local/bin/util/support_scripts/ExitTester.jar 0\n",
            "Friday, December 10, 2021: 03:40:30\tCMD: java -Xmx4g -XX:ParallelGCThreads=2  -jar /usr/local/bin/util/support_scripts/ExitTester.jar 1\n",
            "Friday, December 10, 2021: 03:40:31\tCMD: mkdir -p /content/TRINITY\n",
            "Friday, December 10, 2021: 03:40:31\tCMD: mkdir -p /content/TRINITY/chrysalis\n",
            "\n",
            "\n",
            "----------------------------------------------------------------------------------\n",
            "-------------- Trinity Phase 1: Clustering of RNA-Seq Reads  ---------------------\n",
            "----------------------------------------------------------------------------------\n",
            "\n",
            "---------------------------------------------------------------\n",
            "------------ In silico Read Normalization ---------------------\n",
            "-- (Removing Excess Reads Beyond 200 Coverage --\n",
            "---------------------------------------------------------------\n",
            "\n",
            "# running normalization on reads: $VAR1 = [\n",
            "          [\n",
            "            '/content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.ds.1M.left.fq',\n",
            "            '/content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.log.1M.left.fq',\n",
            "            '/content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.hs.1M.left.fq',\n",
            "            '/content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.plat.1M.left.fq'\n",
            "          ],\n",
            "          [\n",
            "            '/content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.log.1M.right.fq',\n",
            "            '/content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.hs.1M.right.fq',\n",
            "            '/content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.ds.1M.right.fq',\n",
            "            '/content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.plat.1M.right.fq'\n",
            "          ]\n",
            "        ];\n",
            "\n",
            "\n",
            "Friday, December 10, 2021: 03:40:31\tCMD: /usr/local/bin/util/insilico_read_normalization.pl --seqType fq --JM 20G  --max_cov 200 --min_cov 1 --CPU 10 --output /content/TRINITY/insilico_read_normalization --max_CV 10000  --SS_lib_type RF  --left /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.ds.1M.left.fq,/content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.log.1M.left.fq,/content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.hs.1M.left.fq,/content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.plat.1M.left.fq --right /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.log.1M.right.fq,/content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.hs.1M.right.fq,/content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.ds.1M.right.fq,/content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.plat.1M.right.fq --pairs_together  --PARALLEL_STATS  \n",
            "-prepping seqs\n",
            "Converting input files. (both directions in parallel)CMD: seqtk-trinity seq -r -A -R 1  /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.ds.1M.left.fq >> left.fa\n",
            "CMD: seqtk-trinity seq -A -R 2  /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.log.1M.right.fq >> right.fa\n",
            "CMD finished (1 seconds)\n",
            "CMD: seqtk-trinity seq -A -R 2  /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.hs.1M.right.fq >> right.fa\n",
            "CMD finished (1 seconds)\n",
            "CMD: seqtk-trinity seq -r -A -R 1  /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.log.1M.left.fq >> left.fa\n",
            "CMD finished (1 seconds)\n",
            "CMD: seqtk-trinity seq -A -R 2  /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.ds.1M.right.fq >> right.fa\n",
            "CMD finished (1 seconds)\n",
            "CMD: seqtk-trinity seq -r -A -R 1  /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.hs.1M.left.fq >> left.fa\n",
            "CMD finished (1 seconds)\n",
            "CMD: seqtk-trinity seq -A -R 2  /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.plat.1M.right.fq >> right.fa\n",
            "CMD finished (3 seconds)\n",
            "CMD: seqtk-trinity seq -r -A -R 1  /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.plat.1M.left.fq >> left.fa\n",
            "CMD finished (5 seconds)\n",
            "CMD finished (4 seconds)\n",
            "CMD: touch left.fa.ok\n",
            "CMD finished (0 seconds)\n",
            "CMD: touch right.fa.ok\n",
            "CMD finished (0 seconds)\n",
            "Done converting input files.CMD: cat left.fa right.fa > both.fa\n",
            "CMD finished (20 seconds)\n",
            "CMD: touch both.fa.ok\n",
            "CMD finished (0 seconds)\n",
            "-kmer counting.\n",
            "-------------------------------------------\n",
            "----------- Jellyfish  --------------------\n",
            "-- (building a k-mer catalog from reads) --\n",
            "-------------------------------------------\n",
            "\n",
            "CMD: jellyfish count -t 10 -m 25 -s 100000000  both.fa\n",
            "CMD finished (69 seconds)\n",
            "CMD: jellyfish histo -t 10 -o jellyfish.K25.min2.kmers.fa.histo mer_counts.jf\n",
            "CMD finished (3 seconds)\n",
            "CMD: jellyfish dump -L 2 mer_counts.jf > jellyfish.K25.min2.kmers.fa\n",
            "CMD finished (5 seconds)\n",
            "CMD: touch jellyfish.K25.min2.kmers.fa.success\n",
            "CMD finished (0 seconds)\n",
            "-generating stats files\n",
            "CMD: /usr/local/bin/util/..//Inchworm/bin/fastaToKmerCoverageStats --reads left.fa --kmers jellyfish.K25.min2.kmers.fa --kmer_size 25  --num_threads 5  > left.fa.K25.stats\n",
            "CMD: /usr/local/bin/util/..//Inchworm/bin/fastaToKmerCoverageStats --reads right.fa --kmers jellyfish.K25.min2.kmers.fa --kmer_size 25  --num_threads 5  > right.fa.K25.stats\n",
            "-reading Kmer occurrences...\n",
            "-reading Kmer occurrences...\n",
            "\n",
            " done parsing 11516918 Kmers, 10858353 added, taking 26 seconds.\n",
            "\n",
            " done parsing 11516918 Kmers, 10858353 added, taking 27 seconds.\n",
            "STATS_GENERATION_TIME: 183 seconds.\n",
            "STATS_GENERATION_TIME: 183 seconds.\n",
            "CMD finished (214 seconds)\n",
            "CMD finished (214 seconds)\n",
            "CMD: touch left.fa.K25.stats.ok\n",
            "CMD finished (0 seconds)\n",
            "CMD: touch right.fa.K25.stats.ok\n",
            "CMD finished (0 seconds)\n",
            "-sorting each stats file by read name.\n",
            "CMD: head -n1 left.fa.K25.stats > left.fa.K25.stats.sort && tail -n +2 left.fa.K25.stats | /usr/bin/sort --parallel=10 -k1,1 -T . -S 10G >> left.fa.K25.stats.sort\n",
            "CMD: head -n1 right.fa.K25.stats > right.fa.K25.stats.sort && tail -n +2 right.fa.K25.stats | /usr/bin/sort --parallel=10 -k1,1 -T . -S 10G >> right.fa.K25.stats.sort\n",
            "tcmalloc: large alloc 10737426432 bytes == 0x55f09577c000 @  0x7f5915f551e7 0x55f094af4718 0x55f094af35a1 0x7f5915933bf7 0x55f094af402a\n",
            "tcmalloc: large alloc 10737426432 bytes == 0x55adb458e000 @  0x7f9fcbbc01e7 0x55adb34ac718 0x55adb34ab5a1 0x7f9fcb59ebf7 0x55adb34ac02a\n",
            "CMD finished (5 seconds)\n",
            "CMD finished (5 seconds)\n",
            "CMD: touch left.fa.K25.stats.sort.ok\n",
            "CMD finished (0 seconds)\n",
            "CMD: touch right.fa.K25.stats.sort.ok\n",
            "CMD finished (0 seconds)\n",
            "-defining normalized reads\n",
            "CMD: /usr/local/bin/util/..//util/support_scripts//nbkc_merge_left_right_stats.pl --left left.fa.K25.stats.sort --right right.fa.K25.stats.sort --sorted > pairs.K25.stats\n",
            "-opening left.fa.K25.stats.sort\n",
            "-opening right.fa.K25.stats.sort\n",
            "-done opening files.\n",
            "CMD finished (60 seconds)\n",
            "CMD: touch pairs.K25.stats.ok\n",
            "CMD finished (0 seconds)\n",
            "CMD: /usr/local/bin/util/..//util/support_scripts//nbkc_normalize.pl --stats_file pairs.K25.stats --max_cov 200  --min_cov 1 --max_CV 10000 > pairs.K25.stats.C200.maxCV10000.accs\n",
            "2788162 / 4000000 = 69.70% reads selected during normalization.\n",
            "0 / 4000000 = 0.00% reads discarded as likely aberrant based on coverage profiles.\n",
            "0 / 4000000 = 0.00% reads discarded as below minimum coverage threshold=1\n",
            "CMD finished (38 seconds)\n",
            "CMD: touch pairs.K25.stats.C200.maxCV10000.accs.ok\n",
            "CMD finished (0 seconds)\n",
            "-search and capture.\n",
            "-preparing to extract selected reads from: /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.log.1M.right.fq /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.hs.1M.right.fq /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.ds.1M.right.fq /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.plat.1M.right.fq ... done prepping, now search and capture.\n",
            "-capturing normalized reads from: /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.log.1M.right.fq\n",
            "-preparing to extract selected reads from: /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.ds.1M.left.fq /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.log.1M.left.fq /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.hs.1M.left.fq /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.plat.1M.left.fq ... done prepping, now search and capture.\n",
            "-capturing normalized reads from: /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.ds.1M.left.fq\n",
            "-capturing normalized reads from: /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.log.1M.left.fq\n",
            "-capturing normalized reads from: /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.hs.1M.right.fq\n",
            "-capturing normalized reads from: /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.ds.1M.right.fq\n",
            "-capturing normalized reads from: /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.hs.1M.left.fq\n",
            "-capturing normalized reads from: /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.plat.1M.right.fq\n",
            "-capturing normalized reads from: /content/TrinityNatureProtocolTutorial/1M_READS_sample/Sp.plat.1M.left.fq\n",
            "CMD: touch /content/TRINITY/insilico_read_normalization/Sp.ds.1M.left.fq_ext_all_reads.normalized_K25_maxC200_minC1_maxCV10000.fq.ok\n",
            "CMD finished (0 seconds)\n",
            "CMD: touch /content/TRINITY/insilico_read_normalization/Sp.log.1M.right.fq_ext_all_reads.normalized_K25_maxC200_minC1_maxCV10000.fq.ok\n",
            "CMD finished (0 seconds)\n",
            "CMD: ln -sf /content/TRINITY/insilico_read_normalization/Sp.ds.1M.left.fq_ext_all_reads.normalized_K25_maxC200_minC1_maxCV10000.fq left.norm.fq\n",
            "CMD finished (0 seconds)\n",
            "CMD: ln -sf /content/TRINITY/insilico_read_normalization/Sp.log.1M.right.fq_ext_all_reads.normalized_K25_maxC200_minC1_maxCV10000.fq right.norm.fq\n",
            "CMD finished (0 seconds)\n",
            "-removing tmp dir /content/TRINITY/insilico_read_normalization/tmp_normalized_reads\n",
            "\n",
            "\n",
            "Normalization complete. See outputs: \n",
            "\t/content/TRINITY/insilico_read_normalization/Sp.ds.1M.left.fq_ext_all_reads.normalized_K25_maxC200_minC1_maxCV10000.fq\n",
            "\t/content/TRINITY/insilico_read_normalization/Sp.log.1M.right.fq_ext_all_reads.normalized_K25_maxC200_minC1_maxCV10000.fq\n",
            "Friday, December 10, 2021: 03:49:04\tCMD: touch /content/TRINITY/insilico_read_normalization/normalization.ok\n",
            "Converting input files. (in parallel)Friday, December 10, 2021: 03:49:04\tCMD: cat /content/TRINITY/insilico_read_normalization/left.norm.fq | seqtk-trinity seq -r -A -R 1 - >> left.fa\n",
            "Friday, December 10, 2021: 03:49:04\tCMD: cat /content/TRINITY/insilico_read_normalization/right.norm.fq | seqtk-trinity seq -A -R 2 - >> right.fa\n",
            "Friday, December 10, 2021: 03:49:07\tCMD: touch right.fa.ok\n",
            "Friday, December 10, 2021: 03:49:07\tCMD: touch left.fa.ok\n",
            "Friday, December 10, 2021: 03:49:07\tCMD: touch left.fa.ok right.fa.ok\n",
            "Friday, December 10, 2021: 03:49:07\tCMD: cat left.fa right.fa > /content/TRINITY/both.fa\n",
            "Friday, December 10, 2021: 03:49:09\tCMD: touch /content/TRINITY/both.fa.ok\n",
            "-------------------------------------------\n",
            "----------- Jellyfish  --------------------\n",
            "-- (building a k-mer (25) catalog from reads) --\n",
            "-------------------------------------------\n",
            "\n",
            "* [Fri Dec 10 03:49:10 2021] Running CMD: jellyfish count -t 10 -m 25 -s 100000000 -o mer_counts.25.asm.jf /content/TRINITY/both.fa\n",
            "* [Fri Dec 10 03:50:04 2021] Running CMD: jellyfish dump -L 1 mer_counts.25.asm.jf > jellyfish.kmers.25.asm.fa\n",
            "* [Fri Dec 10 03:50:14 2021] Running CMD: jellyfish histo -t 10 -o jellyfish.kmers.25.asm.fa.histo mer_counts.25.asm.jf\n",
            "----------------------------------------------\n",
            "--------------- Inchworm (K=25, asm) ---------------------\n",
            "-- (Linear contig construction from k-mers) --\n",
            "----------------------------------------------\n",
            "\n",
            "* [Fri Dec 10 03:50:16 2021] Running CMD: /usr/local/bin/Inchworm/bin//inchworm --kmers jellyfish.kmers.25.asm.fa --run_inchworm -K 25 --monitor 1   --num_threads 6  --PARALLEL_IWORM   --min_any_entropy 1.0   -L 25  --no_prune_error_kmers  > /content/TRINITY/inchworm.fa.tmp\n",
            "Kmer length set to: 25\n",
            "Min assembly length set to: 25\n",
            "Monitor turned on, set to: 1\n",
            "min entropy set to: 1\n",
            "setting number of threads to: 6\n",
            "-setting parallel iworm mode.\n",
            "-reading Kmer occurrences...\n",
            " [37M] Kmers parsed.     \n",
            " done parsing 37080747 Kmers, 37080747 added, taking 47 seconds.\n",
            "\n",
            "TIMING KMER_DB_BUILDING 47 s.\n",
            "Pruning kmers (min_kmer_count=1 min_any_entropy=1 min_ratio_non_error=0.005)\n",
            "Pruned 77632 kmers from catalog.\n",
            "\tPruning time: 13 seconds = 0.216667 minutes.\n",
            "\n",
            "TIMING PRUNING 13 s.\n",
            "-populating the kmer seed candidate list.\n",
            "Kcounter hash size: 37080747\n",
            "tcmalloc: large alloc 1073741824 bytes == 0x557b02f02000 @  0x7ff7fcf3f887 0x557a6181def5 0x557a6181c98f 0x557a618110c4 0x557a61809b73 0x557a6180355b 0x7ff7fbfccbf7 0x557a6180377a\n",
            "Processed 37003115 non-zero abundance kmers in kcounter.\n",
            "-Not sorting list of kmers, given parallel mode in effect.\n",
            "-beginning inchworm contig assembly.\n",
            "Total kcounter hash size: 37080747 vs. sorted list size: 37003115\n",
            "num threads set to: 6\n",
            "Done opening file. tmp.iworm.fa.pid_5636.thread_0\n",
            "Done opening file. tmp.iworm.fa.pid_5636.thread_1\n",
            "Done opening file. tmp.iworm.fa.pid_5636.thread_2\n",
            "Done opening file. tmp.iworm.fa.pid_5636.thread_3\n",
            "Done opening file. tmp.iworm.fa.pid_5636.thread_4\n",
            "Done opening file. tmp.iworm.fa.pid_5636.thread_5\n",
            "\n",
            "\tIworm contig assembly time: 52 seconds = 0.866667 minutes.\n",
            "\n",
            "TIMING CONTIG_BUILDING 52 s.\n",
            "\n",
            "TIMING PROG_RUNTIME 123 s.\n",
            "* [Fri Dec 10 03:52:29 2021] Running CMD: mv /content/TRINITY/inchworm.fa.tmp /content/TRINITY/inchworm.fa\n",
            "Friday, December 10, 2021: 03:52:29\tCMD: touch /content/TRINITY/inchworm.fa.finished\n",
            "--------------------------------------------------------\n",
            "-------------------- Chrysalis -------------------------\n",
            "-- (Contig Clustering & de Bruijn Graph Construction) --\n",
            "--------------------------------------------------------\n",
            "\n",
            "inchworm_target: /content/TRINITY/both.fa\n",
            "bowtie_reads_fa: /content/TRINITY/both.fa\n",
            "chrysalis_reads_fa: /content/TRINITY/both.fa\n",
            "* [Fri Dec 10 03:52:29 2021] Running CMD: /usr/local/bin/util/support_scripts/filter_iworm_by_min_length_or_cov.pl /content/TRINITY/inchworm.fa 100 10 > /content/TRINITY/chrysalis/inchworm.fa.min100\n",
            "* [Fri Dec 10 03:52:31 2021] Running CMD: /usr/bin/bowtie2-build --threads 10 -o 3 /content/TRINITY/chrysalis/inchworm.fa.min100 /content/TRINITY/chrysalis/inchworm.fa.min100 1>/dev/null\n",
            "* [Fri Dec 10 03:53:15 2021] Running CMD: bash -c \" set -o pipefail;/usr/bin/bowtie2 --local -k 2 --no-unal --threads 10 -f --score-min G,20,8 -x /content/TRINITY/chrysalis/inchworm.fa.min100 /content/TRINITY/both.fa  | samtools view -@ 10 -F4 -Sb - | samtools sort -m 1073741824 -@ 10 -no - - > /content/TRINITY/chrysalis/iworm.bowtie.nameSorted.bam\" \n",
            "* [Fri Dec 10 04:00:29 2021] Running CMD: /usr/local/bin/util/support_scripts/scaffold_iworm_contigs.pl /content/TRINITY/chrysalis/iworm.bowtie.nameSorted.bam /content/TRINITY/chrysalis/inchworm.fa.min100 > /content/TRINITY/chrysalis/iworm_scaffolds.txt\n",
            "* [Fri Dec 10 04:01:40 2021] Running CMD: /usr/local/bin/Chrysalis/bin/GraphFromFasta -i /content/TRINITY/chrysalis/inchworm.fa.min100 -r /content/TRINITY/both.fa -min_contig_length 200 -min_glue 2 -glue_factor 0.05 -min_iso_ratio 0.05 -t 10 -k 24 -kk 48  -strand  -scaffolding /content/TRINITY/chrysalis/iworm_scaffolds.txt  > /content/TRINITY/chrysalis/iworm_cluster_welds_graph.txt\n",
            "* [Fri Dec 10 04:02:34 2021] Running CMD: /usr/bin/sort --parallel=10 -T . -S 20G  -k9,9gr /content/TRINITY/chrysalis/iworm_cluster_welds_graph.txt > /content/TRINITY/chrysalis/iworm_cluster_welds_graph.txt.sorted\n",
            "* [Fri Dec 10 04:02:34 2021] Running CMD: /usr/local/bin/util/support_scripts/annotate_chrysalis_welds_with_iworm_names.pl /content/TRINITY/chrysalis/inchworm.fa.min100 /content/TRINITY/chrysalis/iworm_cluster_welds_graph.txt.sorted > /content/TRINITY/chrysalis/iworm_cluster_welds_graph.txt.sorted.wIwormNames\n",
            "* [Fri Dec 10 04:02:35 2021] Running CMD: /usr/local/bin/Chrysalis/bin/BubbleUpClustering -i /content/TRINITY/chrysalis/inchworm.fa.min100  -weld_graph /content/TRINITY/chrysalis/iworm_cluster_welds_graph.txt.sorted -min_contig_length 200 -max_cluster_size 25  > /content/TRINITY/chrysalis/GraphFromIwormFasta.out\n",
            "* [Fri Dec 10 04:02:35 2021] Running CMD: /usr/local/bin/Chrysalis/bin/CreateIwormFastaBundle -i /content/TRINITY/chrysalis/GraphFromIwormFasta.out -o /content/TRINITY/chrysalis/bundled_iworm_contigs.fasta -min 200\n",
            "* [Fri Dec 10 04:02:36 2021] Running CMD: /usr/local/bin/Chrysalis/bin/ReadsToTranscripts -i /content/TRINITY/both.fa -f /content/TRINITY/chrysalis/bundled_iworm_contigs.fasta -o /content/TRINITY/chrysalis/readsToComponents.out -t 10 -max_mem_reads 50000000  -strand  -p 10 \n",
            "* [Fri Dec 10 04:10:49 2021] Running CMD: /usr/bin/sort --parallel=10 -T . -S 20G -k 1,1n -k3,3nr -k2,2 /content/TRINITY/chrysalis/readsToComponents.out > /content/TRINITY/chrysalis/readsToComponents.out.sort\n",
            "Friday, December 10, 2021: 04:11:09\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_0\n",
            "Friday, December 10, 2021: 04:11:10\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_1\n",
            "Friday, December 10, 2021: 04:11:11\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_2\n",
            "Friday, December 10, 2021: 04:11:11\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_3\n",
            "Friday, December 10, 2021: 04:11:12\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_4\n",
            "Friday, December 10, 2021: 04:11:12\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_5\n",
            "Friday, December 10, 2021: 04:11:13\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_6\n",
            "Friday, December 10, 2021: 04:11:13\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_7\n",
            "Friday, December 10, 2021: 04:11:14\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_8\n",
            "Friday, December 10, 2021: 04:11:14\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_9\n",
            "Friday, December 10, 2021: 04:11:15\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_10\n",
            "Friday, December 10, 2021: 04:11:15\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_11\n",
            "Friday, December 10, 2021: 04:11:15\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_12\n",
            "Friday, December 10, 2021: 04:11:16\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_13\n",
            "Friday, December 10, 2021: 04:11:16\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_14\n",
            "Friday, December 10, 2021: 04:11:17\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_15\n",
            "Friday, December 10, 2021: 04:11:17\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_16\n",
            "Friday, December 10, 2021: 04:11:17\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_17\n",
            "Friday, December 10, 2021: 04:11:18\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_18\n",
            "Friday, December 10, 2021: 04:11:18\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_19\n",
            "Friday, December 10, 2021: 04:11:19\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_20\n",
            "Friday, December 10, 2021: 04:11:19\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_21\n",
            "Friday, December 10, 2021: 04:11:20\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_22\n",
            "Friday, December 10, 2021: 04:11:20\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_23\n",
            "Friday, December 10, 2021: 04:11:20\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_24\n",
            "Friday, December 10, 2021: 04:11:21\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_25\n",
            "Friday, December 10, 2021: 04:11:21\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_26\n",
            "Friday, December 10, 2021: 04:11:22\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_27\n",
            "Friday, December 10, 2021: 04:11:22\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_28\n",
            "Friday, December 10, 2021: 04:11:22\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_29\n",
            "Friday, December 10, 2021: 04:11:23\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_30\n",
            "Friday, December 10, 2021: 04:11:23\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_31\n",
            "Friday, December 10, 2021: 04:11:24\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_32\n",
            "Friday, December 10, 2021: 04:11:24\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_33\n",
            "Friday, December 10, 2021: 04:11:24\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_34\n",
            "Friday, December 10, 2021: 04:11:25\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_35\n",
            "Friday, December 10, 2021: 04:11:25\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_36\n",
            "Friday, December 10, 2021: 04:11:26\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_37\n",
            "Friday, December 10, 2021: 04:11:26\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_38\n",
            "Friday, December 10, 2021: 04:11:26\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_39\n",
            "Friday, December 10, 2021: 04:11:27\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_40\n",
            "Friday, December 10, 2021: 04:11:27\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_41\n",
            "Friday, December 10, 2021: 04:11:28\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_42\n",
            "Friday, December 10, 2021: 04:11:28\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_43\n",
            "Friday, December 10, 2021: 04:11:29\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_44\n",
            "Friday, December 10, 2021: 04:11:29\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_45\n",
            "Friday, December 10, 2021: 04:11:30\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_46\n",
            "Friday, December 10, 2021: 04:11:30\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_47\n",
            "Friday, December 10, 2021: 04:11:31\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_48\n",
            "Friday, December 10, 2021: 04:11:31\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_49\n",
            "Friday, December 10, 2021: 04:11:31\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_50\n",
            "Friday, December 10, 2021: 04:11:32\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_51\n",
            "Friday, December 10, 2021: 04:11:32\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_52\n",
            "Friday, December 10, 2021: 04:11:33\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_53\n",
            "Friday, December 10, 2021: 04:11:33\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_54\n",
            "Friday, December 10, 2021: 04:11:34\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_55\n",
            "Friday, December 10, 2021: 04:11:34\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_56\n",
            "Friday, December 10, 2021: 04:11:35\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_57\n",
            "Friday, December 10, 2021: 04:11:35\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_58\n",
            "Friday, December 10, 2021: 04:11:35\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_59\n",
            "Friday, December 10, 2021: 04:11:36\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_60\n",
            "Friday, December 10, 2021: 04:11:36\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_61\n",
            "Friday, December 10, 2021: 04:11:37\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_62\n",
            "Friday, December 10, 2021: 04:11:37\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_63\n",
            "Friday, December 10, 2021: 04:11:37\tCMD: mkdir -p /content/TRINITY/read_partitions/Fb_0/CBin_64\n",
            "Friday, December 10, 2021: 04:11:37\tCMD: touch /content/TRINITY/partitioned_reads.files.list.ok\n",
            "Friday, December 10, 2021: 04:11:37\tCMD: /usr/local/bin/util/support_scripts/write_partitioned_trinity_cmds.pl --reads_list_file /content/TRINITY/partitioned_reads.files.list --CPU 1 --max_memory 1G  --run_as_paired  --SS_lib_type F  --seqType fa --trinity_complete --full_cleanup  --no_salmon  > recursive_trinity.cmds\n",
            "Friday, December 10, 2021: 04:11:38\tCMD: touch recursive_trinity.cmds.ok\n",
            "Friday, December 10, 2021: 04:11:38\tCMD: touch recursive_trinity.cmds.ok\n",
            "\n",
            "\n",
            "--------------------------------------------------------------------------------\n",
            "------------ Trinity Phase 2: Assembling Clusters of Reads ---------------------\n",
            "------- (involving the Inchworm, Chrysalis, Butterfly trifecta ) ---------------\n",
            "--------------------------------------------------------------------------------\n",
            "\n",
            "Friday, December 10, 2021: 04:11:38\tCMD: /usr/local/bin/trinity-plugins/BIN/ParaFly -c recursive_trinity.cmds -CPU 10 -v -shuffle \n",
            "Number of Commands: 6413\n",
            "succeeded(6413)   100% completed.    \n",
            "\n",
            "All commands completed successfully. :-)\n",
            "\n",
            "\n",
            "\n",
            "** Harvesting all assembled transcripts into a single multi-fasta file...\n",
            "\n",
            "Friday, December 10, 2021: 08:36:08\tCMD: find /content/TRINITY/read_partitions/ -name '*inity.fasta'  | /usr/local/bin/util/support_scripts/partitioned_trinity_aggregator.pl --token_prefix TRINITY_DN --output_prefix /content/TRINITY/Trinity.tmp\n",
            "Friday, December 10, 2021: 08:36:08\tCMD: mv Trinity.tmp.fasta /content/TRINITY.Trinity.fasta\n",
            "\n",
            "\n",
            "#############################################################################\n",
            "Finished.  Final Trinity assemblies are written to /content/TRINITY.Trinity.fasta\n",
            "#############################################################################\n",
            "\n",
            "\n",
            "Friday, December 10, 2021: 08:36:08\tCMD: /usr/local/bin/util/support_scripts/get_Trinity_gene_to_trans_map.pl /content/TRINITY.Trinity.fasta > /content/TRINITY.Trinity.fasta.gene_trans_map\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "source": [
        "%cd ..\n",
        "%cd .."
      ],
      "metadata": {
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "id": "EbG_Sj5G097-",
        "outputId": "0b22fce9-0a8a-46d0-d5f7-9aed53dd3b18"
      },
      "execution_count": 19,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "/content/TrinityNatureProtocolTutorial\n",
            "/content\n"
          ]
        }
      ]
    },
    {
      "cell_type": "markdown",
      "source": [
        "**04. VISUALIZATION**"
      ],
      "metadata": {
        "id": "tEaGDVdo2vOe"
      }
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "1P-r_CHeYIaA",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "55bfcdbf-1594-4a20-dafa-2ab8d0a48e91"
      },
      "source": [
        "#VIEW THE HEAD OF ASSEMBLED FILE\n",
        "!head /content/TRINITY.Trinity.fasta"
      ],
      "execution_count": 20,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            ">TRINITY_DN4869_c0_g1_i1 len=1148 path=[0:0-1147]\n",
            "ACACATTGTATCTTCAGAGGCTTCAATTTTTTACGAACAACATCAATTTTTTAAAACCTGAGCATAATCTATAATTTTATAAACGTAGTTTTCCGAAACAACGAATCACTCTTTTATTAATTTGCAGATCAAATCACTCTTTCACAAAACAAAATGCTTTCCTTTTTTACAAAAAATACCTTGACTAAAAGGAAGCTTATTATGTTAGCTTTGGCTATAGTTTTCACCTTTTTTGCCTTCGGGCTATATTTTATTCCGCACGACGAGATTTCTGTTTTTGATTTTAAACTTCCTGCACTTCAATATGAAACTACGGTGACATCTCTCGATAATTTTTTGATCGGCGGAAGCACAACTTTGTATACAGCAACCGTTAATCACGAAGATCTCAATGAACCAAAATCACATGAAATAGTAATGCTTCTTGCTAGCGATGGAAACGTTGGAAGTTTTGAATCTAACTTACTAGATGATTGCTTTAAAAATCGTATAGAGTATGCGAAACTTCAAAACTACAATTTTGAATTCGTTAACGTTTCCAGTCTGGTAGTGCCACCTGTATGGGGAAAGATGCCCGCAATTCTACAAACAATGCGCAAGTATCCATCCGCAAAATGGATCTGGTGGCTAGATCAGGATGCTCTTATAATGAACAAAAATCTGTCCTTACAGGAATTATTTCTCTCACCTGCAATGCTCCAAAAATCATTGCTTAGGGAGCAACCAATTATAAACTCTTTCGGAGAAGATAATTTCAGAATCACTCCAGCGGCTTACTCCAAGGAAATGATAGAACAAATACAATTTCTAATTTCTCAGGATCATAACGGATTGAATGCAGGAAGCTTTTTAGTACGAAATTCAAGATCCATCGCTTTACTAATGGATTTATTGACTGATCCTTCTTTGGCAGATGCAGGAGTCGTGCGTCATGAACAAGATTTGATCGGCTATTTCATTCAAAAACACTCGCAGGTAGCTTCCATGGTTGGCATTCTTCCTCAAAGATTTATCAATGCTTTTCATGAAGGACCCCCTACAATGCAATGGCAAGAAGGAGATCTAGCTCTTCATTTCGCTGGATGTTGGGTTGAAAACAAATGCGCCGAGTTATGGACTAAATACAAAGATAAAATAATTTAAAAGGC\n",
            ">TRINITY_DN4888_c0_g1_i1 len=298 path=[0:0-297]\n",
            "GTTGAATTACGTTATTGTGGGTGTTTAGGTTTGGGTTCTCTTCATTTGCAAGAACAGACATCTGATCCCAGCATTGTTTCTGATCCACGCACCGCACGCAACATCGTTTCCATTCCACATCAAATTCACTTTTCAGGAACTTGCTATTCAGTTGCAGCCGGCGGATGGCAAAGTGCTGCTCTCGCCATTTCGGAATCTGTTTTACCCGAAGTCAATGTGCCTCCAATTACAACTTTTTCTATCATCCGACCATCCAGAGTTCCAGCGTTGCGCTTTTTAGCTGCCCCTGCGACTCGGA\n",
            ">TRINITY_DN4875_c0_g1_i1 len=218 path=[0:0-217]\n",
            "ATCATTATCGATGAAATTGCAGGTTACTGGGTTCAAAGAAAAAAGTAGTTTTATCGCGTAAAAAGAACATAGGGCTGGAAAAATTATTCCTTACATGTTTAAAAGTAAGAGTGGGCGATATAAAAACATAAAAAACAAATGTTTGTAAGAGAACGACTGAGGAGTATCCGATTAATATAATCAACTGAATGTGATCATATCTTTCTAAGAAAAAAACA\n",
            ">TRINITY_DN4877_c0_g1_i1 len=244 path=[0:0-243]\n",
            "AGTATTGAATTTTCTTTTAACAAATTTGTCTTTTCCAAAACTAATTCGTCAAATTTGGTACGGATATCACCAATTTCCGCATGCATGTGCCGATTATCTTGTCTAATGCGCAAGTTCTTCTCACGTTCTTCTTCCAACTGTACTTTGAAACCATTTAGCTCCTTTTGATAACGTTGAAAGAGTAGTTGATCCTTCTTTTCTCGTTCCTCGAGATCTTGAACACGAATGACATGAAACTCATCAA\n",
            ">TRINITY_DN4874_c0_g1_i1 len=879 path=[0:0-878]\n",
            "AATCGTTACAGAGGATTTTTACTTCGCAAGTAAGCGTGCATGCAGAATTCCAATTTTCTACTAAGTGTATCCAGCTAACTCCTCTATTATTTGGCCAGTCAACTTATTTTCATTGATCTGATCCCTTTTTATGTCTAGTGGTTTTTGAGTTTTGCATTAGAACATCAAACATTTCGTTGAAATCTCCTTCTTACAATTTTTAAAATTTAGAATAATGCGACCAGCAAAATCTGTCGAAGGTTATATCATCATAGTCACTGGAGTTCATCCAGAAGCTACCGAAGAACAAGTAGAGGATTTATTTGCAGATTTTGGACCGGTCAAAAACTTGCACCTCAATTTAGATCGACGTACAGGTTATGTAAAGGGATATGCACTGATAGAATATGCTACATTGGAGCAAGCCCAGAAAGCTGTAGACGAAAAAAACCTCTCTTTGTTAGATGAAAAACTGGAGGTTGACTTTGCGTTTTTAGAGCCTCCAGAACGAGCTCCCAGGCCATCTATTTCTACGAGGAGCCGAAGTCAATCTCCTGAAGTTCAACATCGAGATAGAGATGTAGCAATGGCTGAGCCCTAAATGGGCTTCTGAAAATACACAAAACTTATTTATGTTAGTAGAGAAAAGGAGACAGGCTTGGAACTTAAGGGCATGTAACAACACTTGGATCTACGCTTCGAAAACAAAACTTATTCTCAAATCTCAACGTTCTTCAGCTATGAAGAGCCTTAACGAATAATTTAATGACTTTATTGAAGAAGAAACCTCGTTTCTTCAATCTCTTTCTCTTGTACAGATACACGATACAATTTACAATGCAAGCATGGAGATTTATATGGTTATTTTGTATTTTGCTTAATAATAAGCAGAATTTTATA\n"
          ]
        }
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "JSIeckdZYOl6",
        "colab": {
          "base_uri": "https://localhost:8080/"
        },
        "outputId": "c853d04a-0683-4161-8ac0-3adfb9769afb"
      },
      "source": [
        "#CHECK THE STATISTICS OF THE ASSEMBLED CONTIGS\n",
        "!/content/trinityrnaseq-v2.13.2/util/TrinityStats.pl TRINITY.Trinity.fasta"
      ],
      "execution_count": 21,
      "outputs": [
        {
          "output_type": "stream",
          "name": "stdout",
          "text": [
            "\n",
            "\n",
            "################################\n",
            "## Counts of transcripts, etc.\n",
            "################################\n",
            "Total trinity 'genes':\t8188\n",
            "Total trinity transcripts:\t9041\n",
            "Percent GC: 38.23\n",
            "\n",
            "########################################\n",
            "Stats based on ALL transcript contigs:\n",
            "########################################\n",
            "\n",
            "\tContig N10: 3303\n",
            "\tContig N20: 2703\n",
            "\tContig N30: 2300\n",
            "\tContig N40: 1959\n",
            "\tContig N50: 1694\n",
            "\n",
            "\tMedian contig length: 840\n",
            "\tAverage contig: 1107.42\n",
            "\tTotal assembled bases: 10012195\n",
            "\n",
            "\n",
            "#####################################################\n",
            "## Stats based on ONLY LONGEST ISOFORM per 'GENE':\n",
            "#####################################################\n",
            "\n",
            "\tContig N10: 3256\n",
            "\tContig N20: 2680\n",
            "\tContig N30: 2272\n",
            "\tContig N40: 1944\n",
            "\tContig N50: 1685\n",
            "\n",
            "\tMedian contig length: 814\n",
            "\tAverage contig: 1087.64\n",
            "\tTotal assembled bases: 8905570\n",
            "\n",
            "\n",
            "\n"
          ]
        }
      ]
    }
  ]
}